---
title: "Anotación 3 RBPs"
subtitle: ""
author: "Cristina Muntañola Valero"
date: "`r Sys.Date()`"
output:
  pdf_document:
    toc: true 
    toc_depth: 3
    df_print: paged
  html_document:
    toc: true
    toc_depth: 3
    df_print: paged
urlcolor: blue
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

Esta parte se corresponde con la anotación de las comparaciones de los GSE8879, GSE10609, GSE14818 y GSE26713.

# Carga datos artículos

## Transcriptome-wide discovery

Huang, Rongbing, et al. "Transcriptome-wide discovery of coding and noncoding RNA-binding proteins." Proceedings of the National Academy of Sciences 115.17 (2018): E3879-E3887.

Wang, E., Lu, S. X., Pastore, A., Chen, X., Imig, J., Lee, S. C. W., ... & Aifantis, I. (2019). Targeting an RNA-binding protein network in acute myeloid leukemia. Cancer cell, 35(3), 369-384.


Sebestyén, E., Singh, B., Miñana, B., Pagès, A., Mateo, F., Pujana, M. A., ... & Eyras, E. (2016). Large-scale analysis of genome and transcriptome alterations in multiple tumors unveils novel cancer-relevant splicing networks. Genome research, 26(6), 732-744.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
# setwd('C:/Users/laura/Downloads/')
setwd('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library("readxl")
# Transcriptome1 <- read_excel("C:/Users/laura/Downloads/Transcriptome_wide_discovery_1.xlsx")

Transcriptome1 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Transcriptome_wide_discovery_1.xlsx")

```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library("readxl")

# Targeting1 <- read_excel("C:/Users/laura/Downloads/Targeting_an_RNA_Binding_Protein_Network_1.xlsx")

Targeting1 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Targeting_an_RNA_Binding_Protein_Network_1.xlsx")
Targeting2 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Targeting2.xlsx")
Targeting5 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Targeting5.xlsx")
```

```{r, echo=TRUE, warning=FALSE, message=TRUE, include=TRUE}


# Large_scale <- read_excel("C:/Users/laura/Downloads/Large_scale_analysis.xlsx", sheet = 3, skip = 1)

Large_scale <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Large_scale_analysis.xlsx", sheet = 2, skip = 1)
dim(Large_scale)
```


Quitamos las filas y las columnas de cada dataset que no nos interesan.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Transcriptome1)<-Transcriptome1[1,]
Transcriptome1<-Transcriptome1[c(-1,-2,-3),]
Transcriptome1<-Transcriptome1[,-c(6:41, 43)]
colnames(Transcriptome1)<-c("Representative_protein_IDs", "Protein_IDs", "Majority_protein_IDs",
                            "Protein_name", "Gene", "Adjusted_P_values","Classification")
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Targeting1)<-Targeting1[1,]
Targeting1<-Targeting1[c(-1),]
Targeting1<-Targeting1[,-c(4, 5, 7)]

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

colnames(Targeting2) <- Targeting2[1,]
Targeting2<-Targeting2[-1,]
Targeting2<-Targeting2[,-c(3:13)]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Targeting5)<-Targeting5[1,]
Targeting5<-Targeting5[-1,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
Targeting5<-Targeting5[,-c(1:5, 8)]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Targeting5)<-c('id', 'gene', 'gene_description')
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
Large_scale<-Large_scale[,-c(2,3)]

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Large_scale)<-c('HGNC_symbol', 'type')

```


# GSE62156


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE, error=TRUE}
library(readxl)
CALMAF10_CALMHOX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMHOX.xlsx')


CALMAF10_CALMHOXAUnk<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMHOXAUnk.xlsx')
CALMAF10_CALMImmature<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMImmature.xlsx')

CALMAF10_CALMImmatureHOXA<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMImmatureHOXA.xlsx')

CALMAF10_CALMMLL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMMLL.xlsx')
CALMAF10_CALMTAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMTAL.xlsx')

CALMAF10_CALMTLX1<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMTLX1.xlsx')
CALMAF10_CALMTLX3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMTLX3.xlsx')
HOX_CALM<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_CALM.xlsx')



HOX_ImHOXA<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_ImHOXA.xlsx')
HOX_Immature<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_Immature.xlsx')
HOX_ImmHOXA<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_ImmHOXA.xlsx')

HOX_MLL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_MLL.xlsx')
HOX_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_TAL.xlsx')
HOX_TLX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_TLX.xlsx')

HOX_TLX1<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_TLX1.xlsx')
HOX_TLX3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_TLX3.xlsx')
HOXAUnk_HOX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXAUnk_HOX.xlsx')

HOXAUnk_Imm<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXAUnk_Imm.xlsx')
HOXAUnk_ImmHOXA<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXAUnk_ImmHOXA.xlsx')

HOXAUnk_MLL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXAUnk_MLL.xlsx')
HOXAUnk_TLX1<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXAUnk_TLX1.xlsx')


HOX_HOXA<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_HOXA.xlsx')


HOX_HOXA<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOX_Imm.xlsx')


HOXAUnk_TLX3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXAUnk_TLX3.xlsx')
HOXs_MLL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXs_MLL.xlsx')

HOXs_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXs_TAL.xlsx')
HOXs_TLX1<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXs_TLX1.xlsx')
HOXs_TLX3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_HOXs_TLX3.xlsx')

Imm_CALM<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Imm_CALM.xlsx')
Imm_HOX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Imm_HOX.xlsx')

Imm_HOXAUnk<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Imm_HOXAUnk.xlsx')
Imm_MLL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Imm_MLL.xlsx')


Imm_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Imm_TAL.xlsx')
Imm_TLX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Imm_TLX.xlsx')


Imm_TLX1<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Imm_TLX1.xlsx')
Imm_TLX3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Imm_TLX3.xlsx')

Immature_ImmHOXA<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Immature_ImmHOXA.xlsx')

Immature_MLL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Immature_MLL.xlsx')
Immature_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Immature_TAL.xlsx')

Immature_TLX1<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Immature_TLX1.xlsx')
Immature_TLX3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_Immature_TLX3.xlsx')
ImmHOXA_MLL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_ImmHOXA_MLL.xlsx')

ImmHOXA_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_ImmHOXA_TAL.xlsx')
ImmHOXA_TLX1<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_ImmHOXA_TLX1.xlsx')
ImmHOXA_TLX3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_ImmHOXA_TLX3.xlsx')

MLL_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_MLL_TAL.xlsx')


MLL_TLX1<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_MLL_TLX1.xlsx')
MLL_TLX3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_MLL_TLX3.xlsx')

TLX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/Resultado_GSE62156_TLX.xlsx')

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
ls()

```

## Transcriptome


Haremos una funcion para tener los nombres de los genes comunes a nuestra base de datos y a la de Transcriptome1. 

Los Gene de la base de Resultados son correspondientes también al Gene de Transcriptome.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CALMAF10_CALMHOX$Gene)){
  
  x<-str_split(CALMAF10_CALMHOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMAF10_CALMHOX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_CALMAF10_CALMAF10_CALMHOX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMAF10_CALMHOX_Transcriptome1[order(comparison_GSE62156_CALMAF10_CALMAF10_CALMHOX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMAF10_CALMHOX_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CALMAF10_CALMHOXAUnk$Gene)){
  
  x<-str_split(CALMAF10_CALMHOXAUnk$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMHOXAUnk_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_CALMAF10_CALMHOXAUnk_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMHOXAUnk_Transcriptome1[order(comparison_GSE62156_CALMAF10_CALMHOXAUnk_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMHOXAUnk_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CALMAF10_CALMImmature$Gene)){
  
  x<-str_split(CALMAF10_CALMImmature$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMImmature_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_CALMAF10_CALMImmature_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMImmature_Transcriptome1[order(comparison_GSE62156_CALMAF10_CALMImmature_Transcriptome1),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (!is.na(yoquese$Proteina[i]) &&
      !is.na(yoquese$Proteina[i+1]) &&
    yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMImmature_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CALMAF10_CALMImmatureHOXA$Gene)){
  
  x<-str_split(CALMAF10_CALMImmatureHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Transcriptome1[order(comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CALMAF10_CALMTAL$Gene)){
  
  x<-str_split(CALMAF10_CALMTAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_CALMAF10_CALMTAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTAL_Transcriptome1[order(comparison_GSE62156_CALMAF10_CALMTAL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTAL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CALMAF10_CALMTLX3$Gene)){
  
  x<-str_split(CALMAF10_CALMTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_CALMAF10_CALMTLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTLX3_Transcriptome1[order(comparison_GSE62156_CALMAF10_CALMTLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTLX3_Transcriptome1.xlsx", row.names = FALSE)
```


AQUI

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX_CALM$Gene)){
  
  x<-str_split(HOX_CALM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOX_CALM_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOX_CALM_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOX_CALM_Transcriptome1[order(comparison_GSE33470_HOX_CALM_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOX_CALM_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX_ImHOXA$Gene)){
  
  x<-str_split(HOX_ImHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOX_ImHOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOX_ImHOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOX_ImHOXA_Transcriptome1[order(comparison_GSE33470_HOX_ImHOXA_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOX_ImHOXA_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX_Immature$Gene)){
  
  x<-str_split(HOX_Immature$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOX_Immature_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOX_Immature_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOX_Immature_Transcriptome1[order(comparison_GSE33470_HOX_Immature_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOX_Immature_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX_ImmHOXA$Gene)){
  
  x<-str_split(HOX_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOX_ImmHOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOX_ImmHOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOX_ImmHOXA_Transcriptome1[order(comparison_GSE33470_HOX_ImmHOXA_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOX_ImmHOXA_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX_MLL$Gene)){
  
  x<-str_split(HOX_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOX_MLL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOX_MLL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOX_MLL_Transcriptome1[order(comparison_GSE33470_HOX_MLL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOX_MLL_Transcriptome1.xlsx", row.names = FALSE)
```

Salía nulo al final, bien..

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX_TAL$Gene)){
  
  x<-str_split(HOX_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOX_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOX_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOX_TAL_Transcriptome1[order(comparison_GSE33470_HOX_TAL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOX_TAL_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX_TLX$Gene)){
  
  x<-str_split(HOX_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOX_TLX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOX_TLX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOX_TLX_Transcriptome1[order(comparison_GSE33470_HOX_TLX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOX_TLX_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX_TLX1$Gene)){
  
  x<-str_split(HOX_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOX_TLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOX_TLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOX_TLX1_Transcriptome1[order(comparison_GSE33470_HOX_TLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOX_TLX1_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX_TLX3$Gene)){
  
  x<-str_split(HOX_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOX_TLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOX_TLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOX_TLX3_Transcriptome1[order(comparison_GSE33470_HOX_TLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOX_TLX3_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXAUnk_HOX$Gene)){
  
  x<-str_split(HOXAUnk_HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXAUnk_HOX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXAUnk_HOX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXAUnk_HOX_Transcriptome1[order(comparison_GSE33470_HOXAUnk_HOX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOXAUnk_HOX_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXAUnk_Imm$Gene)){
  
  x<-str_split(HOXAUnk_Imm$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXAUnk_Imm_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXAUnk_Imm_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXAUnk_Imm_Transcriptome1[order(comparison_GSE33470_HOXAUnk_Imm_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOXAUnk_Imm_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXAUnk_ImmHOXA$Gene)){
  
  x<-str_split(HOXAUnk_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXAUnk_ImmHOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXAUnk_ImmHOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXAUnk_ImmHOXA_Transcriptome1[order(comparison_GSE33470_HOXAUnk_ImmHOXA_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOXAUnk_ImmHOXA_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXAUnk_MLL$Gene)){
  
  x<-str_split(HOXAUnk_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXAUnk_MLL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXAUnk_MLL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXAUnk_MLL_Transcriptome1[order(comparison_GSE33470_HOXAUnk_MLL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE33470_HOXAUnk_MLL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXAUnk_TLX1$Gene)){
  
  x<-str_split(HOXAUnk_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXAUnk_TLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXAUnk_TLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXAUnk_TLX1_Transcriptome1[order(comparison_GSE33470_HOXAUnk_TLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_TLX1_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXAUnk_TLX3$Gene)){
  
  x<-str_split(HOXAUnk_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXAUnk_TLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXAUnk_TLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXAUnk_TLX3_Transcriptome1[order(comparison_GSE33470_HOXAUnk_TLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_TLX3_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXs_MLL$Gene)){
  
  x<-str_split(HOXs_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXs_MLL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXs_MLL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXs_MLL_Transcriptome1[order(comparison_GSE33470_HOXs_MLL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_MLL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXs_TAL$Gene)){
  
  x<-str_split(HOXs_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXs_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXs_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXs_TAL_Transcriptome1[order(comparison_GSE33470_HOXs_TAL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TAL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXs_TLX1$Gene)){
  
  x<-str_split(HOXs_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXs_TLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXs_TLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXs_TLX1_Transcriptome1[order(comparison_GSE33470_HOXs_TLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TLX1_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXs_TLX3$Gene)){
  
  x<-str_split(HOXs_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_HOXs_TLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_HOXs_TLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_HOXs_TLX3_Transcriptome1[order(comparison_GSE33470_HOXs_TLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TLX3_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Imm_CALM$Gene)){
  
  x<-str_split(Imm_CALM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_CALM_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Imm_CALM_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_CALM_Transcriptome1[order(comparison_GSE62156_Imm_CALM_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_CALM_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Imm_HOX$Gene)){
  
  x<-str_split(Imm_HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_HOX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Imm_HOX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_HOX_Transcriptome1[order(comparison_GSE62156_Imm_HOX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_HOX_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Imm_HOXAUnk$Gene)){
  
  x<-str_split(Imm_HOXAUnk$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_HOXAUnk_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Imm_HOXAUnk_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_HOXAUnk_Transcriptome1[order(comparison_GSE62156_Imm_HOXAUnk_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_HOXAUnk_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Imm_MLL$Gene)){
  
  x<-str_split(Imm_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_MLL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Imm_MLL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_MLL_Transcriptome1[order(comparison_GSE62156_Imm_MLL_Transcriptome1),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (!is.na(yoquese$Proteina[i]) &&
      !is.na(yoquese$Proteina[i+1]) &&
    yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_MLL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Imm_TAL$Gene)){
  
  x<-str_split(Imm_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Imm_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TAL_Transcriptome1[order(comparison_GSE62156_Imm_TAL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TAL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Imm_TLX$Gene)){
  
  x<-str_split(Imm_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Imm_TLX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX_Transcriptome1[order(comparison_GSE62156_Imm_TLX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Imm_TLX1$Gene)){
  
  x<-str_split(Imm_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Imm_TLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX1_Transcriptome1[order(comparison_GSE62156_Imm_TLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Imm_TLX3$Gene)){
  
  x<-str_split(Imm_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Imm_TLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX3_Transcriptome1[order(comparison_GSE62156_Imm_TLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX3_Transcriptome1.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Immature_ImmHOXA$Gene)){
  
  x<-str_split(Immature_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_ImmHOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Immature_ImmHOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_ImmHOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Immature_ImmHOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_ImmHOXA_Transcriptome1[order(comparison_GSE62156_Immature_ImmHOXA_Transcriptome1$Proteina),]

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_ImmHOXA_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Immature_MLL$Gene)){
  
  x<-str_split(Immature_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_MLL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Immature_MLL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_MLL_Transcriptome1[order(comparison_GSE62156_Immature_MLL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_MLL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Immature_TAL$Gene)){
  
  x<-str_split(Immature_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Immature_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TAL_Transcriptome1[order(comparison_GSE62156_Immature_TAL_Transcriptome1),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (!is.na(yoquese$Proteina[i]) &&
      !is.na(yoquese$Proteina[i+1]) &&
    yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TAL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Immature_TLX1$Gene)){
  
  x<-str_split(Immature_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Immature_TLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TLX1_Transcriptome1[order(comparison_GSE62156_Immature_TLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TLX1_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Immature_TLX3$Gene)){
  
  x<-str_split(Immature_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_Immature_TLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TLX3_Transcriptome1[order(comparison_GSE62156_Immature_TLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TLX3_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(ImmHOXA_MLL$Gene)){
  
  x<-str_split(ImmHOXA_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_MLL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_ImmHOXA_MLL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_MLL_Transcriptome1[order(comparison_GSE62156_ImmHOXA_MLL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_MLL_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(ImmHOXA_TAL$Gene)){
  
  x<-str_split(ImmHOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_ImmHOXA_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TAL_Transcriptome1[order(comparison_GSE62156_ImmHOXA_TAL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TAL_Transcriptome1.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(ImmHOXA_TLX1$Gene)){
  
  x<-str_split(ImmHOXA_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_ImmHOXA_TLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TLX1_Transcriptome1[order(comparison_GSE62156_ImmHOXA_TLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TLX1_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(ImmHOXA_TLX3$Gene)){
  
  x<-str_split(ImmHOXA_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_ImmHOXA_TLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TLX3_Transcriptome1[order(comparison_GSE62156_ImmHOXA_TLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TLX3_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(MLL_TAL$Gene)){
  
  x<-str_split(MLL_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_MLL_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TAL_Transcriptome1[order(comparison_GSE62156_MLL_TAL_Transcriptome1),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (!is.na(yoquese$Proteina[i]) &&
      !is.na(yoquese$Proteina[i+1]) &&
    yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TAL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(MLL_TLX1$Gene)){
  
  x<-str_split(MLL_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_MLL_TLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TLX1_Transcriptome1[order(comparison_GSE62156_MLL_TLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TLX1_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(MLL_TLX3$Gene)){
  
  x<-str_split(MLL_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_MLL_TLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TLX3_Transcriptome1[order(comparison_GSE62156_MLL_TLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TLX3_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(TLX$Gene)){
  
  x<-str_split(TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_TLX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE62156_TLX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_TLX_Transcriptome1[order(comparison_GSE62156_TLX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_TLX_Transcriptome1.xlsx", row.names = FALSE)
```



## Targeting2

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMHOX$Gene)){
  
  x<-str_split(CALMAF10_CALMHOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMHOX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_CALMAF10_CALMHOX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Salía bien, nulo...




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMHOXAUnk$Gene)){
  
  x<-str_split(CALMAF10_CALMHOXAUnk$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Salía nulo al final, bien...



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMImmature$Gene)){
  
  x<-str_split(CALMAF10_CALMImmature$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Nulo, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMImmatureHOXA$Gene)){
  
  x<-str_split(CALMAF10_CALMImmatureHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Salía nuelo, bien...



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMMLL$Gene)){
  
  x<-str_split(CALMAF10_CALMMLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMMLL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_CALMAF10_CALMMLL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMMLL_Targeting2[order(comparison_GSE62156_CALMAF10_CALMMLL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMMLL_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMTAL$Gene)){
  
  x<-str_split(CALMAF10_CALMTAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTAL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_CALMAF10_CALMTAL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTAL_Targeting2[order(comparison_GSE62156_CALMAF10_CALMTAL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTAL_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMTLX1$Gene)){
  
  x<-str_split(CALMAF10_CALMTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_CALMAF10_CALMTLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTLX1_Targeting2[order(comparison_GSE62156_CALMAF10_CALMTLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTLX1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMTLX3$Gene)){
  
  x<-str_split(CALMAF10_CALMTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_CALMAF10_CALMTLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTLX3_Targeting2[order(comparison_GSE62156_CALMAF10_CALMTLX3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTLX3_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_CALM$Gene)){
  
  x<-str_split(HOX_CALM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_CALM_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOX_CALM_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_CALM_Targeting2[order(comparison_GSE62156_HOX_CALM_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_CALM_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_ImHOXA$Gene)){
  
  x<-str_split(HOX_ImHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_ImHOXA_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOX_ImHOXA_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```


Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_ImHOXA_Targeting2[order(comparison_GSE62156_HOX_ImHOXA_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_ImHOXA_Targeting2.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_Immature$Gene)){
  
  x<-str_split(HOX_Immature$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Salía nulo, bien


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_ImmHOXA$Gene)){
  
  x<-str_split(HOX_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Nulo, bien

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_MLL$Gene)){
  
  x<-str_split(HOX_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Nulo, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_TAL$Gene)){
  
  x<-str_split(HOX_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Nulo, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_TLX$Gene)){
  
  x<-str_split(HOX_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Nulo, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_TLX1$Gene)){
  
  x<-str_split(HOX_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Salía bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_TLX3$Gene)){
  
  x<-str_split(HOX_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Salía bien, nulo...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_HOX$Gene)){
  
  x<-str_split(HOXAUnk_HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Bien, nulos...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_Imm$Gene)){
  
  x<-str_split(HOXAUnk_Imm$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```


Nulos, bien...


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_ImmHOXA$Gene)){
  
  x<-str_split(HOXAUnk_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

Nulos bien...



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_MLL$Gene)){
  
  x<-str_split(HOXAUnk_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_MLL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOXAUnk_MLL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_MLL_Targeting2[order(comparison_GSE62156_HOXAUnk_MLL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_MLL_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_TLX1$Gene)){
  
  x<-str_split(HOXAUnk_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_TLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOXAUnk_TLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_TLX1_Targeting2[order(comparison_GSE62156_HOXAUnk_TLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_TLX1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_TLX3$Gene)){
  
  x<-str_split(HOXAUnk_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```


NULO BIEN



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXs_MLL$Gene)){
  
  x<-str_split(HOXs_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_MLL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOXs_MLL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_MLL_Targeting2[order(comparison_GSE62156_HOXs_MLL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_MLL_Targeting2.xlsx", row.names = FALSE)
```

Salía bien, nulo...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXs_TAL$Gene)){
  
  x<-str_split(HOXs_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_TAL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOXs_TAL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Nulo, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_TAL_Targeting2[order(comparison_GSE62156_HOXs_TAL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TAL_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXs_TLX1$Gene)){
  
  x<-str_split(HOXs_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_TLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOXs_TLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_TLX1_Targeting2[order(comparison_GSE62156_HOXs_TLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TLX1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXs_TLX3$Gene)){
  
  x<-str_split(HOXs_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_TLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOXs_TLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_TLX3_Targeting2[order(comparison_GSE62156_HOXs_TLX3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TLX3_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_CALM$Gene)){
  
  x<-str_split(Imm_CALM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_CALM_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Imm_CALM_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_CALM_Targeting2[order(comparison_GSE62156_Imm_CALM_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_CALM_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_HOX$Gene)){
  
  x<-str_split(Imm_HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_HOX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Imm_HOX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Salía nulo al final, bien...


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_HOX_Targeting2[order(comparison_GSE62156_Imm_HOX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_HOX_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_HOXAUnk$Gene)){
  
  x<-str_split(Imm_HOXAUnk$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_HOXAUnk_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Imm_HOXAUnk_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Salía nulo al final, bien...


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_HOXAUnk_Targeting2[order(comparison_GSE62156_Imm_HOXAUnk_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_HOXAUnk_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_MLL$Gene)){
  
  x<-str_split(Imm_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_MLL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Imm_MLL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_MLL_Targeting2[order(comparison_GSE62156_Imm_MLL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_MLL_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_TAL$Gene)){
  
  x<-str_split(Imm_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TAL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Imm_TAL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TAL_Targeting2[order(comparison_GSE62156_Imm_TAL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TAL_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_TLX$Gene)){
  
  x<-str_split(Imm_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Imm_TLX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX_Targeting2[order(comparison_GSE62156_Imm_TLX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_TLX1$Gene)){
  
  x<-str_split(Imm_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Imm_TLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX1_Targeting2[order(comparison_GSE62156_Imm_TLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_TLX3$Gene)){
  
  x<-str_split(Imm_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Imm_TLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX3_Targeting2[order(comparison_GSE62156_Imm_TLX3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX3_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_ImmHOXA$Gene)){
  
  x<-str_split(Immature_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

NULO BIEN

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_MLL$Gene)){
  
  x<-str_split(Immature_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_MLL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Immature_MLL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_MLL_Targeting2[order(comparison_GSE62156_Immature_MLL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_MLL_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_TAL$Gene)){
  
  x<-str_split(Immature_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TAL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Immature_TAL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TAL_Targeting2[order(comparison_GSE62156_Immature_TAL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TAL_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_TLX1$Gene)){
  
  x<-str_split(Immature_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Immature_TLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TLX1_Targeting2[order(comparison_GSE62156_Immature_TLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TLX1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_TLX3$Gene)){
  
  x<-str_split(Immature_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_Immature_TLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TLX3_Targeting2[order(comparison_GSE62156_Immature_TLX3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TLX3_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(ImmHOXA_MLL$Gene)){
  
  x<-str_split(ImmHOXA_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_MLL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_ImmHOXA_MLL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_MLL_Targeting2[order(comparison_GSE62156_ImmHOXA_MLL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_MLL_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(ImmHOXA_TAL$Gene)){
  
  x<-str_split(ImmHOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TAL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_ImmHOXA_TAL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TAL_Targeting2[order(comparison_GSE62156_ImmHOXA_TAL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TAL_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(ImmHOXA_TLX1$Gene)){
  
  x<-str_split(ImmHOXA_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_ImmHOXA_TLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TLX1_Targeting2[order(comparison_GSE62156_ImmHOXA_TLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TLX1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(ImmHOXA_TLX3$Gene)){
  
  x<-str_split(ImmHOXA_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_ImmHOXA_TLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TLX3_Targeting2[order(comparison_GSE62156_ImmHOXA_TLX3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TLX3_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MLL_TAL$Gene)){
  
  x<-str_split(MLL_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```


NULO BIEN

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MLL_TLX1$Gene)){
  
  x<-str_split(MLL_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_MLL_TLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TLX1_Targeting2[order(comparison_GSE62156_MLL_TLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TLX1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MLL_TLX3$Gene)){
  
  x<-str_split(MLL_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_MLL_TLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TLX3_Targeting2[order(comparison_GSE62156_MLL_TLX3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TLX3_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLX$Gene)){
  
  x<-str_split(TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```


NULO BIEN


## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMHOX$Gene)){
  
  x<-str_split(CALMAF10_CALMHOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMHOX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_CALMAF10_CALMHOX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMHOX_Targeting5[order(comparison_GSE62156_CALMAF10_CALMHOX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMHOX_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMHOXAUnk$Gene)){
  
  x<-str_split(CALMAF10_CALMHOXAUnk$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMHOXAUnk_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_CALMAF10_CALMHOXAUnk_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMHOXAUnk_Targeting5[order(comparison_GSE62156_CALMAF10_CALMHOXAUnk_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMHOXAUnk_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMImmature$Gene)){
  
  x<-str_split(CALMAF10_CALMImmature$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMImmature_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_CALMAF10_CALMImmature_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMImmature_Targeting5[order(comparison_GSE62156_CALMAF10_CALMImmature_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMImmature_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMImmatureHOXA$Gene)){
  
  x<-str_split(CALMAF10_CALMImmatureHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Targeting5[order(comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMMLL$Gene)){
  
  x<-str_split(CALMAF10_CALMMLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMMLL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_CALMAF10_CALMMLL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMMLL_Targeting5[order(comparison_GSE62156_CALMAF10_CALMMLL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMMLL_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMTAL$Gene)){
  
  x<-str_split(CALMAF10_CALMTAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_CALMAF10_CALMTAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTAL_Targeting5[order(comparison_GSE62156_CALMAF10_CALMTAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTAL_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=FALSE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMTLX1$Gene)){
  
  x<-str_split(CALMAF10_CALMTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_CALMAF10_CALMTLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTLX1_Targeting5[order(comparison_GSE62156_CALMAF10_CALMTLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTLX1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CALMAF10_CALMTLX3$Gene)){
  
  x<-str_split(CALMAF10_CALMTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_CALMAF10_CALMTLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTLX3_Targeting5[order(comparison_GSE62156_CALMAF10_CALMTLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTLX3_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_CALM$Gene)){
  
  x<-str_split(HOX_CALM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_CALM_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOX_CALM_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_CALM_Targeting5[order(comparison_GSE62156_HOX_CALM_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_CALM_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_ImHOXA$Gene)){
  
  x<-str_split(HOX_ImHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_ImHOXA_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOX_ImHOXA_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_ImHOXA_Targeting5[order(comparison_GSE62156_HOX_ImHOXA_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_ImHOXA_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_Immature$Gene)){
  
  x<-str_split(HOX_Immature$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_Immature_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOX_Immature_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_Immature_Targeting5[order(comparison_GSE62156_HOX_Immature_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_Immature_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_ImmHOXA$Gene)){
  
  x<-str_split(HOX_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_ImmHOXA_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOX_ImmHOXA_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_ImmHOXA_Targeting5[order(comparison_GSE62156_HOX_ImmHOXA_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_ImmHOXA_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_MLL$Gene)){
  
  x<-str_split(HOX_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_MLL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOX_MLL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_MLL_Targeting5[order(comparison_GSE62156_HOX_MLL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_MLL_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_TAL$Gene)){
  
  x<-str_split(HOX_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOX_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_TAL_Targeting5[order(comparison_GSE62156_HOX_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_TAL_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_TLX$Gene)){
  
  x<-str_split(HOX_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOX_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_TLX_Targeting5[order(comparison_GSE62156_HOX_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_TLX_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_TLX1$Gene)){
  
  x<-str_split(HOX_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_TLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOX_TLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_TLX1_Targeting5[order(comparison_GSE62156_HOX_TLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_TLX1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX_TLX3$Gene)){
  
  x<-str_split(HOX_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_TLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOX_TLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_TLX3_Targeting5[order(comparison_GSE62156_HOX_TLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_TLX3_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_HOX$Gene)){
  
  x<-str_split(HOXAUnk_HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_HOX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXAUnk_HOX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_HOX_Targeting5[order(comparison_GSE62156_HOXAUnk_HOX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_HOX_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_Imm$Gene)){
  
  x<-str_split(HOXAUnk_Imm$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_Imm_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXAUnk_Imm_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_Imm_Targeting5[order(comparison_GSE62156_HOXAUnk_Imm_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_Imm_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_ImmHOXA$Gene)){
  
  x<-str_split(HOXAUnk_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_ImmHOXA_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXAUnk_ImmHOXA_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_ImmHOXA_Targeting5[order(comparison_GSE62156_HOXAUnk_ImmHOXA_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_ImmHOXA_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_MLL$Gene)){
  
  x<-str_split(HOXAUnk_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_MLL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXAUnk_MLL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_MLL_Targeting5[order(comparison_GSE62156_HOXAUnk_MLL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_MLL_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_TLX1$Gene)){
  
  x<-str_split(HOXAUnk_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_TLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXAUnk_TLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_TLX1_Targeting5[order(comparison_GSE62156_HOXAUnk_TLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_TLX1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXAUnk_TLX3$Gene)){
  
  x<-str_split(HOXAUnk_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_TLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXAUnk_TLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_TLX3_Targeting5[order(comparison_GSE62156_HOXAUnk_TLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_TLX3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXs_MLL$Gene)){
  
  x<-str_split(HOXs_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_MLL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXs_MLL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_MLL_Targeting5[order(comparison_GSE62156_HOXs_MLL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_MLL_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=FALSE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXs_TAL$Gene)){
  
  x<-str_split(HOXs_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXs_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_TAL_Targeting5[order(comparison_GSE62156_HOXs_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TAL_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=FALSE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXs_TLX1$Gene)){
  
  x<-str_split(HOXs_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_TLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXs_TLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_TLX1_Targeting5[order(comparison_GSE62156_HOXs_TLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TLX1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXs_TLX3$Gene)){
  
  x<-str_split(HOXs_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_TLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_HOXs_TLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_TLX3_Targeting5[order(comparison_GSE62156_HOXs_TLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TLX3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_CALM$Gene)){
  
  x<-str_split(Imm_CALM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_Imm_CALM_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_CALMAF10_Imm_CALM_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_Imm_CALM_Targeting5[order(comparison_GSE62156_CALMAF10_Imm_CALM_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_Imm_CALM_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_HOX$Gene)){
  
  x<-str_split(Imm_HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_HOX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Imm_HOX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_HOX_Targeting5[order(comparison_GSE62156_Imm_HOX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_HOX_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_HOXAUnk$Gene)){
  
  x<-str_split(Imm_HOXAUnk$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_HOXAUnk_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Imm_HOXAUnk_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_HOXAUnk_Targeting5[order(comparison_GSE62156_Imm_HOXAUnk_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_HOXAUnk_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_MLL$Gene)){
  
  x<-str_split(Imm_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_MLL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Imm_MLL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_MLL_Targeting5[order(comparison_GSE62156_Imm_MLL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_MLL_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_TAL$Gene)){
  
  x<-str_split(Imm_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Imm_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TAL_Targeting5[order(comparison_GSE62156_Imm_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TAL_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_TLX$Gene)){
  
  x<-str_split(Imm_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Imm_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX_Targeting5[order(comparison_GSE62156_Imm_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_TLX1$Gene)){
  
  x<-str_split(Imm_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Imm_TLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX1_Targeting5[order(comparison_GSE62156_Imm_TLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Imm_TLX3$Gene)){
  
  x<-str_split(Imm_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Imm_TLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX3_Targeting5[order(comparison_GSE62156_Imm_TLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_ImmHOXA$Gene)){
  
  x<-str_split(Immature_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_ImmHOXA_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Immature_ImmHOXA_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_ImmHOXA_Targeting5[order(comparison_GSE62156_Immature_ImmHOXA_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_ImmHOXA_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_MLL$Gene)){
  
  x<-str_split(Immature_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_MLL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Immature_MLL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_MLL_Targeting5[order(comparison_GSE62156_Immature_MLL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_MLL_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_TAL$Gene)){
  
  x<-str_split(Immature_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Immature_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TAL_Targeting5[order(comparison_GSE62156_Immature_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TAL_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_TLX1$Gene)){
  
  x<-str_split(Immature_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Immature_TLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TLX1_Targeting5[order(comparison_GSE62156_Immature_TLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TLX1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Immature_TLX3$Gene)){
  
  x<-str_split(Immature_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_Immature_TLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TLX3_Targeting5[order(comparison_GSE62156_Immature_TLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TLX3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(ImmHOXA_MLL$Gene)){
  
  x<-str_split(ImmHOXA_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_MLL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_ImmHOXA_MLL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_MLL_Targeting5[order(comparison_GSE62156_ImmHOXA_MLL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_MLL_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(ImmHOXA_TAL$Gene)){
  
  x<-str_split(ImmHOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_ImmHOXA_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TAL_Targeting5[order(comparison_GSE62156_ImmHOXA_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TAL_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(ImmHOXA_TLX1$Gene)){
  
  x<-str_split(ImmHOXA_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_ImmHOXA_TLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TLX1_Targeting5[order(comparison_GSE62156_ImmHOXA_TLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TLX1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(ImmHOXA_TLX3$Gene)){
  
  x<-str_split(ImmHOXA_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_ImmHOXA_TLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TLX3_Targeting5[order(comparison_GSE62156_ImmHOXA_TLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TLX3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MLL_TAL$Gene)){
  
  x<-str_split(MLL_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_MLL_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TAL_Targeting5[order(comparison_GSE62156_MLL_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TAL_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MLL_TLX1$Gene)){
  
  x<-str_split(MLL_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_MLL_TLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TLX1_Targeting5[order(comparison_GSE62156_MLL_TLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TLX1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MLL_TLX3$Gene)){
  
  x<-str_split(MLL_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_MLL_TLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TLX3_Targeting5[order(comparison_GSE62156_MLL_TLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TLX3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLX$Gene)){
  
  x<-str_split(TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE62156_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_TLX_Targeting5[order(comparison_GSE62156_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_TLX_Targeting5.xlsx", row.names = FALSE)
```




## Large_scale

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CALMAF10_CALMHOX$Gene)){
  
  x<-str_split(CALMAF10_CALMHOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMHOX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_CALMAF10_CALMHOX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMHOX_Large[order(comparison_GSE62156_CALMAF10_CALMHOX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMHOX_Large.xlsx", row.names = FALSE)
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CALMAF10_CALMHOXAUnk$Gene)){
  
  x<-str_split(CALMAF10_CALMHOXAUnk$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMHOXAUnk_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_CALMAF10_CALMHOXAUnk_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMHOXAUnk_Large[order(comparison_GSE62156_CALMAF10_CALMHOXAUnk_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMHOXAUnk_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CALMAF10_CALMImmature$Gene)){
  
  x<-str_split(CALMAF10_CALMImmature$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMImmature_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_CALMAF10_CALMImmature_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMImmature_Large[order(comparison_GSE62156_CALMAF10_CALMImmature_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMImmature_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CALMAF10_CALMImmatureHOXA$Gene)){
  
  x<-str_split(CALMAF10_CALMImmatureHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Salía nulo al final, bien...


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Large[order(comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMImmatureHOXA_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CALMAF10_CALMMLL$Gene)){
  
  x<-str_split(CALMAF10_CALMMLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMMLL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_CALMAF10_CALMMLL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMMLL_Large[order(comparison_GSE62156_CALMAF10_CALMMLL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMMLL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CALMAF10_CALMTAL$Gene)){
  
  x<-str_split(CALMAF10_CALMTAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_CALMAF10_CALMTAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMHOXAUnk_Large[order(comparison_GSE62156_CALMAF10_CALMHOXAUnk_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTAL_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CALMAF10_CALMTLX1$Gene)){
  
  x<-str_split(CALMAF10_CALMTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_CALMAF10_CALMTLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTLX1_Large[order(comparison_GSE62156_CALMAF10_CALMTLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTLX1_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CALMAF10_CALMTLX3$Gene)){
  
  x<-str_split(CALMAF10_CALMTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_CALMAF10_CALMTLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_CALMAF10_CALMTLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_CALMAF10_CALMTLX3_Large[order(comparison_GSE62156_CALMAF10_CALMTLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_CALMAF10_CALMTLX3_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX_CALM$Gene)){
  
  x<-str_split(HOX_CALM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_CALM_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOX_CALM_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Salía nulo al final, bien...


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_CALM_Large[order(comparison_GSE62156_HOX_CALM_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_CALM_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX_ImHOXA$Gene)){
  
  x<-str_split(HOX_ImHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_ImHOXA_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOX_ImHOXA_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_ImHOXA_Large[order(comparison_GSE62156_HOX_ImHOXA_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_ImHOXA_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX_Immature$Gene)){
  
  x<-str_split(HOX_Immature$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_Immature_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOX_Immature_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Salía nulo al final, bien...


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_Immature_Large[order(comparison_GSE62156_HOX_Immature_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_Immature_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX_ImmHOXA$Gene)){
  
  x<-str_split(HOX_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_ImmHOXA_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOX_ImmHOXA_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_ImmHOXA_Large[order(comparison_GSE62156_HOX_ImmHOXA_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_ImmHOXA_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX_MLL$Gene)){
  
  x<-str_split(HOX_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_MLL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOX_MLL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_MLL_Large[order(comparison_GSE62156_HOX_MLL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_MLL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX_TAL$Gene)){
  
  x<-str_split(HOX_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOX_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_TAL_Large[order(comparison_GSE62156_HOX_TAL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_TAL_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX_TLX$Gene)){
  
  x<-str_split(HOX_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_TLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOX_TLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_TLX_Large[order(comparison_GSE62156_HOX_TLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_TLX_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX_TLX1$Gene)){
  
  x<-str_split(HOX_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_TLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOX_TLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_TLX1_Large[order(comparison_GSE62156_HOX_TLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_TLX1_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX_TLX3$Gene)){
  
  x<-str_split(HOX_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOX_TLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOX_TLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Salía nulo al final, bien...


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOX_TLX3_Large[order(comparison_GSE62156_HOX_TLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOX_TLX3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXAUnk_HOX$Gene)){
  
  x<-str_split(HOXAUnk_HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_HOX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXAUnk_HOX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_HOX_Large[order(comparison_GSE62156_HOXAUnk_HOX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_HOX_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXAUnk_Imm$Gene)){
  
  x<-str_split(HOXAUnk_Imm$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_Imm_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXAUnk_Imm_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_Imm_Large[order(comparison_GSE62156_HOXAUnk_Imm_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_Imm_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXAUnk_ImmHOXA$Gene)){
  
  x<-str_split(HOXAUnk_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_ImmHOXA_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXAUnk_ImmHOXA_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_ImmHOXA_Large[order(comparison_GSE62156_HOXAUnk_ImmHOXA_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_ImmHOXA_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXAUnk_MLL$Gene)){
  
  x<-str_split(HOXAUnk_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_MLL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXAUnk_MLL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_MLL_Large[order(comparison_GSE62156_HOXAUnk_MLL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_MLL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXAUnk_TLX1$Gene)){
  
  x<-str_split(HOXAUnk_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_TLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXAUnk_TLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_TLX1_Large[order(comparison_GSE62156_HOXAUnk_TLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_TLX1_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXAUnk_TLX3$Gene)){
  
  x<-str_split(HOXAUnk_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXAUnk_TLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXAUnk_TLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXAUnk_TLX3_Large[order(comparison_GSE62156_HOXAUnk_TLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXAUnk_TLX3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXs_MLL$Gene)){
  
  x<-str_split(HOXs_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_MLL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXs_MLL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_MLL_Large[order(comparison_GSE62156_HOXs_MLL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_MLL_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXs_TAL$Gene)){
  
  x<-str_split(HOXs_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXs_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_TAL_Large[order(comparison_GSE62156_HOXs_TAL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TAL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXs_TLX1$Gene)){
  
  x<-str_split(HOXs_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_TLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXs_TLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_TLX1_Large[order(comparison_GSE62156_HOXs_TLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TLX1_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXs_TLX3$Gene)){
  
  x<-str_split(HOXs_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXs_TLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_HOXs_TLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXs_TLX3_Large[order(comparison_GSE62156_HOXs_TLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_HOXs_TLX3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Imm_CALM$Gene)){
  
  x<-str_split(Imm_CALM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_CALM_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Imm_CALM_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_CALM_Large[order(comparison_GSE62156_Imm_CALM_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_CALM_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Imm_HOX$Gene)){
  
  x<-str_split(Imm_HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_HOX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Imm_HOX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_HOX_Large[order(comparison_GSE62156_Imm_HOX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_HOX_Large.xlsx", row.names = FALSE)
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Imm_HOXAUnk$Gene)){
  
  x<-str_split(Imm_HOXAUnk$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_HOXAUnk_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Imm_HOXAUnk_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_HOXAUnk_Large[order(comparison_GSE62156_Imm_HOXAUnk_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_HOXAUnk_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Imm_MLL$Gene)){
  
  x<-str_split(Imm_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_MLL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Imm_MLL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_MLL_Large[order(comparison_GSE62156_Imm_MLL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_MLL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Imm_TAL$Gene)){
  
  x<-str_split(Imm_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Imm_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TAL_Large[order(comparison_GSE62156_Imm_TAL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TAL_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Imm_TLX$Gene)){
  
  x<-str_split(Imm_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Imm_TLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX_Large[order(comparison_GSE62156_Imm_TLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Imm_TLX1$Gene)){
  
  x<-str_split(Imm_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Imm_TLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX1_Large[order(comparison_GSE62156_Imm_TLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX1_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Imm_TLX3$Gene)){
  
  x<-str_split(Imm_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Imm_TLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Imm_TLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Imm_TLX3_Large[order(comparison_GSE62156_Imm_TLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Imm_TLX3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Immature_ImmHOXA$Gene)){
  
  x<-str_split(Immature_ImmHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_ImmHOXA_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Immature_ImmHOXA_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Salía nulo al final, bien...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_ImmHOXA_Large[order(comparison_GSE62156_Immature_ImmHOXA_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_ImmHOXA_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Immature_MLL$Gene)){
  
  x<-str_split(Immature_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_MLL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Immature_MLL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_MLL_Large[order(comparison_GSE62156_Immature_MLL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_MLL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Immature_TAL$Gene)){
  
  x<-str_split(Immature_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Immature_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TAL_Large[order(comparison_GSE62156_Immature_TAL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TAL_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Immature_TLX1$Gene)){
  
  x<-str_split(Immature_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Immature_TLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TLX1_Large[order(comparison_GSE62156_Immature_TLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TLX1_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Immature_TLX3$Gene)){
  
  x<-str_split(Immature_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_Immature_TLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_Immature_TLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_Immature_TLX3_Large[order(comparison_GSE62156_Immature_TLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_Immature_TLX3_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(ImmHOXA_MLL$Gene)){
  
  x<-str_split(ImmHOXA_MLL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_MLL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_ImmHOXA_MLL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_MLL_Large[order(comparison_GSE62156_ImmHOXA_MLL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_MLL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(ImmHOXA_TAL$Gene)){
  
  x<-str_split(ImmHOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_ImmHOXA_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TAL_Large[order(comparison_GSE62156_ImmHOXA_TAL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TAL_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(ImmHOXA_TLX1$Gene)){
  
  x<-str_split(ImmHOXA_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_ImmHOXA_TLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TLX1_Large[order(comparison_GSE62156_ImmHOXA_TLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TLX1_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(ImmHOXA_TLX3$Gene)){
  
  x<-str_split(ImmHOXA_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_ImmHOXA_TLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_ImmHOXA_TLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_ImmHOXA_TLX3_Large[order(comparison_GSE62156_ImmHOXA_TLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_ImmHOXA_TLX3_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(MLL_TAL$Gene)){
  
  x<-str_split(MLL_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_MLL_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TAL_Large[order(comparison_GSE62156_MLL_TAL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TAL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(MLL_TLX1$Gene)){
  
  x<-str_split(MLL_TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_MLL_TLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TLX1_Large[order(comparison_GSE62156_MLL_TLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TLX1_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(MLL_TLX3$Gene)){
  
  x<-str_split(MLL_TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_MLL_TLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_MLL_TLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_MLL_TLX3_Large[order(comparison_GSE62156_MLL_TLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_MLL_TLX3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TLX$Gene)){
  
  x<-str_split(TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_TLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE62156_TLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_TLX_Large[order(comparison_GSE62156_TLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/comparison_GSE62156_TLX_Large.xlsx", row.names = FALSE)
```

Salía nulo al final, bien...



# GSE110633

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(readxl)


HOXA_IMM<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/Resultado_GSE110633_HOXA_IMM.xlsx')


HOXA_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/Resultado_GSE110633_HOXA_TAL.xlsx')
HOXA_TLX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/Resultado_GSE110633_HOXA_TLX.xlsx')

IMM_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/Resultado_GSE110633_IMM_TAL.xlsx')
IMM_TLX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/Resultado_GSE110633_IMM_TLX.xlsx')

TAL_TLX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/Resultado_GSE110633_TAL_TLX.xlsx')

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
ls()

```

## Transcriptome


Haremos una funcion para tener los nombres de los genes comunes a nuestra base de datos y a la de Transcriptome1. 

Los Gene de la base de Resultados son correspondientes también al Gene de Transcriptome.



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXA_IMM$Gene)){
  
  x<-str_split(HOXA_IMM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_IMM_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_HOXA_IMM_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_IMM_Transcriptome1[order(comparison_GSE110633_HOXA_IMM_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_IMM_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXA_TAL$Gene)){
  
  x<-str_split(HOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_HOXA_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TAL_Transcriptome1[order(comparison_GSE110633_HOXA_TAL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_TAL_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXA_TLX$Gene)){
  
  x<-str_split(HOXA_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TLX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_HOXA_TLX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TLX_Transcriptome1[order(comparison_GSE110633_HOXA_TLX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_TLX_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(IMM_TAL$Gene)){
  
  x<-str_split(IMM_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_IMM_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_IMM_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_IMM_TAL_Transcriptome1[order(comparison_GSE110633_IMM_TAL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_IMM_TAL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(IMM_TLX$Gene)){
  
  x<-str_split(IMM_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```



Nulo


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(TAL_TLX$Gene)){
  
  x<-str_split(TAL_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_TAL_TLX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_TAL_TLX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_TAL_TLX_Transcriptome1[order(comparison_GSE110633_TAL_TLX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_TAL_TLX_Transcriptome1.xlsx", row.names = FALSE)
```



## Targeting2

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_IMM$Gene)){
  
  x<-str_split(HOXA_IMM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```


Nulo


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_TAL$Gene)){
  
  x<-str_split(HOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TAL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE110633_HOXA_TAL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TAL_Targeting2[order(comparison_GSE110633_HOXA_TAL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_TAL_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_TLX$Gene)){
  
  x<-str_split(HOXA_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TLX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE110633_HOXA_TLX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TLX_Targeting2[order(comparison_GSE110633_HOXA_TLX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_TLX_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(IMM_TAL$Gene)){
  
  x<-str_split(IMM_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```


Nulo



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(IMM_TLX$Gene)){
  
  x<-str_split(IMM_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_IMM_TLX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_IMM_TLX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_IMM_TLX_Targeting2[order(comparison_GSE62156_IMM_TLX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_IMM_TLX_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL_TLX$Gene)){
  
  x<-str_split(TAL_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_TAL_TLX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_TAL_TLX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_TAL_TLX_Targeting2[order(comparison_GSE62156_TAL_TLX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_TAL_TLX_Targeting2.xlsx", row.names = FALSE)
```






## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_IMM$Gene)){
  
  x<-str_split(HOXA_IMM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_IMM_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_HOXA_IMM_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_IMM_Targeting5[order(comparison_GSE110633_HOXA_IMM_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_IMM_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_TAL$Gene)){
  
  x<-str_split(HOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_HOXA_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TAL_Targeting5[order(comparison_GSE110633_HOXA_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_TAL_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_TLX$Gene)){
  
  x<-str_split(HOXA_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_HOXA_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TLX_Targeting5[order(comparison_GSE110633_HOXA_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_TLX_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(IMM_TAL$Gene)){
  
  x<-str_split(IMM_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_IMM_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_IMM_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_IMM_TAL_Targeting5[order(comparison_GSE110633_IMM_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_IMM_TAL_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(IMM_TLX$Gene)){
  
  x<-str_split(IMM_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_IMM_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_IMM_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_IMM_TLX_Targeting5[order(comparison_GSE110633_IMM_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_IMM_TLX_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL_TLX$Gene)){
  
  x<-str_split(TAL_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_TAL_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_TAL_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_TAL_TLX_Targeting5[order(comparison_GSE110633_TAL_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_TAL_TLX_Targeting5.xlsx", row.names = FALSE)
```




## Large_scale



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXA_IMM$Gene)){
  
  x<-str_split(HOXA_IMM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_IMM_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110633_HOXA_IMM_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_IMM_Large[order(comparison_GSE110633_HOXA_IMM_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_IMM_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXA_TAL$Gene)){
  
  x<-str_split(HOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110633_HOXA_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TAL_Large[order(comparison_GSE110633_HOXA_TAL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_TAL_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXA_TLX$Gene)){
  
  x<-str_split(HOXA_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110633_HOXA_TLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TLX_Large[order(comparison_GSE110633_HOXA_TLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_TLX_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(IMM_TAL$Gene)){
  
  x<-str_split(IMM_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_IMM_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110633_IMM_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_IMM_TAL_Large[order(comparison_GSE110633_IMM_TAL_Large$HGNC),]

```

Solo con 1 línea, no eliminamos



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_IMM_TAL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(IMM_TLX$Gene)){
  
  x<-str_split(IMM_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

Nulo



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TAL_TLX$Gene)){
  
  x<-str_split(TAL_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_TAL_TLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110633_TAL_TLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_TAL_TLX_Large[order(comparison_GSE110633_TAL_TLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_TAL_TLX_Large.xlsx", row.names = FALSE)
```




# GSE110636

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(readxl)


HOXA_IMM<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/Resultado_GSE110636_HOXA_IMM.xlsx')


HOXA_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/Resultado_GSE110636_HOXA_TAL.xlsx')
HOXA_TLX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/Resultado_GSE110636_HOXA_TLX.xlsx')

IMM_TAL<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/Resultado_GSE110636_IMM_TAL.xlsx')
IMM_TLX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/Resultado_GSE110636_IMM_TLX.xlsx')

TAL_TLX<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/Resultado_GSE110636_TAL_TLX.xlsx')

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
ls()

```

## Transcriptome


Haremos una funcion para tener los nombres de los genes comunes a nuestra base de datos y a la de Transcriptome1. 

Los Gene de la base de Resultados son correspondientes también al Gene de Transcriptome.



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXA_IMM$Gene)){
  
  x<-str_split(HOXA_IMM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_IMM_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_HOXA_IMM_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_IMM_Transcriptome1[order(comparison_GSE110633_HOXA_IMM_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_IMM_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXA_TAL$Gene)){
  
  x<-str_split(HOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_HOXA_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TAL_Transcriptome1[order(comparison_GSE110633_HOXA_TAL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_TAL_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXA_TLX$Gene)){
  
  x<-str_split(HOXA_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TLX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_HOXA_TLX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TLX_Transcriptome1[order(comparison_GSE110633_HOXA_TLX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/comparison_GSE110633_HOXA_TLX_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(IMM_TAL$Gene)){
  
  x<-str_split(IMM_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_IMM_TAL_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_IMM_TAL_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_IMM_TAL_Transcriptome1[order(comparison_GSE110633_IMM_TAL_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_IMM_TAL_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(IMM_TLX$Gene)){
  
  x<-str_split(IMM_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_IMM_TLX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_IMM_TLX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_IMM_TLX_Transcriptome1[order(comparison_GSE110633_IMM_TLX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_IMM_TLX_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(TAL_TLX$Gene)){
  
  x<-str_split(TAL_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_TAL_TLX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE110633_TAL_TLX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_TAL_TLX_Transcriptome1[order(comparison_GSE110633_TAL_TLX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_TAL_TLX_Transcriptome1.xlsx", row.names = FALSE)
```



## Targeting2

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_IMM$Gene)){
  
  x<-str_split(HOXA_IMM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```



Nulo


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_TAL$Gene)){
  
  x<-str_split(HOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXA_TAL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOXA_TAL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXA_TAL_Targeting2[order(comparison_GSE62156_HOXA_TAL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_TAL_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_TLX$Gene)){
  
  x<-str_split(HOXA_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_HOXA_TLX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_HOXA_TLX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_HOXA_TLX_Targeting2[order(comparison_GSE62156_HOXA_TLX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_TLX_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(IMM_TAL$Gene)){
  
  x<-str_split(IMM_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110636_IMM_TAL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE110636_IMM_TAL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110636_IMM_TAL_Targeting2[order(comparison_GSE110636_IMM_TAL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_IMM_TAL_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(IMM_TLX$Gene)){
  
  x<-str_split(IMM_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```


Nulo

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL_TLX$Gene)){
  
  x<-str_split(TAL_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE62156_TAL_TLX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE62156_TAL_TLX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE62156_TAL_TLX_Targeting2[order(comparison_GSE62156_TAL_TLX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_TAL_TLX_Targeting2.xlsx", row.names = FALSE)
```






## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_IMM$Gene)){
  
  x<-str_split(HOXA_IMM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_IMM_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_HOXA_IMM_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_IMM_Targeting5[order(comparison_GSE110633_HOXA_IMM_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_IMM_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_TAL$Gene)){
  
  x<-str_split(HOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_HOXA_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TAL_Targeting5[order(comparison_GSE110633_HOXA_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_TAL_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXA_TLX$Gene)){
  
  x<-str_split(HOXA_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_HOXA_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TLX_Targeting5[order(comparison_GSE110633_HOXA_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_TLX_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(IMM_TAL$Gene)){
  
  x<-str_split(IMM_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_IMM_TAL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_IMM_TAL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_IMM_TAL_Targeting5[order(comparison_GSE110633_IMM_TAL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_IMM_TAL_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(IMM_TLX$Gene)){
  
  x<-str_split(IMM_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_IMM_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_IMM_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_IMM_TLX_Targeting5[order(comparison_GSE110633_IMM_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_IMM_TLX_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL_TLX$Gene)){
  
  x<-str_split(TAL_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_TAL_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE110633_TAL_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_TAL_TLX_Targeting5[order(comparison_GSE110633_TAL_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_TAL_TLX_Targeting5.xlsx", row.names = FALSE)
```




## Large_scale



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXA_IMM$Gene)){
  
  x<-str_split(HOXA_IMM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_IMM_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110633_HOXA_IMM_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_IMM_Large[order(comparison_GSE110633_HOXA_IMM_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_IMM_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXA_TAL$Gene)){
  
  x<-str_split(HOXA_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110633_HOXA_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TAL_Large[order(comparison_GSE110633_HOXA_TAL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_TAL_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXA_TLX$Gene)){
  
  x<-str_split(HOXA_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_HOXA_TLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110633_HOXA_TLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_HOXA_TLX_Large[order(comparison_GSE110633_HOXA_TLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_HOXA_TLX_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(IMM_TAL$Gene)){
  
  x<-str_split(IMM_TAL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110636_IMM_TAL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110636_IMM_TAL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110636_IMM_TAL_Large[order(comparison_GSE110636_IMM_TAL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_IMM_TAL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(IMM_TLX$Gene)){
  
  x<-str_split(IMM_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110636_IMM_TLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110636_IMM_TLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110636_IMM_TLX_Large[order(comparison_GSE110636_IMM_TLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_IMM_TLX_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TAL_TLX$Gene)){
  
  x<-str_split(TAL_TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE110633_TAL_TLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE110633_TAL_TLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE110633_TAL_TLX_Large[order(comparison_GSE110633_TAL_TLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/comparison_GSE110636_TAL_TLX_Large.xlsx", row.names = FALSE)
```






