---
title: "Selección RBPs"
subtitle: ""
author: "Cristina Muntañola Valero"
date: "`r Sys.Date()`"
output:
  pdf_document:
    toc: true 
    toc_depth: 3
    df_print: paged
  html_document:
    toc: true
    toc_depth: 3
    df_print: paged
urlcolor: blue
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

Esta parte se corresponde con la anotación de las comparaciones de los GSE8879, GSE10609, GSE14818 y GSE26713.

# Carga datos artículos

## Transcriptome-wide discovery

Huang, Rongbing, et al. "Transcriptome-wide discovery of coding and noncoding RNA-binding proteins." Proceedings of the National Academy of Sciences 115.17 (2018): E3879-E3887.

Wang, E., Lu, S. X., Pastore, A., Chen, X., Imig, J., Lee, S. C. W., ... & Aifantis, I. (2019). Targeting an RNA-binding protein network in acute myeloid leukemia. Cancer cell, 35(3), 369-384.


Sebestyén, E., Singh, B., Miñana, B., Pagès, A., Mateo, F., Pujana, M. A., ... & Eyras, E. (2016). Large-scale analysis of genome and transcriptome alterations in multiple tumors unveils novel cancer-relevant splicing networks. Genome research, 26(6), 732-744.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
# setwd('C:/Users/laura/Downloads/')
setwd('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library("readxl")
# Transcriptome1 <- read_excel("C:/Users/laura/Downloads/Transcriptome_wide_discovery_1.xlsx")

Transcriptome1 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Transcriptome_wide_discovery_1.xlsx")

```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library("readxl")

# Targeting1 <- read_excel("C:/Users/laura/Downloads/Targeting_an_RNA_Binding_Protein_Network_1.xlsx")

Targeting1 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Targeting_an_RNA_Binding_Protein_Network_1.xlsx")
Targeting2 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Targeting2.xlsx")
Targeting5 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Targeting5.xlsx")
```

```{r, echo=TRUE, warning=FALSE, message=TRUE, include=TRUE}


# Large_scale <- read_excel("C:/Users/laura/Downloads/Large_scale_analysis.xlsx", sheet = 3, skip = 1)

Large_scale <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Large_scale_analysis.xlsx", sheet = 2, skip = 1)
dim(Large_scale)
```


Quitamos las filas y las columnas de cada dataset que no nos interesan.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Transcriptome1)<-Transcriptome1[1,]
Transcriptome1<-Transcriptome1[c(-1,-2,-3),]
Transcriptome1<-Transcriptome1[,-c(6:41, 43)]
colnames(Transcriptome1)<-c("Representative_protein_IDs", "Protein_IDs", "Majority_protein_IDs",
                            "Protein_name", "Gene", "Adjusted_P_values","Classification")
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Targeting1)<-Targeting1[1,]
Targeting1<-Targeting1[c(-1),]
Targeting1<-Targeting1[,-c(4, 5, 7)]

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

colnames(Targeting2) <- Targeting2[1,]
Targeting2<-Targeting2[-1,]
Targeting2<-Targeting2[,-c(3:13)]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Targeting5)<-Targeting5[1,]
Targeting5<-Targeting5[-1,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
Targeting5<-Targeting5[,-c(1:5, 8)]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Targeting5)<-c('id', 'gene', 'gene_description')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
table(Large_scale$tumor)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
Large_scale<-Large_scale[,-c(2,3)]

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Large_scale)<-c('HGNC_symbol', 'type')

```

# GSE28703


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(readxl)
GSE28703<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE28703/Resultado_GSE28703.xlsx')
```


## Transcriptome


Haremos una funcion para tener los nombres de los genes comunes a nuestra base de datos y a la de Transcriptome1. 

Los Gene de la base de Resultados son correspondientes también al Gene de Transcriptome.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(GSE28703$Gene)){
  
  x<-str_split(GSE28703$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE28703_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE28703_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE28703_Transcriptome1[order(comparison_GSE28703_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE28703/comparison_GSE28703_Transcriptome1.xlsx", row.names = FALSE)
```


## Targeting1

GSE8879_geneIds ENTREZID, y GSE8879_geneIds SYMBOL son Targeting1 entrezgene y Targeting1 hgnc_symbol, respectivamente. También con el topTab_GSE8879_Anot usado anteriormente, con las mismas columnas ENTREZID y SYMBOL.

Lo haremos con SYMBOL, por ejemplo. Primero cambiaremos c, y continuaremos...

También usaremos el P valor, como los autores.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(GSE28703$Gene)){
  
  x<-str_split(GSE28703$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE28703_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE28703_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE28703_Targeting1[order(comparison_GSE28703_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE28703/comparison_GSE28703_Targeting1.xlsx", row.names = FALSE)
```



## Targeting2

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(GSE28703$Gene)){
  
  x<-str_split(GSE28703$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE28703_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE28703_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE28703_Targeting2[order(comparison_GSE28703_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE28703/comparison_GSE28703_Targeting2.xlsx", row.names = FALSE)
```




## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(GSE28703$Gene)){
  
  x<-str_split(GSE28703$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE28703_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE28703_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE28703_Targeting5[order(comparison_GSE28703_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE28703/comparison_GSE28703_Targeting5.xlsx", row.names = FALSE)
```







## Large_scale

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(GSE28703$Gene)){
  
  x<-str_split(GSE28703$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE28703_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE28703_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE28703_Large[order(comparison_GSE28703_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE28703/comparison_GSE28703_Large.xlsx", row.names = FALSE)
```


# GSE32215

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(readxl)
GSE32215<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE32215/Resultado_GSE32215.xlsx')
```


## Transcriptome


Haremos una funcion para tener los nombres de los genes comunes a nuestra base de datos y a la de Transcriptome1. 

Los Gene de la base de Resultados son correspondientes también al Gene de Transcriptome.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(GSE32215$Gene)){
  
  x<-str_split(GSE32215$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE32215_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE32215_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE32215_Transcriptome1[order(comparison_GSE32215_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE32215/comparison_GSE32215_Transcriptome1.xlsx", row.names = FALSE)
```


## Targeting1

GSE8879_geneIds ENTREZID, y GSE8879_geneIds SYMBOL son Targeting1 entrezgene y Targeting1 hgnc_symbol, respectivamente. También con el topTab_GSE8879_Anot usado anteriormente, con las mismas columnas ENTREZID y SYMBOL.

Lo haremos con SYMBOL, por ejemplo. Primero cambiaremos c, y continuaremos...

También usaremos el P valor, como los autores.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(GSE32215$Gene)){
  
  x<-str_split(GSE32215$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE32215_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE32215_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE32215_Targeting1[order(comparison_GSE32215_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE32215/comparison_GSE32215_Targeting1.xlsx", row.names = FALSE)
```



## Targeting2

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(GSE32215$Gene)){
  
  x<-str_split(GSE32215$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE32215_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE32215_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE32215_Targeting2[order(comparison_GSE32215_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE32215/comparison_GSE32215_Targeting2.xlsx", row.names = FALSE)
```




## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(GSE32215$Gene)){
  
  x<-str_split(GSE32215$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE32215_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE32215_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE32215_Targeting5[order(comparison_GSE32215_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE32215/comparison_GSE32215_Targeting5.xlsx", row.names = FALSE)
```







## Large_scale

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(GSE32215$Gene)){
  
  x<-str_split(GSE32215$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE32215_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE32215_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE32215_Large[order(comparison_GSE32215_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE32215/comparison_GSE32215_Large.xlsx", row.names = FALSE)
```




# GSE33470


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(readxl)
CD3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3.xlsx')
CD34CD4<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD34CD4.xlsx')
CD34DPCD3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD34DPCD3.xlsx')
CD34masmasCD4<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD34masmasCD4.xlsx')
CD34masmasDPCD3mas<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD34masmasDPCD3mas.xlsx')
CD34masmasDPCD3menos<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD34masmasDPCD3menos.xlsx')

CD34masmenosCD4<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD34masmenosCD4.xlsx')
CD34masmenosDPCD3mas<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD34masmenosDPCD3mas.xlsx')
CD34masmenosDPCD3menos<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD34masmenosDPCD3menos.xlsx')

CD3CD34<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3CD34.xlsx')
CD3CD4<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3CD4.xlsx')
CD3DPCD3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3DPCD3.xlsx')

CD3masCD4CD34masmeno<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD4CD34masmeno.xlsx')
CD3masCD4CD4<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD4CD4.xlsx')
CD3masCD4DPCD3mas<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD4DPCD3mas.xlsx')

CD3masCD4DPCD3menos<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD4DPCD3menos.xlsx')
CD3masCD8CD34masmas<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD8CD34masmas.xlsx')
CD3masCD8CD34masmenos<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD8CD34masmenos.xlsx')

CD3masCD8CD4<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD8CD4.xlsx')
CD3masCD8DPCD3mas<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD8DPCD3mas.xlsx')

CD3masCD8DPCD3menos<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD8DPCD3menos.xlsx')
CD4CD34masmas<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD4CD34masmas.xlsx')


CD3masCD8CD34masmenos<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD8CD34masmenos.xlsx')
CD3masCD8CD4<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD8CD4.xlsx')

CD3masCD8DPCD3mas<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD8DPCD3mas.xlsx')
CD3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD3masCD8DPCD3menos.xlsx')

CD4CD34masmas<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD4CD34masmas.xlsx')
CD4DPCD3<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD4DPCD3.xlsx')



CD4DPCD3mas<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD4DPCD3mas.xlsx')
CD4DPCD3menos<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/Resultado_GSE33470_CD4DPCD3menos.xlsx')

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
ls()

```

## Transcriptome


Haremos una funcion para tener los nombres de los genes comunes a nuestra base de datos y a la de Transcriptome1. 

Los Gene de la base de Resultados son correspondientes también al Gene de Transcriptome.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3$Gene)){
  
  x<-str_split(CD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3_Transcriptome1[order(comparison_GSE33470_CD3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD34CD4$Gene)){
  
  x<-str_split(CD34CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34CD4_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD34CD4_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34CD4_Transcriptome1[order(comparison_GSE33470_CD34CD4_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34CD4_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD34DPCD3$Gene)){
  
  x<-str_split(CD34DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34DPCD3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD34DPCD3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34DPCD3_Transcriptome1[order(comparison_GSE33470_CD34DPCD3_Transcriptome1),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (!is.na(yoquese$Proteina[i]) &&
      !is.na(yoquese$Proteina[i+1]) &&
    yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34DPCD3_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD34masmasCD4$Gene)){
  
  x<-str_split(CD34masmasCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasCD4_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD34masmasCD4_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasCD4_Transcriptome1[order(comparison_GSE33470_CD34masmasCD4_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasCD4_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD34masmasDPCD3mas$Gene)){
  
  x<-str_split(CD34masmasDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3mas_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD34masmasDPCD3mas_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3mas_Transcriptome1[order(comparison_GSE33470_CD34masmasDPCD3mas_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3mas_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD34masmasDPCD3menos$Gene)){
  
  x<-str_split(CD34masmasDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3menos_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD34masmasDPCD3menos_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3menos_Transcriptome1[order(comparison_GSE33470_CD34masmasDPCD3menos_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3menos_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD34masmenosCD4$Gene)){
  
  x<-str_split(CD34masmenosCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosCD4_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD34masmenosCD4_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosCD4_Transcriptome1[order(comparison_GSE33470_CD34masmenosCD4_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosCD4_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD34masmenosDPCD3mas$Gene)){
  
  x<-str_split(CD34masmenosDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3mas_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD34masmenosDPCD3mas_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3mas_Transcriptome1[order(comparison_GSE33470_CD34masmenosDPCD3mas_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3mas_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD34masmenosDPCD3menos$Gene)){
  
  x<-str_split(CD34masmenosDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3menos_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD34masmenosDPCD3menos_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3menos_Transcriptome1[order(comparison_GSE33470_CD34masmenosDPCD3menos_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3menos_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3CD34$Gene)){
  
  x<-str_split(CD3CD34$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD34_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3CD34_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD34_Transcriptome1[order(comparison_GSE33470_CD3CD34_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD34_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3CD4$Gene)){
  
  x<-str_split(CD3CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD4_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3CD4_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD4_Transcriptome1[order(comparison_GSE33470_CD3CD4_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD4_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3DPCD3$Gene)){
  
  x<-str_split(CD3DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3DPCD3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3DPCD3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3DPCD3_Transcriptome1[order(comparison_GSE33470_CD3DPCD3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3DPCD3_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3masCD4CD34masmeno$Gene)){
  
  x<-str_split(CD3masCD4CD34masmeno$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD34masmeno_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3masCD4CD34masmeno_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD34masmeno_Transcriptome1[order(comparison_GSE33470_CD3masCD4CD34masmeno_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD34masmeno_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3masCD4CD4$Gene)){
  
  x<-str_split(CD3masCD4CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD4_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3masCD4CD4_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD4_Transcriptome1[order(comparison_GSE33470_CD3masCD4CD4_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD4_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3masCD4DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4DPCD3mas_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3masCD4DPCD3mas_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4DPCD3mas_Transcriptome1[order(comparison_GSE33470_CD3masCD4DPCD3mas_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4DPCD3mas_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3masCD4DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4DPCD3menos_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3masCD4DPCD3menos_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4DPCD3menos_Transcriptome1[order(comparison_GSE33470_CD3masCD4DPCD3menos_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4DPCD3menos_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3masCD8CD34masmas$Gene)){
  
  x<-str_split(CD3masCD8CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmas_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3masCD8CD34masmas_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmas_Transcriptome1[order(comparison_GSE33470_CD3masCD8CD34masmas_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmas_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3masCD8CD34masmenos$Gene)){
  
  x<-str_split(CD3masCD8CD34masmenos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmenos_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3masCD8CD34masmenos_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmenos_Transcriptome1[order(comparison_GSE33470_CD3masCD8CD34masmenos_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmenos_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3masCD8CD4$Gene)){
  
  x<-str_split(CD3masCD8CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD4_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3masCD8CD4_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD4_Transcriptome1[order(comparison_GSE33470_CD3masCD8CD4_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD4_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3masCD8DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD8DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8DPCD3mas_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3masCD8DPCD3mas_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8DPCD3mas_Transcriptome1[order(comparison_GSE33470_CD3masCD8DPCD3mas_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8DPCD3mas_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD3masCD8DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD8DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8DPCD3menos_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD3masCD8DPCD3menos_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8DPCD3menos_Transcriptome1[order(comparison_GSE33470_CD3masCD8DPCD3menos_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8DPCD3menos_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD4CD34masmas$Gene)){
  
  x<-str_split(CD4CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4CD34masmas_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD4CD34masmas_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4CD34masmas_Transcriptome1[order(comparison_GSE33470_CD4CD34masmas_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4CD34masmas_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD4DPCD3$Gene)){
  
  x<-str_split(CD4DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD4DPCD3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3_Transcriptome1[order(comparison_GSE33470_CD4DPCD3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3_Transcriptome1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD4DPCD3mas$Gene)){
  
  x<-str_split(CD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3mas_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD4DPCD3mas_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3mas_Transcriptome1[order(comparison_GSE33470_CD4DPCD3mas_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3mas_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CD4DPCD3menos$Gene)){
  
  x<-str_split(CD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}




  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3menos_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE33470_CD4DPCD3menos_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3menos_Transcriptome1[order(comparison_GSE33470_CD4DPCD3menos_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3menos_Transcriptome1.xlsx", row.names = FALSE)
```


## Targeting1

GSE8879_geneIds ENTREZID, y GSE8879_geneIds SYMBOL son Targeting1 entrezgene y Targeting1 hgnc_symbol, respectivamente. También con el topTab_GSE8879_Anot usado anteriormente, con las mismas columnas ENTREZID y SYMBOL.

Lo haremos con SYMBOL, por ejemplo. Primero cambiaremos c, y continuaremos...

También usaremos el P valor, como los autores.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3$Gene)){
  
  x<-str_split(CD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3_Targeting1[order(comparison_GSE33470_CD3_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34CD4$Gene)){
  
  x<-str_split(CD34CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34CD4_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD34CD4_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34CD4_Targeting1[order(comparison_GSE33470_CD34CD4_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34CD4_Targeting1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34DPCD3$Gene)){
  
  x<-str_split(CD34DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34DPCD3_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD34DPCD3_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34DPCD3_Targeting1[order(comparison_GSE33470_CD34DPCD3_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34DPCD3_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmasCD4$Gene)){
  
  x<-str_split(CD34masmasCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasCD4_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD34masmasCD4_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasCD4_Targeting1[order(comparison_GSE33470_CD34masmasCD4_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasCD4_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmasDPCD3mas$Gene)){
  
  x<-str_split(CD34masmasDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3mas_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD34masmasDPCD3mas_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3mas_Targeting1[order(comparison_GSE33470_CD34masmasDPCD3mas_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3mas_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmasDPCD3menos$Gene)){
  
  x<-str_split(CD34masmasDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3menos_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD34masmasDPCD3menos_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3menos_Targeting1[order(comparison_GSE33470_CD34masmasDPCD3menos_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3menos_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmenosCD4$Gene)){
  
  x<-str_split(CD34masmenosCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosCD4_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD34masmenosCD4_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosCD4_Targeting1[order(comparison_GSE33470_CD34masmenosCD4_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosCD4_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmenosDPCD3mas$Gene)){
  
  x<-str_split(CD34masmenosDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3mas_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD34masmenosDPCD3mas_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3mas_Targeting1[order(comparison_GSE33470_CD34masmenosDPCD3mas_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3mas_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmenosDPCD3menos$Gene)){
  
  x<-str_split(CD34masmenosDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3menos_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD34masmenosDPCD3menos_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3menos_Targeting1[order(comparison_GSE33470_CD34masmenosDPCD3menos_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3menos_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3CD34$Gene)){
  
  x<-str_split(CD3CD34$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD34_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3CD34_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD34_Targeting1[order(comparison_GSE33470_CD3CD34_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD34_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3CD4$Gene)){
  
  x<-str_split(CD3CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD4_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3CD4_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD4_Targeting1[order(comparison_GSE33470_CD3CD4_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD4_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3DPCD3$Gene)){
  
  x<-str_split(CD3DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3DPCD3_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3DPCD3_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3DPCD3_Targeting1[order(comparison_GSE33470_CD3DPCD3_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3DPCD3_Targeting1.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4CD34masmeno$Gene)){
  
  x<-str_split(CD3masCD4CD34masmeno$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD34masmeno_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3masCD4CD34masmeno_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD34masmeno_Targeting1[order(comparison_GSE33470_CD3masCD4CD34masmeno_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD34masmeno_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4CD4$Gene)){
  
  x<-str_split(CD3masCD4CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD4_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3masCD4CD4_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD4_Targeting1[order(comparison_GSE33470_CD3masCD4CD4_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD4_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4DPCD3mas_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3masCD4DPCD3mas_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


VACIO BIEN



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4DPCD3menos_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3masCD4DPCD3menos_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


VACIO BIEN



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8CD34masmas$Gene)){
  
  x<-str_split(CD3masCD8CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmas_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3masCD8CD34masmas_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmas_Targeting1[order(comparison_GSE33470_CD3masCD8CD34masmas_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmas_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8CD34masmenos$Gene)){
  
  x<-str_split(CD3masCD8CD34masmenos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmenos_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3masCD8CD34masmenos_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmenos_Targeting1[order(comparison_GSE33470_CD3masCD8CD34masmenos_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmenos_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8CD4$Gene)){
  
  x<-str_split(CD3masCD8CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD4_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3masCD8CD4_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD4_Targeting1[order(comparison_GSE33470_CD3masCD8CD4_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD4_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD8DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

NULO BIEN


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD8DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8DPCD3menos_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD3masCD8DPCD3menos_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8DPCD3menos_Targeting1[order(comparison_GSE33470_CD3masCD8DPCD3menos_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8DPCD3menos_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4CD34masmas$Gene)){
  
  x<-str_split(CD4CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4CD34masmas_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD4CD34masmas_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4CD34masmas_Targeting1[order(comparison_GSE33470_CD4CD34masmas_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4CD34masmas_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4DPCD3$Gene)){
  
  x<-str_split(CD4DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD4DPCD3_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3_Targeting1[order(comparison_GSE33470_CD4DPCD3_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4DPCD3mas$Gene)){
  
  x<-str_split(CD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3mas_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD4DPCD3mas_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3mas_Targeting1[order(comparison_GSE33470_CD4DPCD3mas_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3mas_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4DPCD3menos$Gene)){
  
  x<-str_split(CD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3menos_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE33470_CD4DPCD3menos_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3menos_Targeting1[order(comparison_GSE33470_CD4DPCD3menos_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3menos_Targeting1.xlsx", row.names = FALSE)
```












## Targeting2

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3$Gene)){
  
  x<-str_split(CD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3_Targeting2[order(comparison_GSE33470_CD3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34CD4$Gene)){
  
  x<-str_split(CD34CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34CD4_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD34CD4_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34CD4_Targeting2[order(comparison_GSE33470_CD34CD4_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34CD4_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34DPCD3$Gene)){
  
  x<-str_split(CD34DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34DPCD3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD34DPCD3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34DPCD3_Targeting2[order(comparison_GSE33470_CD34DPCD3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34DPCD3_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmasCD4$Gene)){
  
  x<-str_split(CD34masmasCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasCD4_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD34masmasCD4_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasCD4_Targeting2[order(comparison_GSE33470_CD34masmasCD4_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasCD4_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmasDPCD3mas$Gene)){
  
  x<-str_split(CD34masmasDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3mas_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD34masmasDPCD3mas_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3mas_Targeting2[order(comparison_GSE33470_CD34masmasDPCD3mas_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3mas_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmasDPCD3menos$Gene)){
  
  x<-str_split(CD34masmasDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3menos_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD34masmasDPCD3menos_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3menos_Targeting2[order(comparison_GSE33470_CD34masmasDPCD3menos_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3menos_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmenosCD4$Gene)){
  
  x<-str_split(CD34masmenosCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosCD4_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD34masmenosCD4_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosCD4_Targeting2[order(comparison_GSE33470_CD34masmenosCD4_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosCD4_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmenosDPCD3mas$Gene)){
  
  x<-str_split(CD34masmenosDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3mas_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD34masmenosDPCD3mas_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3mas_Targeting2[order(comparison_GSE33470_CD34masmenosDPCD3mas_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3mas_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmenosDPCD3menos$Gene)){
  
  x<-str_split(CD34masmenosDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3menos_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD34masmenosDPCD3menos_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3menos_Targeting2[order(comparison_GSE33470_CD34masmenosDPCD3menos_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3menos_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3CD34$Gene)){
  
  x<-str_split(CD3CD34$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD34_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3CD34_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD34_Targeting2[order(comparison_GSE33470_CD3CD34_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD34_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3CD4$Gene)){
  
  x<-str_split(CD3CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD4_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3CD4_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD4_Targeting2[order(comparison_GSE33470_CD3CD4_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD4_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3DPCD3$Gene)){
  
  x<-str_split(CD3DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3DPCD3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3DPCD3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3DPCD3_Targeting2[order(comparison_GSE33470_CD3DPCD3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3DPCD3_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4CD34masmeno$Gene)){
  
  x<-str_split(CD3masCD4CD34masmeno$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD34masmeno_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3masCD4CD34masmeno_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD34masmeno_Targeting2[order(comparison_GSE33470_CD3masCD4CD34masmeno_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD34masmeno_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4CD4$Gene)){
  
  x<-str_split(CD3masCD4CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD4_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3masCD4CD4_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD4_Targeting2[order(comparison_GSE33470_CD3masCD4CD4_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD4_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```


VACIO BIEN


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

VACIO BIEN



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8CD34masmas$Gene)){
  
  x<-str_split(CD3masCD8CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmas_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3masCD8CD34masmas_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmas_Targeting2[order(comparison_GSE33470_CD3masCD8CD34masmas_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmas_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8CD34masmenos$Gene)){
  
  x<-str_split(CD3masCD8CD34masmenos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmenos_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3masCD8CD34masmenos_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmenos_Targeting2[order(comparison_GSE33470_CD3masCD8CD34masmenos_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmenos_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8CD4$Gene)){
  
  x<-str_split(CD3masCD8CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD4_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3masCD8CD4_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD4_Targeting2[order(comparison_GSE33470_CD3masCD8CD4_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD4_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD8DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```
VACIO BIEN



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD8DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8DPCD3menos_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD3masCD8DPCD3menos_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8DPCD3menos_Targeting2[order(comparison_GSE33470_CD3masCD8DPCD3menos_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8DPCD3menos_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4CD34masmas$Gene)){
  
  x<-str_split(CD4CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4CD34masmas_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD4CD34masmas_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4CD34masmas_Targeting2[order(comparison_GSE33470_CD4CD34masmas_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4CD34masmas_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4DPCD3$Gene)){
  
  x<-str_split(CD4DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD4DPCD3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3_Targeting2[order(comparison_GSE33470_CD4DPCD3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4DPCD3mas$Gene)){
  
  x<-str_split(CD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3mas_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD4DPCD3mas_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3mas_Targeting2[order(comparison_GSE33470_CD4DPCD3mas_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3mas_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4DPCD3menos$Gene)){
  
  x<-str_split(CD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3menos_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE33470_CD4DPCD3menos_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3menos_Targeting2[order(comparison_GSE33470_CD4DPCD3menos_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3menos_Targeting2.xlsx", row.names = FALSE)
```



## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3$Gene)){
  
  x<-str_split(CD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3_Targeting5[order(comparison_GSE33470_CD3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34CD4$Gene)){
  
  x<-str_split(CD34CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34CD4_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD34CD4_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34CD4_Targeting5[order(comparison_GSE33470_CD34CD4_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34CD4_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34DPCD3$Gene)){
  
  x<-str_split(CD34DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34DPCD3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD34DPCD3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34DPCD3_Targeting5[order(comparison_GSE33470_CD34DPCD3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34DPCD3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmasCD4$Gene)){
  
  x<-str_split(CD34masmasCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasCD4_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD34masmasCD4_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasCD4_Targeting5[order(comparison_GSE33470_CD34masmasCD4_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasCD4_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmasDPCD3mas$Gene)){
  
  x<-str_split(CD34masmasDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3mas_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD34masmasDPCD3mas_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3mas_Targeting5[order(comparison_GSE33470_CD34masmasDPCD3mas_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3mas_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmasDPCD3menos$Gene)){
  
  x<-str_split(CD34masmasDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3menos_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD34masmasDPCD3menos_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3menos_Targeting5[order(comparison_GSE33470_CD34masmasDPCD3menos_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3menos_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmenosCD4$Gene)){
  
  x<-str_split(CD34masmenosCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosCD4_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD34masmenosCD4_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosCD4_Targeting5[order(comparison_GSE33470_CD34masmenosCD4_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosCD4_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmenosDPCD3mas$Gene)){
  
  x<-str_split(CD34masmenosDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3mas_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD34masmenosDPCD3mas_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3mas_Targeting5[order(comparison_GSE33470_CD34masmenosDPCD3mas_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3mas_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD34masmenosDPCD3menos$Gene)){
  
  x<-str_split(CD34masmenosDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3menos_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD34masmenosDPCD3menos_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3menos_Targeting5[order(comparison_GSE33470_CD34masmenosDPCD3menos_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3menos_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3CD34$Gene)){
  
  x<-str_split(CD3CD34$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD34_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3CD34_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD34_Targeting5[order(comparison_GSE33470_CD3CD34_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD34_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3CD4$Gene)){
  
  x<-str_split(CD3CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD4_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3CD4_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD4_Targeting5[order(comparison_GSE33470_CD3CD4_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD4_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3DPCD3$Gene)){
  
  x<-str_split(CD3DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3DPCD3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3DPCD3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3DPCD3_Targeting5[order(comparison_GSE33470_CD3DPCD3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3DPCD3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4CD34masmeno$Gene)){
  
  x<-str_split(CD3masCD4CD34masmeno$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD34masmeno_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3masCD4CD34masmeno_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD34masmeno_Targeting5[order(comparison_GSE33470_CD3masCD4CD34masmeno_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD34masmeno_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4CD4$Gene)){
  
  x<-str_split(CD3masCD4CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD4_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3masCD4CD4_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD4_Targeting5[order(comparison_GSE33470_CD3masCD4CD4_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD4_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4DPCD3mas_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3masCD4DPCD3mas_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4DPCD3mas_Targeting5[order(comparison_GSE33470_CD3masCD4DPCD3mas_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4DPCD3mas_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD4DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4DPCD3menos_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3masCD4DPCD3menos_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4DPCD3menos_Targeting5[order(comparison_GSE33470_CD3masCD4DPCD3menos_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4DPCD3menos_Targeting5.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8CD34masmas$Gene)){
  
  x<-str_split(CD3masCD8CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmas_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3masCD8CD34masmas_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmas_Targeting5[order(comparison_GSE33470_CD3masCD8CD34masmas_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmas_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8CD34masmenos$Gene)){
  
  x<-str_split(CD3masCD8CD34masmenos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmenos_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3masCD8CD34masmenos_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmenos_Targeting5[order(comparison_GSE33470_CD3masCD8CD34masmenos_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmenos_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8CD4$Gene)){
  
  x<-str_split(CD3masCD8CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD4_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3masCD8CD4_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD4_Targeting5[order(comparison_GSE33470_CD3masCD8CD4_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD4_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD8DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8DPCD3mas_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3masCD8DPCD3mas_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8DPCD3mas_Targeting5[order(comparison_GSE33470_CD3masCD8DPCD3mas_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8DPCD3mas_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD3masCD8DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD8DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8DPCD3menos_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD3masCD8DPCD3menos_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8DPCD3menos_Targeting5[order(comparison_GSE33470_CD3masCD8DPCD3menos_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8DPCD3menos_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4CD34masmas$Gene)){
  
  x<-str_split(CD4CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4CD34masmas_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD4CD34masmas_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4CD34masmas_Targeting5[order(comparison_GSE33470_CD4CD34masmas_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4CD34masmas_Targeting5.xlsx", row.names = FALSE)
```

VAS POR:

```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4DPCD3$Gene)){
  
  x<-str_split(CD4DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD4DPCD3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3_Targeting5[order(comparison_GSE33470_CD4DPCD3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4DPCD3mas$Gene)){
  
  x<-str_split(CD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3mas_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD4DPCD3mas_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3mas_Targeting5[order(comparison_GSE33470_CD4DPCD3mas_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3mas_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CD4DPCD3menos$Gene)){
  
  x<-str_split(CD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3menos_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE33470_CD4DPCD3menos_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3menos_Targeting5[order(comparison_GSE33470_CD4DPCD3menos_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3menos_Targeting5.xlsx", row.names = FALSE)
```



## Large_scale

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3$Gene)){
  
  x<-str_split(CD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3_Large[order(comparison_GSE33470_CD3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD34CD4$Gene)){
  
  x<-str_split(CD34CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34CD4_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD34CD4_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34CD4_Large[order(comparison_GSE33470_CD34CD4_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34CD4_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD34DPCD3$Gene)){
  
  x<-str_split(CD34DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34DPCD3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD34DPCD3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34DPCD3_Large[order(comparison_GSE33470_CD34DPCD3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34DPCD3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD34masmasCD4$Gene)){
  
  x<-str_split(CD34masmasCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasCD4_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD34masmasCD4_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasCD4_Large[order(comparison_GSE33470_CD34masmasCD4_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasCD4_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD34masmasDPCD3mas$Gene)){
  
  x<-str_split(CD34masmasDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3mas_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD34masmasDPCD3mas_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3mas_Large[order(comparison_GSE33470_CD34masmasDPCD3mas_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3mas_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD34masmasDPCD3menos$Gene)){
  
  x<-str_split(CD34masmasDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmasDPCD3menos_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD34masmasDPCD3menos_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmasDPCD3menos_Large[order(comparison_GSE33470_CD34masmasDPCD3menos_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmasDPCD3menos_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD34masmenosCD4$Gene)){
  
  x<-str_split(CD34masmenosCD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosCD4_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD34masmenosCD4_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosCD4_Large[order(comparison_GSE33470_CD34masmenosCD4_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosCD4_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD34masmenosDPCD3mas$Gene)){
  
  x<-str_split(CD34masmenosDPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3mas_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD34masmenosDPCD3mas_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3mas_Large[order(comparison_GSE33470_CD34masmenosDPCD3mas_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3mas_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD34masmenosDPCD3menos$Gene)){
  
  x<-str_split(CD34masmenosDPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD34masmenosDPCD3menos_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD34masmenosDPCD3menos_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD34masmenosDPCD3menos_Large[order(comparison_GSE33470_CD34masmenosDPCD3menos_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD34masmenosDPCD3menos_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3CD34$Gene)){
  
  x<-str_split(CD3CD34$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD34_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3CD34_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD34_Large[order(comparison_GSE33470_CD3CD34_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD34_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3CD4$Gene)){
  
  x<-str_split(CD3CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3CD4_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3CD4_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3CD4_Large[order(comparison_GSE33470_CD3CD4_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3CD4_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3DPCD3$Gene)){
  
  x<-str_split(CD3DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3DPCD3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3DPCD3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3DPCD3_Large[order(comparison_GSE33470_CD3DPCD3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3DPCD3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3masCD4CD34masmeno$Gene)){
  
  x<-str_split(CD3masCD4CD34masmeno$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD34masmeno_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3masCD4CD34masmeno_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD34masmeno_Large[order(comparison_GSE33470_CD3masCD4CD34masmeno_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD34masmeno_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3masCD4CD4$Gene)){
  
  x<-str_split(CD3masCD4CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD4CD4_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3masCD4CD4_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD4CD4_Large[order(comparison_GSE33470_CD3masCD4CD4_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD4CD4_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3masCD4DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

Vacío



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3masCD4DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

Vacíos


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3masCD8CD34masmas$Gene)){
  
  x<-str_split(CD3masCD8CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmas_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3masCD8CD34masmas_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmas_Large[order(comparison_GSE33470_CD3masCD8CD34masmas_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmas_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3masCD8CD34masmenos$Gene)){
  
  x<-str_split(CD3masCD8CD34masmenos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD34masmenos_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3masCD8CD34masmenos_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD34masmenos_Large[order(comparison_GSE33470_CD3masCD8CD34masmenos_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD34masmenos_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3masCD8CD4$Gene)){
  
  x<-str_split(CD3masCD8CD4$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8CD4_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3masCD8CD4_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8CD4_Large[order(comparison_GSE33470_CD3masCD8CD4_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8CD4_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3masCD8DPCD3mas$Gene)){
  
  x<-str_split(CD3masCD8DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8DPCD3mas_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3masCD8DPCD3mas_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8DPCD3mas_Large[order(comparison_GSE33470_CD3masCD8DPCD3mas_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8DPCD3mas_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD3masCD8DPCD3menos$Gene)){
  
  x<-str_split(CD3masCD8DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD3masCD8DPCD3menos_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD3masCD8DPCD3menos_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD3masCD8DPCD3menos_Large[order(comparison_GSE33470_CD3masCD8DPCD3menos_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD3masCD8DPCD3menos_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD4CD34masmas$Gene)){
  
  x<-str_split(CD4CD34masmas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4CD34masmas_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD4CD34masmas_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4CD34masmas_Large[order(comparison_GSE33470_CD4CD34masmas_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4CD34masmas_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD4DPCD3$Gene)){
  
  x<-str_split(CD4DPCD3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD4DPCD3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3_Large[order(comparison_GSE33470_CD4DPCD3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD4DPCD3mas$Gene)){
  
  x<-str_split(CD4DPCD3mas$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3mas_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD4DPCD3mas_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3mas_Large[order(comparison_GSE33470_CD4DPCD3mas_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3mas_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CD4DPCD3menos$Gene)){
  
  x<-str_split(CD4DPCD3menos$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE33470_CD4DPCD3menos_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE33470_CD4DPCD3menos_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE33470_CD4DPCD3menos_Large[order(comparison_GSE33470_CD4DPCD3menos_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/comparison_GSE33470_CD4DPCD3menos_Large.xlsx", row.names = FALSE)
```

































