---
title: "Selección RBPs"
subtitle: ""
author: "Cristina Muntañola Valero"
date: "`r Sys.Date()`"
output:
  pdf_document:
    toc: true 
    toc_depth: 3
    df_print: paged
  html_document:
    toc: true
    toc_depth: 3
    df_print: paged
urlcolor: blue
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

Esta parte se corresponde con la anotación de las comparaciones de los GSE8879, GSE10609, GSE14818 y GSE26713.

# Carga datos artículos

## Transcriptome-wide discovery

Huang, Rongbing, et al. "Transcriptome-wide discovery of coding and noncoding RNA-binding proteins." Proceedings of the National Academy of Sciences 115.17 (2018): E3879-E3887.

Wang, E., Lu, S. X., Pastore, A., Chen, X., Imig, J., Lee, S. C. W., ... & Aifantis, I. (2019). Targeting an RNA-binding protein network in acute myeloid leukemia. Cancer cell, 35(3), 369-384.


Sebestyén, E., Singh, B., Miñana, B., Pagès, A., Mateo, F., Pujana, M. A., ... & Eyras, E. (2016). Large-scale analysis of genome and transcriptome alterations in multiple tumors unveils novel cancer-relevant splicing networks. Genome research, 26(6), 732-744.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
# setwd('C:/Users/laura/Downloads/')
setwd('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library("readxl")
# Transcriptome1 <- read_excel("C:/Users/laura/Downloads/Transcriptome_wide_discovery_1.xlsx")

Transcriptome1 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Transcriptome_wide_discovery_1.xlsx")

```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library("readxl")

# Targeting1 <- read_excel("C:/Users/laura/Downloads/Targeting_an_RNA_Binding_Protein_Network_1.xlsx")

Targeting1 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Targeting_an_RNA_Binding_Protein_Network_1.xlsx")
Targeting2 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Targeting2.xlsx")
Targeting5 <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Targeting5.xlsx")
```

```{r, echo=TRUE, warning=FALSE, message=TRUE, include=TRUE}


# Large_scale <- read_excel("C:/Users/laura/Downloads/Large_scale_analysis.xlsx", sheet = 3, skip = 1)

Large_scale <- read_excel("C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/Large_scale_analysis.xlsx", sheet = 2, skip = 1)
dim(Large_scale)
```


Quitamos las filas y las columnas de cada dataset que no nos interesan.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Transcriptome1)<-Transcriptome1[1,]
Transcriptome1<-Transcriptome1[c(-1,-2,-3),]
Transcriptome1<-Transcriptome1[,-c(6:41, 43)]
colnames(Transcriptome1)<-c("Representative_protein_IDs", "Protein_IDs", "Majority_protein_IDs",
                            "Protein_name", "Gene", "Adjusted_P_values","Classification")
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Targeting1)<-Targeting1[1,]
Targeting1<-Targeting1[c(-1),]
Targeting1<-Targeting1[,-c(4, 5, 7)]

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

colnames(Targeting2) <- Targeting2[1,]
Targeting2<-Targeting2[-1,]
Targeting2<-Targeting2[,-c(3:13)]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Targeting5)<-Targeting5[1,]
Targeting5<-Targeting5[-1,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
Targeting5<-Targeting5[,-c(1:5, 8)]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Targeting5)<-c('id', 'gene', 'gene_description')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
table(Large_scale$tumor)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
Large_scale<-Large_scale[,-c(2,3)]

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
colnames(Large_scale)<-c('HGNC_symbol', 'type')

```

# GSE8879


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(readxl)
load('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/GSE8879topTab_Anot.Rda')
Resultado_GSE8879<-read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/Resultado_GSE8879.xlsx')
```


## Transcriptome


Haremos una funcion para tener los nombres de los genes comunes a nuestra base de datos y a la de Transcriptome1. 

Los Gene de la base de Resultados son correspondientes también al Gene de Transcriptome.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(Resultado_GSE8879$Gene)){
  
  x<-str_split(Resultado_GSE8879$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  
  

```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE8879_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE8879_Transcriptome1)<-c('Numero_col_topTab_GSE8879_Anot',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE8879_Transcriptome1[order(comparison_GSE8879_Transcriptome1$Numero_col_topTab_GSE8879_Anot),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Numero_col_topTab_GSE8879_Anot[i]){
  
  if (yoquese$Numero_col_topTab_GSE8879_Anot[i]==
      yoquese$Numero_col_topTab_GSE8879_Anot[i+1] &&
      yoquese$Proteina[i]==
      yoquese$Proteina[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/comparison_GSE8879_Transcriptome.xlsx", row.names = FALSE)
```


## Targeting1

GSE8879_geneIds ENTREZID, y GSE8879_geneIds SYMBOL son Targeting1 entrezgene y Targeting1 hgnc_symbol, respectivamente. También con el topTab_GSE8879_Anot usado anteriormente, con las mismas columnas ENTREZID y SYMBOL.

Lo haremos con SYMBOL, por ejemplo. Primero cambiaremos c, y continuaremos...

También usaremos el P valor, como los autores.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
table(Targeting1$Pvalue>=0.05)
table(is.na(Targeting1$entrezgene))
table(is.na(Targeting1$ensembl_gene_id))
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Resultado_GSE8879$Gene)){
  
  x<-str_split(Resultado_GSE8879$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE8879_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE8879_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE8879_Targeting1[order(comparison_GSE8879_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/comparison_GSE8879_Targeting1.xlsx", row.names = FALSE)
```


## Targeting2

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Resultado_GSE8879$Gene)){
  
  x<-str_split(Resultado_GSE8879$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE8879_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE8879_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE8879_Targeting2[order(comparison_GSE8879_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Proteína[i]==
      yoquese$Proteína[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/comparison_GSE8879_Targeting2.xlsx", row.names = FALSE)
```




## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(Resultado_GSE8879$Gene)){
  
  x<-str_split(Resultado_GSE8879$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE8879_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE8879_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE8879_Targeting5[order(comparison_GSE8879_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/comparison_GSE8879_Targeting5.xlsx", row.names = FALSE)
```







## Large_scale

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(Resultado_GSE8879$Gene)){
  
  x<-str_split(Resultado_GSE8879$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE8879_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE8879_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE8879_Large[order(comparison_GSE8879_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/comparison_GSE8879_Large.xlsx", row.names = FALSE)
```



# GSE10609



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library("readxl")


HOX<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX.xlsx')
HOX11HOX11L2HOXA<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX11HOX11L2HOXA.xlsx')
HOX11HOX11L2LMO2<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX11HOX11L2LMO2.xlsx')
HOX11HOX11L2TAL1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX11HOX11L2TAL1.xlsx')
HOX11HOXA<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX11HOXA.xlsx')
HOX11L2HOXA<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX11L2HOXA.xlsx')
HOX11L2LMO2<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX11L2LMO2.xlsx')
HOX11L2TAL1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX11L2TAL1.xlsx')
HOX11LMO2<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX11LMO2.xlsx')
HOX11TAL1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOX11TAL1.xlsx')
HOXATAL1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/Resultado_HOXATAL1.xlsx')

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
ls()
```


## Transcriptome



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOX$Gene)){
  
  x<-str_split(HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX_Transcriptome1[order(comparison_GSE10609_HOX_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX_Transcriptome.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()

for (y in 1:length(HOX11HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2HOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX11HOX11L2HOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2HOXA_Transcriptome1[order(comparison_GSE10609_HOX11HOX11L2HOXA_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2HOXA_Transcriptome.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()
emp_vec=c()
emp_vec2=c()


for (y in 1:length(HOX11HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome1[order(comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
dim(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()
emp_vec=c()
emp_vec2=c()

for (y in 1:length(HOX11HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome1[order(comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
dim(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()
emp_vec=c()
emp_vec2=c()

for (y in 1:length(HOX11HOXA$Gene)){
  
  x<-str_split(HOX11HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX11HOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOXA_Transcriptome1[order(comparison_GSE10609_HOX11HOXA_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
dim(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOXA_Transcriptome.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()
emp_vec=c()
emp_vec2=c()

for (y in 1:length(HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2HOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX11L2HOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2HOXA_Transcriptome1[order(comparison_GSE10609_HOX11L2HOXA_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
dim(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2HOXA_Transcriptome.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()
emp_vec=c()
emp_vec2=c()

for (y in 1:length(HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2LMO2_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX11L2LMO2_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2LMO2_Transcriptome1[order(comparison_GSE10609_HOX11L2LMO2_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
dim(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2LMO2_Transcriptome.xlsx", row.names = FALSE)
```






```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()
emp_vec=c()
emp_vec2=c()

for (y in 1:length(HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2TAL1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX11L2TAL1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2TAL1_Transcriptome1[order(comparison_GSE10609_HOX11L2TAL1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
dim(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2TAL1_Transcriptome.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()
emp_vec=c()
emp_vec2=c()

for (y in 1:length(HOX11LMO2$Gene)){
  
  x<-str_split(HOX11LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11LMO2_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX11LMO2_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11LMO2_Transcriptome1[order(comparison_GSE10609_HOX11LMO2_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
dim(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11LMO2_Transcriptome.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()
emp_vec=c()
emp_vec2=c()

for (y in 1:length(HOX11TAL1$Gene)){
  
  x<-str_split(HOX11TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11TAL1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOX11TAL1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11TAL1_Transcriptome1[order(comparison_GSE10609_HOX11TAL1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
dim(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11TAL1_Transcriptome.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()
emp_vec=c()
emp_vec2=c()

for (y in 1:length(HOXATAL1$Gene)){
  
  x<-str_split(HOXATAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]

      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOXATAL1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE10609_HOXATAL1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOXATAL1_Transcriptome1[order(comparison_GSE10609_HOXATAL1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
dim(yoquese)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOXATAL1_Transcriptome.xlsx", row.names = FALSE)
```





## Targeting1

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX$Gene)){
  
  x<-str_split(HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX_Targeting1[order(comparison_GSE10609_HOX_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2HOXA_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX11HOX11L2HOXA_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2HOXA_Targeting1[order(comparison_GSE10609_HOX11HOX11L2HOXA_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2HOXA_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2LMO2_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX11HOX11L2LMO2_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2LMO2_Targeting1[order(comparison_GSE10609_HOX11HOX11L2LMO2_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2TAL1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX11HOX11L2TAL1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2TAL1_Targeting1[order(comparison_GSE10609_HOX11HOX11L2TAL1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2TAL1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOXA$Gene)){
  
  x<-str_split(HOX11HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOXA_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX11HOXA_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOXA_Targeting1[order(comparison_GSE10609_HOX11HOXA_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOXA_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2HOXA_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX11L2HOXA_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2HOXA_Targeting1[order(comparison_GSE10609_HOX11L2HOXA_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2HOXA_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2LMO2_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX11L2LMO2_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2LMO2_Targeting1[order(comparison_GSE10609_HOX11L2LMO2_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2LMO2_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2TAL1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX11L2TAL1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2TAL1_Targeting1[order(comparison_GSE10609_HOX11L2TAL1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2TAL1_Targeting1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11LMO2$Gene)){
  
  x<-str_split(HOX11LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11LMO2_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX11LMO2_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11LMO2_Targeting1[order(comparison_GSE10609_HOX11LMO2_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11LMO2_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11TAL1$Gene)){
  
  x<-str_split(HOX11TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11TAL1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOX11TAL1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11TAL1_Targeting1[order(comparison_GSE10609_HOX11TAL1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11TAL1_Targeting1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATAL1$Gene)){
  
  x<-str_split(HOXATAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOXATAL1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE10609_HOXATAL1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```

Solo hay una fila

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = comparison_GSE10609_HOXATAL1_Targeting1, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOXATAL1_Targeting1.xlsx", row.names = FALSE)
```


## Targeting2


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX$Gene)){
  
  x<-str_split(HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE10609_HOX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX_Targeting2[order(comparison_GSE10609_HOX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2HOXA_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE10609_HOX11HOX11L2HOXA_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Nulo


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2LMO2_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE10609_HOX11HOX11L2LMO2_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2LMO2_Targeting2[order(comparison_GSE10609_HOX11HOX11L2LMO2_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2TAL1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE10609_HOX11HOX11L2TAL1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2TAL1_Targeting2[order(comparison_GSE10609_HOX11HOX11L2TAL1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2TAL1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, error=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOXA$Gene)){
  
  x<-str_split(HOX11HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      
      
      for (i in 1:length(Targeting2$Gene)){
        
        if(!is.na(Targeting2$Gene[i])){
        
          b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
          if (nose2==b && !is.na(nose2) && !is.na(b)){

            nombres_HGNC<-append(nombres_HGNC, b)
            
            if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
            else{descript<-append(descript, NA)}
          
            if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
            else{prot<-append(prot, NA)}
          
            n_col_genes<-append(n_col_genes, i)
          
        }}
      }
  }}}
  
```

No anotamos, vacío.

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2HOXA_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE10609_HOX11L2HOXA_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2HOXA_Targeting2[order(comparison_GSE10609_HOX11L2HOXA_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2HOXA_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2LMO2_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE10609_HOX11L2LMO2_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2LMO2_Targeting2[order(comparison_GSE10609_HOX11L2LMO2_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2LMO2_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2TAL1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE10609_HOX11L2TAL1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2TAL1_Targeting2[order(comparison_GSE10609_HOX11L2TAL1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2TAL1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11LMO2$Gene)){
  
  x<-str_split(HOX11LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11LMO2_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE10609_HOX11LMO2_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11LMO2_Targeting2[order(comparison_GSE10609_HOX11LMO2_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11LMO2_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11TAL1$Gene)){
  
  x<-str_split(HOX11TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        
        if (!is.na(b) && !is.na(nose2) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}}}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11TAL1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE10609_HOX11TAL1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11TAL1_Targeting2[order(comparison_GSE10609_HOX11TAL1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11TAL1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATAL1$Gene)){
  
  x<-str_split(HOXATAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

BIEN; Nulo


## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX$Gene)){
  
  x<-str_split(HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE10609_HOX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX_Targeting5[order(comparison_GSE10609_HOX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2HOXA_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE10609_HOX11HOX11L2HOXA_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2HOXA_Targeting5[order(comparison_GSE10609_HOX11HOX11L2HOXA_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2HOXA_Targeting5.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2LMO2_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE10609_HOX11HOX11L2LMO2_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2LMO2_Targeting5[order(comparison_GSE10609_HOX11HOX11L2LMO2_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2TAL1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE10609_HOX11HOX11L2TAL1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2TAL1_Targeting5[order(comparison_GSE10609_HOX11HOX11L2TAL1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2TAL1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11HOXA$Gene)){
  
  x<-str_split(HOX11HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOXA_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE10609_HOX11HOXA_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOXA_Targeting5[order(comparison_GSE10609_HOX11HOXA_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOXA_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2HOXA_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE10609_HOX11L2HOXA_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2HOXA_Targeting5[order(comparison_GSE10609_HOX11L2HOXA_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2HOXA_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2LMO2_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE10609_HOX11L2LMO2_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2LMO2_Targeting5[order(comparison_GSE10609_HOX11L2LMO2_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2LMO2_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2TAL1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE10609_HOX11L2TAL1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2TAL1_Targeting5[order(comparison_GSE10609_HOX11L2TAL1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2TAL1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOX11LMO2$Gene)){
  
  x<-str_split(HOX11LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11LMO2_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE10609_HOX11LMO2_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11LMO2_Targeting5[order(comparison_GSE10609_HOX11LMO2_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11LMO2_Targeting5.xlsx", row.names = FALSE)
```








## Large_scale


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
ls()


```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX$Gene)){
  
  x<-str_split(HOX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX_Large[order(comparison_GSE10609_HOX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX_Large.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX11HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2HOXA_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX11HOX11L2HOXA_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2HOXA_Large[order(comparison_GSE10609_HOX11HOX11L2HOXA_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2HOXA_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX11HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2LMO2_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX11HOX11L2LMO2_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2LMO2_Large[order(comparison_GSE10609_HOX11HOX11L2LMO2_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX11HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOX11L2TAL1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX11HOX11L2TAL1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOX11L2TAL1_Large[order(comparison_GSE10609_HOX11HOX11L2TAL1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2TAL1_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX11HOXA$Gene)){
  
  x<-str_split(HOX11HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11HOXA_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX11HOXA_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11HOXA_Large[order(comparison_GSE10609_HOX11HOXA_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11HOXA_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX11L2HOXA$Gene)){
  
  x<-str_split(HOX11L2HOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2HOXA_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX11L2HOXA_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2HOXA_Large[order(comparison_GSE10609_HOX11L2HOXA_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2HOXA_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX11L2LMO2$Gene)){
  
  x<-str_split(HOX11L2LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2LMO2_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX11L2LMO2_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2LMO2_Large[order(comparison_GSE10609_HOX11L2LMO2_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2LMO2_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX11L2TAL1$Gene)){
  
  x<-str_split(HOX11L2TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11L2TAL1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX11L2TAL1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11L2TAL1_Large[order(comparison_GSE10609_HOX11L2TAL1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11L2TAL1_Large.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX11LMO2$Gene)){
  
  x<-str_split(HOX11LMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11LMO2_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX11LMO2_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11LMO2_Large[order(comparison_GSE10609_HOX11LMO2_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11LMO2_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOX11TAL1$Gene)){
  
  x<-str_split(HOX11TAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOX11TAL1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOX11TAL1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOX11TAL1_Large[order(comparison_GSE10609_HOX11TAL1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOX11TAL1_Large.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXATAL1$Gene)){
  
  x<-str_split(HOXATAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE10609_HOXATAL1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE10609_HOXATAL1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE10609_HOXATAL1_Large[order(comparison_GSE10609_HOXATAL1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/comparison_GSE10609_HOXATAL1_Large.xlsx", row.names = FALSE)
```




# GSE14518_GPL570

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library("readxl")


CNR<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/Resultado_GPL570_CNR.xlsx')
FNR<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/Resultado_GPL570_FNR.xlsx')


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
ls()
```

## Transcriptome



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14518_GPL570_CNR_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE14518_GPL570_CNR_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14518_GPL570_CNR_Transcriptome1[order(comparison_GSE14518_GPL570_CNR_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14518_GPL570_CNR_Transcriptome.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14518_GPL570_FNR_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE14518_GPL570_FNR_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14518_GPL570_FNR_Transcriptome1[order(comparison_GSE14518_GPL570_FNR_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14518_GPL570_FNR_Transcriptome.xlsx", row.names = FALSE)
```




## Targeting1

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14518_GPL570_CNR_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE14518_GPL570_CNR_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14518_GPL570_CNR_Targeting1[order(comparison_GSE14518_GPL570_CNR_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL570_CNR_Targeting.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL570_FNR_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE14618_GPL570_FNR_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL570_FNR_Targeting1[order(comparison_GSE14618_GPL570_FNR_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL570_FNR_Targeting.xlsx", row.names = FALSE)
```



## Targeting2


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL570_CNR_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE14618_GPL570_CNR_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL570_CNR_Targeting2[order(comparison_GSE14618_GPL570_CNR_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL570_CNR_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL570_FNR_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE14618_GPL570_FNR_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL570_FNR_Targeting2[order(comparison_GSE14618_GPL570_FNR_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL570_FNR_Targeting2.xlsx", row.names = FALSE)
```



## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL570_CNR_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE14618_GPL570_CNR_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL570_CNR_Targeting5[order(comparison_GSE14618_GPL570_CNR_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL570_CNR_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL570_FNR_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE14618_GPL570_FNR_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL570_FNR_Targeting5[order(comparison_GSE14618_GPL570_FNR_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL570_FNR_Targeting5.xlsx", row.names = FALSE)
```


## Large_scale





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL570_CNR_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE14618_GPL570_CNR_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL570_CNR_Large[order(comparison_GSE14618_GPL570_CNR_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL570_CNR_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL570_FNR_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE14618_GPL570_FNR_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL570_FNR_Large[order(comparison_GSE14618_GPL570_FNR_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL570_FNR_Large.xlsx", row.names = FALSE)
```





# GSE14618_GPL96




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library("readxl")


CNR<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/Resultado_GPL96_CNR.xlsx')
FNR<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/Resultado_GPL96_FNR.xlsx')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
ls()
```


## Transcriptome



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14518_GPL96_CNR_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE14518_GPL96_CNR_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14518_GPL96_CNR_Transcriptome1[order(comparison_GSE14518_GPL96_CNR_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14518_GPL96_CNR_Transcriptome.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14518_GPL96_FNR_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE14518_GPL96_FNR_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14518_GPL96_FNR_Transcriptome1[order(comparison_GSE14518_GPL96_FNR_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14518_GPL570_FNR_Transcriptome.xlsx", row.names = FALSE)
```




## Targeting1

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14518_GPL96_CNR_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE14518_GPL96_CNR_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14518_GPL96_CNR_Targeting1[order(comparison_GSE14518_GPL96_CNR_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14518_GPL96_CNR_Targeting.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL96_FNR_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE14618_GPL96_FNR_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL96_FNR_Targeting1[order(comparison_GSE14618_GPL96_FNR_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL96_FNR_Targeting.xlsx", row.names = FALSE)
```



## Targeting2


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL96_CNR_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE14618_GPL96_CNR_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL96_CNR_Targeting2[order(comparison_GSE14618_GPL96_CNR_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL96_CNR_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL96_FNR_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE14618_GPL96_FNR_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL96_FNR_Targeting2[order(comparison_GSE14618_GPL96_FNR_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL96_FNR_Targeting2.xlsx", row.names = FALSE)
```



## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL96_CNR_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE14618_GPL96_CNR_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL96_CNR_Targeting5[order(comparison_GSE14618_GPL96_CNR_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL96_CNR_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL96_FNR_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE14618_GPL96_FNR_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL96_FNR_Targeting5[order(comparison_GSE14618_GPL96_FNR_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL96_FNR_Targeting5.xlsx", row.names = FALSE)
```


## Large_scale





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(CNR$Gene)){
  
  x<-str_split(CNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL96_CNR_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE14618_GPL96_CNR_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL96_CNR_Large[order(comparison_GSE14618_GPL96_CNR_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL96_CNR_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(FNR$Gene)){
  
  x<-str_split(FNR$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE14618_GPL96_FNR_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE14618_GPL96_FNR_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE14618_GPL96_FNR_Large[order(comparison_GSE14618_GPL96_FNR_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE14618_GPL96_FNR_Large.xlsx", row.names = FALSE)
```









# GSE26713





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library("readxl")


BMHOXA<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_BMHOXA.xlsx')

BMLMO1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_BMLMO1.xlsx')


BMLMO2<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_BMLMO2.xlsx')

BMMYB<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_BMMYB.xlsx')


BMTAL1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_BMTAL1.xlsx')

BMTAL2<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_BMTAL2.xlsx')

BMTAL2LMO1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_BMTAL2LMO1.xlsx')

BMTLX1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_BMTLX1.xlsx')


BMTLX3<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_BMTLX3.xlsx')

HOXALMO2<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_HOXALMO2.xlsx')

HOXATAL1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_HOXATAL1.xlsx')

HOXATAL2LMO1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_HOXATAL2LMO1.xlsx')


HOXATLX1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_HOXATLX1.xlsx')

HOXATLX3<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_HOXATLX3.xlsx')

LMO1LMO2MYB<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMO1LMO2MYB.xlsx')


LMO2TAL2LMO1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMO2TAL2LMO1.xlsx')

LMO2TLX1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMO2TLX1.xlsx')

LMO2TLX3<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMO2TLX3.xlsx')

LMOBM<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMOBM.xlsx')



LMOTAL1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMOTAL1.xlsx')

LMOTAL2<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMOTAL2.xlsx')

LMOTAL2LMO1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMOTAL2LMO1.xlsx')

LMOTLX1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMOTLX1.xlsx')


LMOTLX3<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_LMOTLX3.xlsx')

HOXALMO2<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_MYBTAL2.xlsx')

MYBTLX1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_MYBTLX1.xlsx')

MYBTLX3<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_MYBTLX3.xlsx')


TAL1TAL2LMO1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TAL1TAL2LMO1.xlsx')

TAL1TLX1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TAL1TLX1.xlsx')


TAL1TLX3<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TAL1TLX3.xlsx')

TAL2LMO1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TAL2LMO1.xlsx')


TALBM<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TALBM.xlsx')

TALMYB<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TALMYB.xlsx')

TALTAL2LMO1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TALTAL2LMO1.xlsx')

TALTLX<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TALTLX.xlsx')


TALTLX1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TALTLX1.xlsx')

TALTLX3<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TALTLX3.xlsx')

TALTLXL<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TALTLXL.xlsx')

TLX<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TLX.xlsx')


TLXBM<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TLXBM.xlsx')

TLXMYB<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TLXMYB.xlsx')

TLXTAL1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TLXTAL1.xlsx')

TLXTAL2<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TLXTAL2.xlsx')

TLXTAL2LMO1<- read_excel('C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/Resultado_TLXTAL2LMO1.xlsx')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
ls()
```


## Transcriptome



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


  
for (y in 1:length(BMHOXA$Gene)){
  
  x<-str_split(BMHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
          }
          }
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMHOXA_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_BMHOXA_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMHOXA_Transcriptome1[order(comparison_GSE26713_BMHOXA_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMHOXA_Transcriptome1.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(BMLMO1$Gene)){
  
  x<-str_split(BMLMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_BMLMO1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO1_Transcriptome1[order(comparison_GSE26713_BMLMO1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMLMO1_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(BMLMO2$Gene)){
  
  x<-str_split(BMLMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO2_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_BMLMO2_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO2_Transcriptome1[order(comparison_GSE26713_BMLMO2_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMLMO2_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(BMMYB$Gene)){
  
  x<-str_split(BMMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    if (length(Transcriptome1$Gene)>1){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") }

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMMYB_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_BMMYB_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMMYB_Transcriptome1[order(comparison_GSE26713_BMMYB_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMMYB_Transcriptome1.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(BMTAL1$Gene)){
  
  x<-str_split(BMTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_BMTAL1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL1_Transcriptome1[order(comparison_GSE26713_BMTAL1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(BMTAL2$Gene)){
  
  x<-str_split(BMTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_BMTAL2_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2_Transcriptome1[order(comparison_GSE26713_BMTAL2_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL2_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(BMTAL2LMO1$Gene)){
  
  x<-str_split(BMTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2LMO1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_BMTAL2LMO1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2LMO1_Transcriptome1[order(comparison_GSE26713_BMTAL2LMO1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL2LMO1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(BMTLX1$Gene)){
  
  x<-str_split(BMTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_BMTLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX1_Transcriptome1[order(comparison_GSE26713_BMTLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTLX1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(BMTLX3$Gene)){
  
  x<-str_split(BMTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_BMTLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX3_Transcriptome1[order(comparison_GSE26713_BMTLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTLX3_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_HOXALMO2_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Transcriptome1[order(comparison_GSE26713_HOXALMO2_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXALMO2_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXATAL1$Gene)){
  
  x<-str_split(HOXATAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_HOXATAL1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL1_Transcriptome1[order(comparison_GSE26713_HOXATAL1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATAL1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXATAL2LMO1$Gene)){
  
  x<-str_split(HOXATAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL2LMO1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_HOXATAL2LMO1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL2LMO1_Transcriptome1[order(comparison_GSE26713_HOXATAL2LMO1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATAL2LMO1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXATLX1$Gene)){
  
  x<-str_split(HOXATLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_HOXATLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX1_Transcriptome1[order(comparison_GSE26713_HOXATLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATLX1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXATLX3$Gene)){
  
  x<-str_split(HOXATLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_HOXATLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX3_Transcriptome1[order(comparison_GSE26713_HOXATLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATLX3_Transcriptome1.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(LMO1LMO2MYB$Gene)){
  
  x<-str_split(LMO1LMO2MYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO1LMO2MYB_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_LMO1LMO2MYB_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO1LMO2MYB_Transcriptome1[order(comparison_GSE26713_LMO1LMO2MYB_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO1LMO2MYB_Transcriptome1.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(LMO2TAL2LMO1$Gene)){
  
  x<-str_split(LMO2TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TAL2LMO1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_LMO2TAL2LMO1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TAL2LMO1_Transcriptome1[order(comparison_GSE26713_LMO2TAL2LMO1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TAL2LMO1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(LMO2TLX1$Gene)){
  
  x<-str_split(LMO2TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_LMO2TLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX1_Transcriptome1[order(comparison_GSE26713_LMO2TLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TLX1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(LMO2TLX3$Gene)){
  
  x<-str_split(LMO2TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_LMO2TLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX3_Transcriptome1[order(comparison_GSE26713_LMO2TLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TLX3_Transcriptome1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(LMOBM$Gene)){
  
  x<-str_split(LMOBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOBM_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_LMOBM_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOBM_Transcriptome1[order(comparison_GSE26713_LMOBM_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOBM_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(LMOTAL1$Gene)){
  
  x<-str_split(LMOTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_LMOTAL1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL1_Transcriptome1[order(comparison_GSE26713_LMOTAL1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(LMOTAL2$Gene)){
  
  x<-str_split(LMOTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL2_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_LMOTAL2_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL2_Transcriptome1[order(comparison_GSE26713_LMOTAL2_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL2_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(LMOTLX1$Gene)){
  
  x<-str_split(LMOTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTLX1_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_LMOTLX1_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTLX1_Transcriptome1[order(comparison_GSE26713_LMOTLX1_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTLX1_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(LMOTLX3$Gene)){
  
  x<-str_split(LMOTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTLX3_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_LMOTLX3_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTLX3_Transcriptome1[order(comparison_GSE26713_LMOTLX3_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTLX3_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Transcriptome1<-data.frame(n_col_genes, nombres_genes, clasificacion_RBPs, funcion_biologica)
colnames(comparison_GSE26713_HOXALMO2_Transcriptome1)<-c('Numero_col_Transcriptome',
                                               'Proteina',
                                               'Clasificacion_RBPs',
                                               'Funcion_Biologica')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Transcriptome1[order(comparison_GSE26713_HOXALMO2_Transcriptome1$Proteina),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  for (j in yoquese$Proteina[i]){
  
  if (yoquese$Proteina[i]==
      yoquese$Proteina[i+1] &&
      yoquese$Numero_col_Transcriptome[i]==
      yoquese$Numero_col_Transcriptome[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXALMO2_Transcriptome1.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}

library(stringr)
library(dplyr)
nombres_genes<-c()
clasificacion_RBPs<-c()
funcion_biologica<-c()
n_col_genes<-c()


emp_vec=c()

emp_vec2=c()


for (y in 1:length(MYBTLX1$Gene)){
  
  x<-str_split(MYBTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                  # BIEN, VECTOR. Elementos [1] "PLCG1" [2]"LAT"  etc
    

    
    
  for (i in 1:length(Transcriptome1$Gene)){
    
    b<-str_split(Transcriptome1$Gene[i], pattern = ";") 

    
    for (nose in 1:length(b)){
      
      emp_vec2<-b[[nose]]
        
      
      
      
      
      
      for (nose2 in emp_vec){ # BIEN. Valores del vector
        for (nose3 in emp_vec2){
          
          if (nose2==nose3 && !is.na(nose2) && !is.na(nose3)){
          
          nombres_genes<-append(nombres_genes, nose2)
          
          clasificacion_RBPs<-append(clasificacion_RBPs, Transcriptome1$Classification[i])
          
          n_col_genes<-append(n_col_genes, i)

          if (!is.na(Transcriptome1$Protein_name[i])){
          funcion_biologica<-append(funcion_biologica, Transcriptome1$Protein_name[i])}
          
          else{funcion_biologica<-append(funcion_biologica, 'Sin Función')}
        }}
      }
  }}}}


  

```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(clasificacion_RBPs)
length(n_col_genes)==length(funcion_biologica)
```

Este estña vacío.



## Targeting1

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMHOXA$Gene)){
  
  x<-str_split(BMHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMHOXA_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_BMHOXA_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMHOXA_Targeting1[order(comparison_GSE26713_BMHOXA_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMHOXA_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMLMO1$Gene)){
  
  x<-str_split(BMLMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_BMLMO1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO1_Targeting1[order(comparison_GSE26713_BMLMO1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMLMO1_Targeting1.xlsx", row.names = FALSE)
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMLMO2$Gene)){
  
  x<-str_split(BMLMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO2_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_BMLMO2_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO2_Targeting1[order(comparison_GSE26713_BMLMO2_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMLMO2_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMMYB$Gene)){
  
  x<-str_split(BMMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMMYB_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_BMMYB_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMMYB_Targeting1[order(comparison_GSE26713_BMMYB_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMMYB_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTAL1$Gene)){
  
  x<-str_split(BMTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_BMTAL1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL1_Targeting1[order(comparison_GSE26713_BMTAL1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTAL2$Gene)){
  
  x<-str_split(BMTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_BMTAL2_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2_Targeting1[order(comparison_GSE26713_BMTAL2_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL2_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTAL2LMO1$Gene)){
  
  x<-str_split(BMTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2LMO1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_BMTAL2LMO1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2LMO1_Targeting1[order(comparison_GSE26713_BMTAL2LMO1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL2LMO1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTLX1$Gene)){
  
  x<-str_split(BMTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_BMTLX1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX1_Targeting1[order(comparison_GSE26713_BMTLX1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTLX1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTLX3$Gene)){
  
  x<-str_split(BMTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX3_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_BMTLX3_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX3_Targeting1[order(comparison_GSE26713_BMTLX3_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTLX3_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_HOXALMO2_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Targeting1[order(comparison_GSE26713_HOXALMO2_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXALMO2_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATAL1$Gene)){
  
  x<-str_split(HOXATAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_HOXATAL1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL1_Targeting1[order(comparison_GSE26713_HOXATAL1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATAL1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATAL2LMO1$Gene)){
  
  x<-str_split(HOXATAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL2LMO1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_HOXATAL2LMO1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL2LMO1_Targeting1[order(comparison_GSE26713_HOXATAL2LMO1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATAL2LMO1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATLX1$Gene)){
  
  x<-str_split(HOXATLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_HOXATLX1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX1_Targeting1[order(comparison_GSE26713_HOXATLX1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATLX1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATLX3$Gene)){
  
  x<-str_split(HOXATLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX3_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_HOXATLX3_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX3_Targeting1[order(comparison_GSE26713_HOXATLX3_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATLX3_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO1LMO2MYB$Gene)){
  
  x<-str_split(LMO1LMO2MYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO1LMO2MYB_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMO1LMO2MYB_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO1LMO2MYB_Targeting1[order(comparison_GSE26713_LMO1LMO2MYB_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO1LMO2MYB_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO2TAL2LMO1$Gene)){
  
  x<-str_split(LMO2TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TAL2LMO1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMO2TAL2LMO1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TAL2LMO1_Targeting1[order(comparison_GSE26713_LMO2TAL2LMO1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TAL2LMO1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO2TLX1$Gene)){
  
  x<-str_split(LMO2TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMO2TLX1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX1_Targeting1[order(comparison_GSE26713_LMO2TLX1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TLX1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO2TLX3$Gene)){
  
  x<-str_split(LMO2TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX3_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMO2TLX3_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX3_Targeting1[order(comparison_GSE26713_LMO2TLX3_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TLX3_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOBM$Gene)){
  
  x<-str_split(LMOBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOBM_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMOBM_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOBM_Targeting1[order(comparison_GSE26713_LMOBM_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOBM_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTAL1$Gene)){
  
  x<-str_split(LMOTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMOTAL1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL1_Targeting1[order(comparison_GSE26713_LMOTAL1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTAL2$Gene)){
  
  x<-str_split(LMOTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL2_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMOTAL2_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL2_Targeting1[order(comparison_GSE26713_LMOTAL2_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL2_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTAL2LMO1$Gene)){
  
  x<-str_split(LMOTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL2LMO1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMOTAL2LMO1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL2LMO1_Targeting1[order(comparison_GSE26713_LMOTAL2LMO1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL2LMO1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTLX1$Gene)){
  
  x<-str_split(LMOTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTLX1_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMOTLX1_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTLX1_Targeting1[order(comparison_GSE26713_LMOTLX1_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTLX1_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTLX3$Gene)){
  
  x<-str_split(LMOTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTLX3_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_LMOTLX3_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```

BIEN, nulo


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes_entrez)==length(nombres_genes_ensembl)

length(nombres_genes_ensembl)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Targeting1<-data.frame(n_col_genes, nombres_genes_entrez, nombres_genes_ensembl)
colnames(comparison_GSE26713_HOXALMO2_Targeting1)=c('Numero_col_BBDDc',
                                          'Entrez',
                                          'Ensembl')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Targeting1[order(comparison_GSE26713_HOXALMO2_Targeting1$Entrez),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Entrez[i]==
      yoquese$Entrez[i+1]){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXALMO2_Targeting1.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_genes_entrez<-c()
nombres_genes_ensembl<-c()
n_col_genes<-c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MYBTLX1$Gene)){
  
  x<-str_split(MYBTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting1$hgnc_symbol)){
        
        b<-Targeting1$hgnc_symbol[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          nombres_genes_entrez<-append(nombres_genes_entrez, b)
          
          nombres_genes_ensembl<-append(nombres_genes_ensembl, Targeting1$ensembl_gene_id[i])
          
          n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

Este está nulo


## Targeting2


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMHOXA$Gene)){
  
  x<-str_split(BMHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMHOXA_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_BMHOXA_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMHOXA_Targeting2[order(comparison_GSE26713_BMHOXA_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMHOXA_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMLMO1$Gene)){
  
  x<-str_split(BMLMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_BMLMO1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO1_Targeting2[order(comparison_GSE26713_BMLMO1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMLMO1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMLMO2$Gene)){
  
  x<-str_split(BMLMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO2_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_BMLMO2_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO2_Targeting2[order(comparison_GSE26713_BMLMO2_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMLMO2_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMMYB$Gene)){
  
  x<-str_split(BMMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMMYB_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_BMMYB_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMMYB_Targeting2[order(comparison_GSE26713_BMMYB_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMMYB_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTAL1$Gene)){
  
  x<-str_split(BMTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_BMTAL1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL1_Targeting2[order(comparison_GSE26713_BMTAL1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTAL2$Gene)){
  
  x<-str_split(BMTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_BMTAL2_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2_Targeting2[order(comparison_GSE26713_BMTAL2_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL2_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTAL2LMO1$Gene)){
  
  x<-str_split(BMTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2LMO1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_BMTAL2LMO1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2LMO1_Targeting2[order(comparison_GSE26713_BMTAL2LMO1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL2LMO1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTLX1$Gene)){
  
  x<-str_split(BMTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_BMTLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX1_Targeting2[order(comparison_GSE26713_BMTLX1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTLX1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTLX3$Gene)){
  
  x<-str_split(BMTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_BMTLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX3_Targeting2[order(comparison_GSE26713_BMTLX3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTLX3_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_HOXALMO2_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Targeting2[order(comparison_GSE26713_HOXALMO2_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXALMO2_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATAL1$Gene)){
  
  x<-str_split(HOXATAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_HOXATAL1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL1_Targeting2[order(comparison_GSE26713_HOXATAL1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATAL1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATAL2LMO1$Gene)){
  
  x<-str_split(HOXATAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL2LMO1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_HOXATAL2LMO1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL2LMO1_Targeting2[order(comparison_GSE26713_HOXATAL2LMO1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATAL2LMO1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATLX1$Gene)){
  
  x<-str_split(HOXATLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_HOXATLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX1_Targeting2[order(comparison_GSE26713_HOXATLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATLX1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATLX3$Gene)){
  
  x<-str_split(HOXATLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_HOXATLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX3_Targeting2[order(comparison_GSE26713_HOXATLX3_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATLX3_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO1LMO2MYB$Gene)){
  
  x<-str_split(LMO1LMO2MYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO1LMO2MYB_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_LMO1LMO2MYB_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO1LMO2MYB_Targeting2[order(comparison_GSE26713_LMO1LMO2MYB_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO1LMO2MYB_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO2TAL2LMO1$Gene)){
  
  x<-str_split(LMO2TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TAL2LMO1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_LMO2TAL2LMO1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TAL2LMO1_Targeting2[order(comparison_GSE26713_LMO2TAL2LMO1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TAL2LMO1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO2TLX1$Gene)){
  
  x<-str_split(LMO2TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_LMO2TLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX1_Targeting2[order(comparison_GSE26713_LMO2TLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TLX1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO2TLX3$Gene)){
  
  x<-str_split(LMO2TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_LMO2TLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX3_Targeting2[order(comparison_GSE26713_LMO2TLX3_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TLX3_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOBM$Gene)){
  
  x<-str_split(LMOBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOBM_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_LMOBM_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOBM_Targeting2[order(comparison_GSE26713_LMOBM_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOBM_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTAL1$Gene)){
  
  x<-str_split(LMOTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_LMOTAL1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL1_Targeting2[order(comparison_GSE26713_LMOTAL1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTAL2$Gene)){
  
  x<-str_split(LMOTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL2_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_LMOTAL2_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL2_Targeting2[order(comparison_GSE26713_LMOTAL2_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL2_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTAL2LMO1$Gene)){
  
  x<-str_split(LMOTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL2LMO1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_LMOTAL2LMO1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL2LMO1_Targeting2[order(comparison_GSE26713_LMOTAL2LMO1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL2LMO1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTLX1$Gene)){
  
  x<-str_split(LMOTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_LMOTLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTLX1_Targeting2[order(comparison_GSE26713_LMOTLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTLX1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTLX3$Gene)){
  
  x<-str_split(LMOTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```



BIEN: nulo


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_HOXALMO2_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Targeting2[order(comparison_GSE26713_HOXALMO2_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXALMO2_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MYBTLX1$Gene)){
  
  x<-str_split(MYBTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_MYBTLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_MYBTLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Ultimo, bien, nulo...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_MYBTLX1_Targeting2[order(comparison_GSE26713_MYBTLX1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_MYBTLX1_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MYBTLX3$Gene)){
  
  x<-str_split(MYBTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_MYBTLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_MYBTLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

Nulo, ultimo, bien

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_MYBTLX3_Targeting2[order(comparison_GSE26713_MYBTLX3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_MYBTLX3_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL1TAL2LMO1$Gene)){
  
  x<-str_split(TAL1TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL1TAL2LMO1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TAL1TAL2LMO1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL1TAL2LMO1_Targeting2[order(comparison_GSE26713_TAL1TAL2LMO1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TAL1TAL2LMO1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL1TLX1$Gene)){
  
  x<-str_split(TAL1TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL1TLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TAL1TLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL1TLX1_Targeting2[order(comparison_GSE26713_TAL1TLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TAL1TLX1_Targeting2.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL1TLX3$Gene)){
  
  x<-str_split(TAL1TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL1TLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TAL1TLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL1TLX3_Targeting2[order(comparison_GSE26713_TAL1TLX3_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TAL1TLX3_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL2LMO1$Gene)){
  
  x<-str_split(TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL2LMO1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TAL2LMO1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```


Nulo final, bien

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL2LMO1_Targeting2[order(comparison_GSE26713_TAL2LMO1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TAL2LMO1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALBM$Gene)){
  
  x<-str_split(TALBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALBM_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TALBM_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALBM_Targeting2[order(comparison_GSE26713_TALBM_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALBM_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALMYB$Gene)){
  
  x<-str_split(TALMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALMYB_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TALMYB_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALMYB_Targeting2[order(comparison_GSE26713_TALMYB_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALMYB_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTAL2LMO1$Gene)){
  
  x<-str_split(TALTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTAL2LMO1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TALTAL2LMO1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTAL2LMO1_Targeting2[order(comparison_GSE26713_TALTAL2LMO1_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTAL2LMO1_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTLX$Gene)){
  
  x<-str_split(TALTLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TALTLX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLX_Targeting2[order(comparison_GSE26713_TALTLX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTLX_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTLX1$Gene)){
  
  x<-str_split(TALTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLX1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TALTLX1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLX1_Targeting2[order(comparison_GSE26713_TALTLX1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTLX1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTLX3$Gene)){
  
  x<-str_split(TALTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLX3_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TALTLX3_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLX3_Targeting2[order(comparison_GSE26713_TALTLX3_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTLX3_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTLXL$Gene)){
  
  x<-str_split(TALTLXL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLXL_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TALTLXL_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLXL_Targeting2[order(comparison_GSE26713_TALTLXL_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTLXL_Targeting2.xlsx", row.names = FALSE)
```

AQUII


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLX$Gene)){
  
  x<-str_split(TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLX_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TLX_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLX_Targeting2[order(comparison_GSE26713_TLX_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLX_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXBM$Gene)){
  
  x<-str_split(TLXBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXBM_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TLXBM_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXBM_Targeting2[order(comparison_GSE26713_TLXBM_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXBM_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXMYB$Gene)){
  
  x<-str_split(TLXMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXMYB_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TLXMYB_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXMYB_Targeting2[order(comparison_GSE26713_TLXMYB_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXMYB_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXTAL1$Gene)){
  
  x<-str_split(TLXTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXTAL1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TLXTAL1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXTAL1_Targeting2[order(comparison_GSE26713_TLXTAL1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXTAL1_Targeting2.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXTAL2$Gene)){
  
  x<-str_split(TLXTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXTAL2_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TLXTAL2_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXTAL2_Targeting2[order(comparison_GSE26713_TLXTAL2_Targeting2$Numero_col_Targeting2),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$Numero_col_Targeting2[i]==
      yoquese$Numero_col_Targeting2[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXTAL2_Targeting2.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
prot<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXTAL2LMO1$Gene)){
  
  x<-str_split(TLXTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting2$Gene)){
        
        b<-Targeting2$Gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (nose2==b && !is.na(nose2) && !is.na(b)){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting2$Description[i])){descript<-append(descript, Targeting2$Description[i])}
          else{descript<-append(descript, NA)}
          
          if(!is.na(Targeting2$Protein[i])){prot<-append(prot, Targeting2$Protein[i])}
          else{prot<-append(prot, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(nombres_HGNC)==length(n_col_genes)
length(prot)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXTAL2LMO1_Targeting2<-data.frame(n_col_genes, nombres_HGNC, descript, prot)
colnames(comparison_GSE26713_TLXTAL2LMO1_Targeting2)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion',
                                          'Proteína')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXTAL2LMO1_Targeting2[order(comparison_GSE26713_TLXTAL2LMO1_Targeting2$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion[i]==
      yoquese$Descripcion[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXTAL2LMO1_Targeting2.xlsx", row.names = FALSE)
```



## Targeting5

Gene es lo común a Resultado y Targeting2.


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMHOXA$Gene)){
  
  x<-str_split(BMHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMHOXA_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_BMHOXA_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMHOXA_Targeting5[order(comparison_GSE26713_BMHOXA_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_BMHOXA_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMLMO1$Gene)){
  
  x<-str_split(BMLMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_BMLMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO1_Targeting5[order(comparison_GSE26713_BMLMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_BMLMO1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMLMO2$Gene)){
  
  x<-str_split(BMLMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO2_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_BMLMO2_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO2_Targeting5[order(comparison_GSE26713_BMLMO2_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_BMLMO2_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMMYB$Gene)){
  
  x<-str_split(BMMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_BMLMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO1_Targeting5[order(comparison_GSE26713_BMLMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_BMLMO1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTAL1$Gene)){
  
  x<-str_split(BMTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_BMTAL1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL1_Targeting5[order(comparison_GSE26713_BMTAL1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_BMTAL1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTAL2$Gene)){
  
  x<-str_split(BMTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_BMTAL2_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2_Targeting5[order(comparison_GSE26713_BMTAL2_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_BMTAL2_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTAL2LMO1$Gene)){
  
  x<-str_split(BMTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2LMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_BMTAL2LMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2LMO1_Targeting5[order(comparison_GSE26713_BMTAL2LMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_BMTAL2LMO1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTLX1$Gene)){
  
  x<-str_split(BMTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_BMTLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX1_Targeting5[order(comparison_GSE26713_BMTLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_BMTLX1_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(BMTLX3$Gene)){
  
  x<-str_split(BMTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_BMTLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX3_Targeting5[order(comparison_GSE26713_BMTLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_BMTLX3_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_HOXALMO2_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Targeting5[order(comparison_GSE26713_HOXALMO2_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_HOXALMO2_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATAL1$Gene)){
  
  x<-str_split(HOXATAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_HOXATAL1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL1_Targeting5[order(comparison_GSE26713_HOXATAL1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_HOXATAL1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATAL2LMO1$Gene)){
  
  x<-str_split(HOXATAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL2LMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_HOXATAL2LMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL2LMO1_Targeting5[order(comparison_GSE26713_HOXATAL2LMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_HOXATAL2LMO1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATLX1$Gene)){
  
  x<-str_split(HOXATLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_HOXATLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX1_Targeting5[order(comparison_GSE26713_HOXATLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_HOXATLX1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXATLX3$Gene)){
  
  x<-str_split(HOXATLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_HOXATLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX3_Targeting5[order(comparison_GSE26713_HOXATLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_HOXATLX3_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO1LMO2MYB$Gene)){
  
  x<-str_split(LMO1LMO2MYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO1LMO2MYB_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMO1LMO2MYB_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO1LMO2MYB_Targeting5[order(comparison_GSE26713_LMO1LMO2MYB_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_LMO1LMO2MYB_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO2TAL2LMO1$Gene)){
  
  x<-str_split(LMO2TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TAL2LMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMO2TAL2LMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TAL2LMO1_Targeting5[order(comparison_GSE26713_LMO2TAL2LMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_LMO2TAL2LMO1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO2TLX1$Gene)){
  
  x<-str_split(LMO2TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMO2TLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX1_Targeting5[order(comparison_GSE26713_LMO2TLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_LMO2TLX1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMO2TLX3$Gene)){
  
  x<-str_split(LMO2TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMO2TLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX3_Targeting5[order(comparison_GSE26713_LMO2TLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_LMO2TLX3_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOBM$Gene)){
  
  x<-str_split(LMOBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOBM_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMOBM_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOBM_Targeting5[order(comparison_GSE26713_LMOBM_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_LMOBM_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTAL1$Gene)){
  
  x<-str_split(LMOTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMOTAL1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL1_Targeting5[order(comparison_GSE26713_LMOTAL1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_LMOTAL1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTAL2$Gene)){
  
  x<-str_split(LMOTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL2_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMOTAL2_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL2_Targeting5[order(comparison_GSE26713_LMOTAL2_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_LMOTAL2_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTAL2LMO1$Gene)){
  
  x<-str_split(LMOTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL2LMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMOTAL2LMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL2LMO1_Targeting5[order(comparison_GSE26713_LMOTAL2LMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_LMOTAL2LMO1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTLX1$Gene)){
  
  x<-str_split(LMOTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMOTLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTLX1_Targeting5[order(comparison_GSE26713_LMOTLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTLX1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(LMOTLX3$Gene)){
  
  x<-str_split(LMOTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_LMOTLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTLX3_Targeting5[order(comparison_GSE26713_LMOTLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTLX3_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_HOXALMO2_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Targeting5[order(comparison_GSE26713_HOXALMO2_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXALMO2_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MYBTLX1$Gene)){
  
  x<-str_split(MYBTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_MYBTLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_MYBTLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

Nulo, bien..

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_MYBTLX1_Targeting5[order(comparison_GSE26713_MYBTLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_MYBTLX1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(MYBTLX3$Gene)){
  
  x<-str_split(MYBTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_MYBTLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_MYBTLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_MYBTLX3_Targeting5[order(comparison_GSE26713_MYBTLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_MYBTLX3_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL1TAL2LMO1$Gene)){
  
  x<-str_split(TAL1TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL1TAL2LMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TAL1TAL2LMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL1TAL2LMO1_Targeting5[order(comparison_GSE26713_TAL1TAL2LMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TAL1TAL2LMO1_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL1TLX1$Gene)){
  
  x<-str_split(TAL1TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL1TLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TAL1TLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL1TLX1_Targeting5[order(comparison_GSE26713_TAL1TLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/comparison_GSE26713_TAL1TLX1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL1TLX3$Gene)){
  
  x<-str_split(TAL1TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL1TLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TAL1TLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```


```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL1TLX3_Targeting5[order(comparison_GSE26713_TAL1TLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TAL1TLX3_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TAL2LMO1$Gene)){
  
  x<-str_split(TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL2LMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TAL2LMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL2LMO1_Targeting5[order(comparison_GSE26713_TAL2LMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TAL2LMO1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALBM$Gene)){
  
  x<-str_split(TALBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALBM_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TALBM_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALBM_Targeting5[order(comparison_GSE26713_TALBM_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TALBM_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALMYB$Gene)){
  
  x<-str_split(TALMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALMYB_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TALMYB_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALMYB_Targeting5[order(comparison_GSE26713_TALMYB_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TALMYB_Targeting5.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTAL2LMO1$Gene)){
  
  x<-str_split(TALTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTAL2LMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TALTAL2LMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTAL2LMO1_Targeting5[order(comparison_GSE26713_TALTAL2LMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TALTAL2LMO1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTLX$Gene)){
  
  x<-str_split(TALTLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TALTLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLX_Targeting5[order(comparison_GSE26713_TALTLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/comparison_GSE26713_TALTLX_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTLX1$Gene)){
  
  x<-str_split(TALTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLX1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TALTLX1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLX1_Targeting5[order(comparison_GSE26713_TALTLX1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TALTLX1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTLX3$Gene)){
  
  x<-str_split(TALTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLX3_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TALTLX3_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLX3_Targeting5[order(comparison_GSE26713_TALTLX3_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TALTLX3_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TALTLXL$Gene)){
  
  x<-str_split(TALTLXL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLXL_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TALTLXL_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLXL_Targeting5[order(comparison_GSE26713_TALTLXL_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TALTLXL_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLX$Gene)){
  
  x<-str_split(TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLX_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TLX_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLX_Targeting5[order(comparison_GSE26713_TLX_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TLX_Targeting5.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXBM$Gene)){
  
  x<-str_split(TLXBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXBM_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TLXBM_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXBM_Targeting5[order(comparison_GSE26713_TLXBM_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/comparison_GSE26713_TLXBM_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXMYB$Gene)){
  
  x<-str_split(TLXMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXMYB_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TLXMYB_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXMYB_Targeting5[order(comparison_GSE26713_TLXMYB_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TLXMYB_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXTAL1$Gene)){
  
  x<-str_split(TLXTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXTAL1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TLXTAL1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXTAL1_Targeting5[order(comparison_GSE26713_TLXTAL1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TLXTAL1_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXTAL2$Gene)){
  
  x<-str_split(TLXTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXTAL2_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TLXTAL2_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXTAL2_Targeting5[order(comparison_GSE26713_TLXTAL2_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TLXTAL2_Targeting5.xlsx", row.names = FALSE)
```


```{r, echo=TRUE, warning=TRUE, error=TRUE, include=TRUE}
library(stringr)
library(dplyr)
nombres_HGNC<-c()
descript<-c()
n_col_genes=c()

emp_vec=c()

emp_vec2=c()



for (y in 1:length(TLXTAL2LMO1$Gene)){
  
  x<-str_split(TLXTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"
    
    for (nose2 in emp_vec){
      
      for (i in 1:length(Targeting5$gene)){
        
        b<-Targeting5$gene[i]    # [1] "10180" por ejemplo, para i=2
        
        if (!is.na(nose2) && !is.na(b) && !is.null(b) && nose2==b){
          
          
          nombres_HGNC<-append(nombres_HGNC, b)
          
          if(!is.na(Targeting5$gene_description[i])){descript<-append(descript, Targeting5$gene_description[i])}
          else{descript<-append(descript, NA)}
          
          n_col_genes<-append(n_col_genes, i)
          
        }}
      
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_HGNC)==length(descript)
length(descript)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXTAL2LMO1_Targeting5<-data.frame(n_col_genes, nombres_HGNC, descript)
colnames(comparison_GSE26713_TLXTAL2LMO1_Targeting5)=c('Numero_col_Targeting2',
                                          'HGNC',
                                          'Descripcion_gene')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXTAL2LMO1_Targeting5[order(comparison_GSE26713_TLXTAL2LMO1_Targeting5$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Descripcion_gene[i]==
      yoquese$Descripcion_gene[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE14618/comparison_GSE26713_TLXTAL2LMO1_Targeting5.xlsx", row.names = FALSE)
```



































## Large_scale





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(BMHOXA$Gene)){
  
  x<-str_split(BMHOXA$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMHOXA_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_BMHOXA_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMHOXA_Large[order(comparison_GSE26713_BMHOXA_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMHOXA_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(BMLMO1$Gene)){
  
  x<-str_split(BMLMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_BMLMO1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO1_Large[order(comparison_GSE26713_BMLMO1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMLMO1_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(BMLMO2$Gene)){
  
  x<-str_split(BMLMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMLMO2_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_BMLMO2_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMLMO2_Large[order(comparison_GSE26713_BMLMO2_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMLMO2_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(BMMYB$Gene)){
  
  x<-str_split(BMMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMMYB_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_BMMYB_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMMYB_Large[order(comparison_GSE26713_BMMYB_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMMYB_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(BMTAL1$Gene)){
  
  x<-str_split(BMTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_BMTAL1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL1_Large[order(comparison_GSE26713_BMTAL1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL1_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(BMTAL2$Gene)){
  
  x<-str_split(BMTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_BMTAL2_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2_Large[order(comparison_GSE26713_BMTAL2_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL2_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(BMTAL2LMO1$Gene)){
  
  x<-str_split(BMTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTAL2LMO1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_BMTAL2LMO1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTAL2LMO1_Large[order(comparison_GSE26713_BMTAL2LMO1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTAL2LMO1_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(BMTLX1$Gene)){
  
  x<-str_split(BMTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_BMTLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX1_Large[order(comparison_GSE26713_BMTLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTLX1_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(BMTLX3$Gene)){
  
  x<-str_split(BMTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_BMTLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_BMTLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_BMTLX3_Large[order(comparison_GSE26713_BMTLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_BMTLX3_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_HOXALMO2_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Large[order(comparison_GSE26713_HOXALMO2_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXALMO2_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXATAL1$Gene)){
  
  x<-str_split(HOXATAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_HOXATAL1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL1_Large[order(comparison_GSE26713_HOXATAL1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATAL1_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXATAL2LMO1$Gene)){
  
  x<-str_split(HOXATAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATAL2LMO1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_HOXATAL2LMO1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATAL2LMO1_Large[order(comparison_GSE26713_HOXATAL2LMO1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATAL2LMO1_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXATLX1$Gene)){
  
  x<-str_split(HOXATLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_HOXATLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX1_Large[order(comparison_GSE26713_HOXATLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATLX1_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXATLX3$Gene)){
  
  x<-str_split(HOXATLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXATLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_HOXATLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXATLX3_Large[order(comparison_GSE26713_HOXATLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXATLX3_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMO1LMO2MYB$Gene)){
  
  x<-str_split(LMO1LMO2MYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO1LMO2MYB_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMO1LMO2MYB_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO1LMO2MYB_Large[order(comparison_GSE26713_LMO1LMO2MYB_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO1LMO2MYB_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMO2TAL2LMO1$Gene)){
  
  x<-str_split(LMO2TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TAL2LMO1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMO2TAL2LMO1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TAL2LMO1_Large[order(comparison_GSE26713_LMO2TAL2LMO1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TAL2LMO1_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMO2TLX1$Gene)){
  
  x<-str_split(LMO2TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMO2TLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX1_Large[order(comparison_GSE26713_LMO2TLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TLX1_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMO2TLX3$Gene)){
  
  x<-str_split(LMO2TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMO2TLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMO2TLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMO2TLX3_Large[order(comparison_GSE26713_LMO2TLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMO2TLX3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMOBM$Gene)){
  
  x<-str_split(LMOBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOBM_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMOBM_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOBM_Large[order(comparison_GSE26713_LMOBM_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOBM_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMOTAL1$Gene)){
  
  x<-str_split(LMOTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMOTAL1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL1_Large[order(comparison_GSE26713_LMOTAL1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL1_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMOTAL2$Gene)){
  
  x<-str_split(LMOTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL2_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMOTAL2_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL2_Large[order(comparison_GSE26713_LMOTAL2_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL2_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMOTAL2LMO1$Gene)){
  
  x<-str_split(LMOTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTAL2LMO1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMOTAL2LMO1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTAL2LMO1_Large[order(comparison_GSE26713_LMOTAL2LMO1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTAL2LMO1_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMOTLX1$Gene)){
  
  x<-str_split(LMOTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMOTLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTLX1_Large[order(comparison_GSE26713_LMOTLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTLX1_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(LMOTLX3$Gene)){
  
  x<-str_split(LMOTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_LMOTLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_LMOTLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_LMOTLX3_Large[order(comparison_GSE26713_LMOTLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_LMOTLX3_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(HOXALMO2$Gene)){
  
  x<-str_split(HOXALMO2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_HOXALMO2_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_HOXALMO2_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_HOXALMO2_Large[order(comparison_GSE26713_HOXALMO2_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_HOXALMO2_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(MYBTLX1$Gene)){
  
  x<-str_split(MYBTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_MYBTLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_MYBTLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Bien, nulo...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_MYBTLX1_Large[order(comparison_GSE26713_MYBTLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_MYBTLX1_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(MYBTLX3$Gene)){
  
  x<-str_split(MYBTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_MYBTLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_MYBTLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

Bien, nulo...

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_MYBTLX3_Large[order(comparison_GSE26713_MYBTLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_MYBTLX3_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TAL1TAL2LMO1$Gene)){
  
  x<-str_split(TAL1TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL1TAL2LMO1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TAL1TAL2LMO1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL1TAL2LMO1_Large[order(comparison_GSE26713_TAL1TAL2LMO1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TAL1TAL2LMO1_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TAL1TLX1$Gene)){
  
  x<-str_split(TAL1TLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL1TLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TAL1TLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL1TLX1_Large[order(comparison_GSE26713_TAL1TLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TAL1TLX1_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TAL1TLX3$Gene)){
  
  x<-str_split(TAL1TLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL1TLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TAL1TLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL1TLX3_Large[order(comparison_GSE26713_TAL1TLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TAL1TLX3_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TAL2LMO1$Gene)){
  
  x<-str_split(TAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TAL2LMO1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TAL2LMO1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TAL2LMO1_Large[order(comparison_GSE26713_TAL2LMO1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TAL2LMO1_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TALBM$Gene)){
  
  x<-str_split(TALBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALBM_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TALBM_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALBM_Large[order(comparison_GSE26713_TALBM_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALBM_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TALMYB$Gene)){
  
  x<-str_split(TALMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALMYB_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TALMYB_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALMYB_Large[order(comparison_GSE26713_TALMYB_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALMYB_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TALTAL2LMO1$Gene)){
  
  x<-str_split(TALTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTAL2LMO1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TALTAL2LMO1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTAL2LMO1_Large[order(comparison_GSE26713_TALTAL2LMO1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTAL2LMO1_Large.xlsx", row.names = FALSE)
```




```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TALTLX$Gene)){
  
  x<-str_split(TALTLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TALTLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLX_Large[order(comparison_GSE26713_TALTLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTLX_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TALTLX1$Gene)){
  
  x<-str_split(TALTLX1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLX1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TALTLX1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLX1_Large[order(comparison_GSE26713_TALTLX1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTLX1_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TALTLX3$Gene)){
  
  x<-str_split(TALTLX3$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLX3_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TALTLX3_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLX3_Large[order(comparison_GSE26713_TALTLX3_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTLX3_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TALTLXL$Gene)){
  
  x<-str_split(TALTLXL$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TALTLXL_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TALTLXL_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TALTLXL_Large[order(comparison_GSE26713_TALTLXL_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TALTLXL_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TLX$Gene)){
  
  x<-str_split(TLX$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLX_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TLX_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLX_Large[order(comparison_GSE26713_TLX_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLX_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TLXBM$Gene)){
  
  x<-str_split(TLXBM$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXBM_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TLXBM_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXBM_Large[order(comparison_GSE26713_TLXBM_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXBM_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TLXMYB$Gene)){
  
  x<-str_split(TLXMYB$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXMYB_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TLXMYB_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXMYB_Large[order(comparison_GSE26713_TLXMYB_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXMYB_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TLXTAL1$Gene)){
  
  x<-str_split(TLXTAL1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXTAL1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TLXTAL1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXTAL1_Large[order(comparison_GSE26713_TLXTAL1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXTAL1_Large.xlsx", row.names = FALSE)
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TLXTAL2$Gene)){
  
  x<-str_split(TLXTAL2$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXTAL2_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TLXTAL2_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXTAL2_Large[order(comparison_GSE26713_TLXTAL2_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXTAL2_Large.xlsx", row.names = FALSE)
```





```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(stringr)
library(dplyr)
n_col_genes<-c()
typpo<-c()
nombres_genes<-c()

for (y in 1:length(TLXTAL2LMO1$Gene)){
  
  x<-str_split(TLXTAL2LMO1$Gene[y], pattern = "/") # muchas cosas. [[1]]
                                                         # [1] "PLCG1" "LAT"   "LCK"   "ZAP70" "CD3E"  "PLCG2" "CD3D"  "VASP" 
                                                         # [9] "CD247" "GRAP2" "CD3G"  "FYB1"  "ITK"  
  for (z in 1:length(x)){
    
    emp_vec<-x[[z]]                                      # [1] "ENO2"  [2] loquesea etc
    
    for (nose2 in emp_vec){                              # [1] "ENO2"

  for (i in 1:length(Large_scale$HGNC_symbol)){
    
    b<-Large_scale$HGNC_symbol[i]    # [1] "10180" por ejemplo, para i=2
    
    if (nose2==b && !is.na(nose2) && !is.na(b)){
      
      nombres_genes<-append(nombres_genes, b)
      
      typpo<-append(typpo, Large_scale$type[i])
      
      n_col_genes<-append(n_col_genes, i)
          
        }
      }
  }}}
  
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
length(nombres_genes)==length(typpo)

length(typpo)==length(n_col_genes)


```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
comparison_GSE26713_TLXTAL2LMO1_Large<-data.frame(n_col_genes, nombres_genes, typpo)
colnames(comparison_GSE26713_TLXTAL2LMO1_Large)=c('Numero_col_Large',
                                          'HGNC', 'Splicing_factor')
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
filas_eliminar<-c()

yoquese<-comparison_GSE26713_TLXTAL2LMO1_Large[order(comparison_GSE26713_TLXTAL2LMO1_Large$HGNC),]

for (i in 1:(dim(yoquese)[1]-1)){ # menos al final
  
  if (yoquese$HGNC[i]==
      yoquese$HGNC[i+1] && 
     yoquese$Splicing_factor[i]==
      yoquese$Splicing_factor[i+1] ){
    filas_eliminar<-append(filas_eliminar,i)
  
  
}}
```

```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
yoquese<-yoquese[-filas_eliminar,]
```



```{r, echo=TRUE, warning=TRUE, message=TRUE, include=TRUE}
library(xlsx)
xlsx::write.xlsx(x = yoquese, file = "C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/comparison_GSE26713_TLXTAL2LMO1_Large.xlsx", row.names = FALSE)
```

