---
title: "Ejemplo interrogracion"
author: "Cristina Muntañola"
date: "7/5/2022"
output: html_document
---

En este archivo explicaremos cómo hemos determinado y cómo buscar los genes relacionados con ciertas funciones o rutas metabólicas de entre comparaciones de cada serie GSE. Generaremos ejemplos que usaremos posteriormente, y también indicaremos cómo generar data frames para guardar en archivos Excel y poder revisar las comparaciones cuando se necesite. Primeramente cargaremos las bases de datos resultado de nuestros análisis, y luego las bases de datos relacionadas entre nuestras comparaciones y los tres artículos:


Huang, R., Han, M., Meng, L., & Chen, X. (2018). Transcriptome-wide discovery of coding and noncoding RNA-binding proteins. Proceedings of the National Academy of Sciences, 115(17), E3879-E3887.

Sebestyén, E., Singh, B., Miñana, B., Pagès, A., Mateo, F., Pujana, M. A., ... & Eyras, E. (2016). Large-scale analysis of genome and transcriptome alterations in multiple tumors unveils novel cancer-relevant splicing networks. Genome research, 26(6), 732-744.

Wang, E., Lu, S. X., Pastore, A., Chen, X., Imig, J., Lee, S. C. W., ... & Aifantis, I. (2019). Targeting an RNA-binding protein network in acute myeloid leukemia. Cancer cell, 35(3), 369-384.

# GSE8879

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
Resultado_GSE8879<-readxl::read_excel('datasets/GSE8879/Resultado_GSE8879.xlsx')
GSE8879_Large<- readxl::read_excel('datasets/GSE8879/comparison_GSE8879_Large.xlsx')
GSE8879_Targeting2<- readxl::read_excel('datasets/GSE8879/comparison_GSE8879_Targeting2.xlsx')
GSE8879_Targeting5<- readxl::read_excel('datasets/GSE8879/comparison_GSE8879_Targeting5.xlsx')
GSE8879_Transcriptome<- readxl::read_excel('datasets/GSE8879/comparison_GSE8879_Transcriptome.xlsx')
```

Buscaremos genes en cuya descripción aparezca que están relacionados con el ARN o con eventos de splicing. En el primer vector (i) se guardarán las filas que poseen el valor RNA, y en j y k las que poseen la palabra splicing.

En caso de querer buscar una palabra, correr solamente los dos primeros vectores, y no añadir nada a un vector j.

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_GSE8879$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_GSE8879$Biological_Description, value = FALSE) # splicing con minúscula
k<-grep("Splicing", Resultado_GSE8879$Biological_Description, value = FALSE) # splicing con mayúscula

m<-grep("binding protein", Resultado_GSE8879$Biological_Description, value = FALSE) # 
n<-grep("Binding protein", Resultado_GSE8879$Biological_Description, value = FALSE) # 


l=grep("RBP", Resultado_GSE8879$Biological_Description, value = FALSE) # VACÍO, no se usará
j=append(j,k, m)
```


En caso de buscar solo una palabra no hacer este paso. Ahora añadiremos los valores de j a i que no estén repetidos. Con esto tendremos finalmente el número de las filas que indiquen genes con relación al ARN o a eventos de splicing.

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
}}

i=i[order(i)]
```

Ahora crearemos diferentes vectores que contengan la información de cada fila del dataset Resultado_GSE8879 que tengan relación con RNA o splicing.

```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_GSE8879$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_GSE8879$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_GSE8879$Gene[i[long_i]])
  }

```

Finalmente creamos un dataset a partir de los vectores, con los mismos nombres de columna que Resultado_GSE8879.

```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```

Si se quiere generar un documento tipo Excel, se puede correr el siguiente código cambiando la ruta.

```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/RNA_splicing_GSE8879.xlsx',
  col.names = TRUE)
```


Se procederá igual para todos los demás resultados de todas las comparaciones de los GSE.


## Transcriptome

```{r, include=TRUE, message=TRUE, echo=TRUE}

clas_rbps<-grep("RBP", GSE8879_Transcriptome$Clasificacion_RBPs, value = TRUE)
i<-grep("RBP", GSE8879_Transcriptome$Clasificacion_RBPs, value = FALSE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Prot<-c()
Clas_RBP<-c()
Fun_Bio<-c()

for (long_i in 1:length(i)){
  Prot<-append(Prot, GSE8879_Transcriptome$Proteina[i[long_i]])
  Clas_RBP<-append(Clas_RBP, GSE8879_Transcriptome$Clasificacion_RBPs[i[long_i]])
  Fun_Bio<-append(Fun_Bio, GSE8879_Transcriptome$Funcion_Biologica[i[long_i]])
  }

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
RBP_Transcriptome_GSE8879<-data.frame('Protein'=Prot, 'Clasificacion_RBPs'=Clas_RBP, 'Biological_Function'=Fun_Bio)

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RBP_Transcriptome_GSE8879, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/RBP_Transcriptome_GSE8879.xlsx',
  col.names = TRUE)
```

## Targeting2

```{r, include=TRUE, message=TRUE, echo=TRUE}

desc1<-grep("RNA", GSE8879_Targeting2$Descripcion, value = TRUE)
prote1<-grep("RNA", GSE8879_Targeting2$Proteína, value = TRUE)
i1=grep("RNA", GSE8879_Targeting2$Descripcion, value = FALSE)
i2=grep("RNA", GSE8879_Targeting2$Proteína, value = FALSE)

desc2<-grep("splicing", GSE8879_Targeting2$Descripcion, value = TRUE)
prote2<-grep("splicing", GSE8879_Targeting2$Proteína, value = TRUE)
j1=grep("splicing", GSE8879_Targeting2$Descripcion, value = FALSE)
j2=grep("splicing", GSE8879_Targeting2$Proteína, value = FALSE)

desc3<-grep("Splicing", GSE8879_Targeting2$Descripcion, value = TRUE)
prote3<-grep("Splicing", GSE8879_Targeting2$Proteína, value = TRUE)
k1=grep("Splicing", GSE8879_Targeting2$Descripcion, value = FALSE)
k2=grep("Splicing", GSE8879_Targeting2$Proteína, value = FALSE)

desc4<-grep("RBP", GSE8879_Targeting2$Descripcion, value = TRUE)
prote4<-grep("RBP", GSE8879_Targeting2$Proteína, value = TRUE)
l1=grep("RBP", GSE8879_Targeting2$Descripcion, value = FALSE)
l2=grep("RBP", GSE8879_Targeting2$Proteína, value = FALSE)


```

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if(length(i2)>0){
for (nvalj in 1:length(i2)){
  if ((i2 %in% i1)[nvalj]==FALSE){
    i1=append(i1,i2[nvalj])
  }}}

if(length(j1)>0){
for (nvalj in 1:length(j1)){
  if ((j1 %in% i1)[nvalj]==FALSE){
    i1=append(i1,j1[nvalj])
  }}}


if(length(j2)>0){
for (nvalj in 1:length(j2)){
  if ((j2 %in% i1)[nvalj]==FALSE){
    i1=append(i1,j2[nvalj])
  }}}


if(length(k1)>0){
for (nvalj in 1:length(k1)){
  if ((k1 %in% i1)[nvalj]==FALSE){
    i1=append(i1,k1[nvalj])
}}}


if(length(k2)>0){
for (nvalj in 1:length(k2)){
  if ((k2 %in% i1)[nvalj]==FALSE){
    i1=append(i1,k2[nvalj])
  }}}


if(length(l1)>0){
for (nvalj in 1:length(l1)){
  if ((l1 %in% i1)[nvalj]==FALSE){
    i1=append(i1,l1[nvalj])
  }}}


if(length(l2)>0){
for (nvalj in 1:length(l2)){
  if ((l2 %in% i1)[nvalj]==FALSE){
    i1=append(i1,l2[nvalj])
}}}
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
HGN<-c()
Desc<-c()
Prot<-c()

if(length(i1)>0){
for (long_i in 1:length(i1)){
  HGN<-append(HGN, GSE8879_Targeting2$HGNC[i1[long_i]])
  Desc<-append(Clas_RBP, GSE8879_Targeting2$Descripcion[i1[long_i]])
  Prot<-append(Prot, GSE8879_Targeting2$Proteína[i1[long_i]])
}
  RNAsplicn_Targeting2_GSE8879<-data.frame('HGNC'=HGN, 'Descripcion'=Desc, 'Proteína'=Prot)
  }

```


RNAsplicn_Targeting2_GSE8879 nulo.


## Targeting5


```{r, include=TRUE, message=TRUE, echo=TRUE}

HG<-grep("RNA", GSE8879_Targeting5$HGNC, value = TRUE)
Des<-grep("RNA", GSE8879_Targeting5$Descripcion_gene, value = TRUE)
i1=grep("RNA", GSE8879_Targeting5$HGNC, value = FALSE)
i2=grep("RNA", GSE8879_Targeting5$Descripcion_gene, value = FALSE)

HG<-grep("splicing", GSE8879_Targeting5$HGNC, value = TRUE)
Des<-grep("splicing", GSE8879_Targeting5$Descripcion_gene, value = TRUE)
j1=grep("splicing", GSE8879_Targeting5$HGNC, value = FALSE)
j2=grep("splicing", GSE8879_Targeting5$Descripcion_gene, value = FALSE)

HG<-grep("Splicing", GSE8879_Targeting5$HGNC, value = TRUE)
Des<-grep("Splicing", GSE8879_Targeting5$Descripcion_gene, value = TRUE)
k1=grep("Splicing", GSE8879_Targeting5$HGNC, value = FALSE)
k2=grep("Splicing", GSE8879_Targeting5$Descripcion_gene, value = FALSE)

HG<-grep("RBP", GSE8879_Targeting5$HGNC, value = TRUE)
Des<-grep("RBP", GSE8879_Targeting5$Descripcion_gene, value = TRUE)
l1=grep("RBP", GSE8879_Targeting5$HGNC, value = FALSE)
l2=grep("RBP", GSE8879_Targeting5$Descripcion_gene, value = FALSE)


```

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if(length(i2)>0){
for (nvalj in 1:length(i2)){
  if ((i2 %in% i1)[nvalj]==FALSE){
    i1=append(i1,i2[nvalj])
  }}}

if(length(j1)>0){
for (nvalj in 1:length(j1)){
  if ((j1 %in% i1)[nvalj]==FALSE){
    i1=append(i1,j1[nvalj])
  }}}


if(length(j2)>0){
for (nvalj in 1:length(j2)){
  if ((j2 %in% i1)[nvalj]==FALSE){
    i1=append(i1,j2[nvalj])
  }}}


if(length(k1)>0){
for (nvalj in 1:length(k1)){
  if ((k1 %in% i1)[nvalj]==FALSE){
    i1=append(i1,k1[nvalj])
}}}


if(length(k2)>0){
for (nvalj in 1:length(k2)){
  if ((k2 %in% i1)[nvalj]==FALSE){
    i1=append(i1,k2[nvalj])
  }}}


if(length(l1)>0){
for (nvalj in 1:length(l1)){
  if ((l1 %in% i1)[nvalj]==FALSE){
    i1=append(i1,l1[nvalj])
  }}}


if(length(l2)>0){
for (nvalj in 1:length(l2)){
  if ((l2 %in% i1)[nvalj]==FALSE){
    i1=append(i1,l2[nvalj])
  }}}

i1=i1[order(i1)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
HGN<-c()
Desc<-c()

if(length(i1)>0){
for (long_i in 1:length(i1)){
  HGN<-append(HGN, GSE8879_Targeting5$HGNC[i1[long_i]])
  Desc<-append(Desc, GSE8879_Targeting5$Descripcion_gene[i1[long_i]])
}
  RNAsplicn_Targeting5_GSE8879<-data.frame('HGNC'=HGN, 'Descripcion'=Desc)
  }

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNAsplicn_Targeting5_GSE8879, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/RNAsplicn_Targeting5_GSE8879.xlsx',
  col.names = TRUE)
```



## Large


```{r, include=TRUE, message=TRUE, echo=TRUE}

GSE8879_Large_rem<-GSE8879_Large

i1=grep("RBP", GSE8879_Large_rem$Splicing_factor, value = FALSE)
i2=grep("SF", GSE8879_Large_rem$Splicing_factor, value = FALSE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if(length(i2)>0){
for (nvalj in 1:length(i2)){
  if ((i2 %in% i1)[nvalj]==FALSE){
    i1=append(i1,i2[nvalj])
  }}}



i1=i1[order(i1)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
GSE8879_Large_rem<-GSE8879_Large_rem[i1,]
```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")

if (table(GSE8879_Large_rem!=GSE8879_Large)){ # son diferentes
write.xlsx(GSE8879_Large_rem, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE8879/Large_rem_GSE8879.xlsx',
  col.names = TRUE)}
```


# GSE10609

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
Resultado_HOX<-readxl::read_excel('datasets/GSE10609/Resultado_HOX.xlsx')
Resultado_HOX11HOX11L2HOXA<-readxl::read_excel('datasets/GSE10609/Resultado_HOX11HOX11L2HOXA.xlsx')
Resultado_HOX11HOX11L2LMO2<-readxl::read_excel('datasets/GSE10609/Resultado_HOX11HOX11L2LMO2.xlsx')
Resultado_HOX11HOX11L2TAL1<-readxl::read_excel('datasets/GSE10609/Resultado_HOX11HOX11L2TAL1.xlsx')
Resultado_HOX11HOXA<-readxl::read_excel('datasets/GSE10609/Resultado_HOX11HOXA.xlsx')
Resultado_HOX11L2HOXA<-readxl::read_excel('datasets/GSE10609/Resultado_HOX11L2HOXA.xlsx')
Resultado_HOX11L2LMO2<-readxl::read_excel('datasets/GSE10609/Resultado_HOX11L2LMO2.xlsx')
Resultado_HOX11L2TAL1<-readxl::read_excel('datasets/GSE10609/Resultado_HOX11L2TAL1.xlsx')
Resultado_HOX11LMO2<-readxl::read_excel('datasets/GSE10609/Resultado_HOX11LMO2.xlsx')
Resultado_HOX11TAL1<-readxl::read_excel('datasets/GSE10609/Resultado_HOX11TAL1.xlsx')
Resultado_HOXATAL1<-readxl::read_excel('datasets/GSE10609/Resultado_HOXATAL1.xlsx')


HOX_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX_Large.xlsx')
HOX_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX_Transcriptome.xlsx')
HOX_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX_Targeting2.xlsx')
HOX_Targeting5<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX_Targeting5.xlsx')

HOX11HOX11L2HOXA_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2HOXA_Large.xlsx')
HOX11HOX11L2HOXA_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2HOXA_Targeting2.xlsx')
HOX_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2HOXA_Targeting5.xlsx')
HOX11HOX11L2HOXA_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2HOXA_Transcriptome.xlsx')



HOX11HOX11L2LMO2_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Large.xlsx')
HOX11HOX11L2LMO2_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Targeting2.xlsx')
HOX11HOX11L2LMO2_Targeting5<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Targeting5.xlsx')
HOX11HOX11L2LMO2_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2LMO2_Transcriptome.xlsx')



HOX11HOX11L2TAL1_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2TAL1_Large.xlsx')
HOX11HOX11L2TAL1_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2TAL1_Targeting2.xlsx')
HOX11HOX11L2TAL1_Targeting5<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOX11L2TAL1_Targeting5.xlsx')

HOX11HOXA_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOXA_Large.xlsx')
HOX11HOXA_Targeting5<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOXA_Targeting5.xlsx')
HOX11HOXA_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11HOXA_Transcriptome.xlsx')


HOX11L2HOXA_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2HOXA_Large.xlsx')
HOX11L2HOXA_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2HOXA_Targeting2.xlsx')
HOX11L2HOXA_Targeting5<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2HOXA_Targeting5.xlsx')
HOX11L2HOXA_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2HOXA_Transcriptome.xlsx')



HOX11L2LMO2_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2LMO2_Large.xlsx')
HOX11L2LMO2_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2LMO2_Targeting2.xlsx')
HOX11L2LMO2_Targeting5<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2LMO2_Targeting5.xlsx')
HOX11L2LMO2_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2LMO2_Transcriptome.xlsx')



HOX11L2TAL1_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2TAL1_Large.xlsx')
HOX11L2TAL1_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2TAL1_Targeting2.xlsx')
HOX11L2TAL1_Targeting5<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2TAL1_Targeting5.xlsx')
HOX11L2TAL1_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11L2TAL1_Transcriptome.xlsx')



HOX11LMO2_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11LMO2_Large.xlsx')
HOX11LMO2_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11LMO2_Targeting2.xlsx')
HOX11LMO2_Targeting5<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11LMO2_Targeting5.xlsx')
HOX11LMO2_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11LMO2_Transcriptome.xlsx')



HOX11TAL1_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11TAL1_Large.xlsx')
HOX11TAL1_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11TAL1_Targeting2.xlsx')
HOX11TAL1_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOX11TAL1_Transcriptome.xlsx')


HOXATAL1_Large<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOXATAL1_Large.xlsx')
HOXATAL1_Targeting2<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOXATAL1_Targeting2.xlsx')
HOXATAL1_Transcriptome<- readxl::read_excel('datasets/GSE10609/comparison_GSE10609_HOXATAL1_Transcriptome.xlsx')

```

## Resultados sin comparar

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX$Biological_Description, value = FALSE) 


m<-grep("RBP", Resultado_HOX$Biological_Description, value = FALSE) 
n<-grep("binding protein", Resultado_HOX$Biological_Description, value = FALSE) 



l=grep("RBP", Resultado_HOX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}


if (length(m)>0){
for (nvalj in 1:length(m)){
  if ((m %in% i)[nvalj]==FALSE){
    i=append(i,m[nvalj])
  }}}

if (length(n)>0){
for (nvalj in 1:length(n)){
  if ((n %in% i)[nvalj]==FALSE){
    i=append(i,n[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOXATAL1_GSE10609.xlsx',
  col.names = TRUE)
```




```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX11HOX11L2HOXA$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX11HOX11L2HOXA$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX11HOX11L2HOXA$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOX11HOX11L2HOXA$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX11HOX11L2HOXA$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX11HOX11L2HOXA$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX11HOX11L2HOXA$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOX11HOX11L2HOXA_GSE10609.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX11HOX11L2LMO2$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX11HOX11L2LMO2$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX11HOX11L2LMO2$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOX11HOX11L2LMO2$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX11HOX11L2LMO2$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX11HOX11L2LMO2$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX11HOX11L2LMO2$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOX11HOX11L2LMO2_GSE10609.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX11HOX11L2TAL1$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX11HOX11L2TAL1$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX11HOX11L2TAL1$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOX11HOX11L2TAL1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX11HOX11L2TAL1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX11HOX11L2TAL1$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX11HOX11L2TAL1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOX11HOX11L2TAL1_GSE10609.xlsx',
  col.names = TRUE)
```




```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX11HOXA$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX11HOXA$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX11HOXA$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOX11HOXA$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX11HOXA$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX11HOXA$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX11HOXA$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOX11HOXA_GSE10609.xlsx',
  col.names = TRUE)
```




```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX11L2HOXA$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX11L2HOXA$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX11L2HOXA$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOX11L2HOXA$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX11L2HOXA$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX11L2HOXA$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX11L2HOXA$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOX11L2HOXA_GSE10609.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX11L2LMO2$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX11L2LMO2$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX11L2LMO2$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOX11L2LMO2$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX11L2LMO2$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX11L2LMO2$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX11L2LMO2$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOX11L2LMO2_GSE10609.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX11L2TAL1$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX11L2TAL1$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX11L2TAL1$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOX11L2TAL1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX11L2TAL1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX11L2TAL1$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX11L2TAL1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOX11L2TAL1_GSE10609.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX11LMO2$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX11LMO2$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX11LMO2$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOX11LMO2$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX11LMO2$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX11LMO2$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX11LMO2$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOX11LMO2_GSE10609.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOX11TAL1$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOX11TAL1$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOX11TAL1$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOX11TAL1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOX11TAL1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOX11TAL1$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOX11TAL1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOX11TAL1_GSE10609.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", Resultado_HOXATAL1$Biological_Description, value = FALSE)
j<-grep("splicing", Resultado_HOXATAL1$Biological_Description, value = FALSE) 
k<-grep("Splicing", Resultado_HOXATAL1$Biological_Description, value = FALSE) 
l=grep("RBP", Resultado_HOXATAL1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, Resultado_HOXATAL1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, Resultado_HOXATAL1$Biological_Description[i[long_i]])
  Gen<-append(Gen, Resultado_HOXATAL1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_HOXATAL1_GSE10609.xlsx',
  col.names = TRUE)
```





# GSE14618

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
GPL570_CNR<-readxl::read_excel('datasets/GSE14618/Resultado_GPL570_CNR.xlsx')
GPL570_FNR<-readxl::read_excel('datasets/GSE14618/Resultado_GPL570_FNR.xlsx')

GPL96_CNR<-readxl::read_excel('datasets/GSE14618/Resultado_GPL96_CNR.xlsx')
GPL96_FNR<-readxl::read_excel('datasets/GSE14618/Resultado_GPL96_FNR.xlsx')



```

## Resultados sin comparar

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GPL570_CNR$Biological_Description, value = FALSE)
j<-grep("splicing", GPL570_CNR$Biological_Description, value = FALSE) 
k<-grep("Splicing", GPL570_CNR$Biological_Description, value = FALSE) 
l=grep("RBP", GPL570_CNR$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GPL570_CNR$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GPL570_CNR$Biological_Description[i[long_i]])
  Gen<-append(Gen, GPL570_CNR$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_GPL570_CNR.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GPL570_FNR$Biological_Description, value = FALSE)
j<-grep("splicing", GPL570_FNR$Biological_Description, value = FALSE) 
k<-grep("Splicing", GPL570_FNR$Biological_Description, value = FALSE) 
l=grep("RBP", GPL570_FNR$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GPL570_FNR$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GPL570_FNR$Biological_Description[i[long_i]])
  Gen<-append(Gen, GPL570_FNR$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_GPL570_FNR.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GPL96_CNR$Biological_Description, value = FALSE)
j<-grep("splicing", GPL96_CNR$Biological_Description, value = FALSE) 
k<-grep("Splicing", GPL96_CNR$Biological_Description, value = FALSE) 
l=grep("RBP", GPL96_CNR$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GPL96_CNR$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GPL96_CNR$Biological_Description[i[long_i]])
  Gen<-append(Gen, GPL96_CNR$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_GPL96_CNR.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GPL96_FNR$Biological_Description, value = FALSE)
j<-grep("splicing", GPL96_FNR$Biological_Description, value = FALSE) 
k<-grep("Splicing", GPL96_FNR$Biological_Description, value = FALSE) 
l=grep("RBP", GPL96_FNR$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GPL96_FNR$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GPL96_FNR$Biological_Description[i[long_i]])
  Gen<-append(Gen, GPL96_FNR$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE10609/RNA_splicing_GPL96_FNR.xlsx',
  col.names = TRUE)
```

# GSE26713

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
GSE26713_BMHOXA<-readxl::read_excel('datasets/GSE26713/Resultado_BMHOXA.xlsx')
GSE26713_BMLMO1<-readxl::read_excel('datasets/GSE26713/Resultado_BMLMO1.xlsx')

GSE26713_BMLMO2<-readxl::read_excel('datasets/GSE26713/Resultado_BMLMO2.xlsx')
GSE26713_BMMYB<-readxl::read_excel('datasets/GSE26713/Resultado_BMMYB.xlsx')

GSE26713_BMTAL1<-readxl::read_excel('datasets/GSE26713/Resultado_BMTAL1.xlsx')
GSE26713_BMTAL2<-readxl::read_excel('datasets/GSE26713/Resultado_BMTAL2.xlsx')

GSE26713_BMTAL2LMO1<-readxl::read_excel('datasets/GSE26713/Resultado_BMTAL2LMO1.xlsx')
GSE26713_BMTLX1<-readxl::read_excel('datasets/GSE26713/Resultado_BMTLX1.xlsx')


GSE26713_BMTLX3<-readxl::read_excel('datasets/GSE26713/Resultado_BMTLX3.xlsx')
GSE26713_HOXALMO2<-readxl::read_excel('datasets/GSE26713/Resultado_HOXALMO2.xlsx')

GSE26713_HOXATAL1<-readxl::read_excel('datasets/GSE26713/Resultado_HOXATAL1.xlsx')
GSE26713_HOXATAL2LMO1<-readxl::read_excel('datasets/GSE26713/Resultado_HOXATAL2LMO1.xlsx')


GSE26713_HOXATLX1<-readxl::read_excel('datasets/GSE26713/Resultado_HOXATLX1.xlsx')
GSE26713_HOXATLX3<-readxl::read_excel('datasets/GSE26713/Resultado_HOXATLX3.xlsx')

GSE26713_LMO1LMO2MYB<-readxl::read_excel('datasets/GSE26713/Resultado_LMO1LMO2MYB.xlsx')
GSE26713_LMO2TAL2LMO1<-readxl::read_excel('datasets/GSE26713/Resultado_LMO2TAL2LMO1.xlsx')


GSE26713_LMO2TLX1<-readxl::read_excel('datasets/GSE26713/Resultado_LMO2TLX1.xlsx')
GSE26713_LMO2TLX3<-readxl::read_excel('datasets/GSE26713/Resultado_LMO2TLX3.xlsx')

GSE26713_LMOBM<-readxl::read_excel('datasets/GSE26713/Resultado_LMOBM.xlsx')
GSE26713_LMOTAL1<-readxl::read_excel('datasets/GSE26713/Resultado_LMOTAL1.xlsx')


GSE26713_LMOTAL2<-readxl::read_excel('datasets/GSE26713/Resultado_LMOTAL2.xlsx') # vas po aqui
GSE26713_LMOTAL2LMO1<-readxl::read_excel('datasets/GSE26713/Resultado_LMOTAL2LMO1.xlsx')

GSE26713_LMOTLX1<-readxl::read_excel('datasets/GSE26713/Resultado_LMOTLX1.xlsx')
GSE26713_LMOTLX3<-readxl::read_excel('datasets/GSE26713/Resultado_LMOTLX3.xlsx')


GSE26713_MYBTAL2<-readxl::read_excel('datasets/GSE26713/Resultado_MYBTAL2.xlsx')
GSE26713_MYBTLX1<-readxl::read_excel('datasets/GSE26713/Resultado_MYBTLX1.xlsx')

GSE26713_MYBTLX3<-readxl::read_excel('datasets/GSE26713/Resultado_MYBTLX3.xlsx')
GSE26713_TAL1TAL2LMO1<-readxl::read_excel('datasets/GSE26713/Resultado_TAL1TAL2LMO1.xlsx')


GSE26713_TAL1TLX1<-readxl::read_excel('datasets/GSE26713/Resultado_TAL1TLX1.xlsx')
GSE26713_TAL1TLX3<-readxl::read_excel('datasets/GSE26713/Resultado_TAL1TLX3.xlsx')

GSE26713_TAL2LMO1<-readxl::read_excel('datasets/GSE26713/Resultado_TAL2LMO1.xlsx')
GSE26713_TALBM<-readxl::read_excel('datasets/GSE26713/Resultado_TALBM.xlsx')


GSE26713_TALMYB<-readxl::read_excel('datasets/GSE26713/Resultado_TALMYB.xlsx')
GSE26713_TALTAL2LMO1<-readxl::read_excel('datasets/GSE26713/Resultado_TALTAL2LMO1.xlsx')

GSE26713_TALTLX<-readxl::read_excel('datasets/GSE26713/Resultado_TALTLX.xlsx')
GSE26713_TALTLX1<-readxl::read_excel('datasets/GSE26713/Resultado_TALTLX1.xlsx')


GSE26713_TALTLX3<-readxl::read_excel('datasets/GSE26713/Resultado_TALTLX3.xlsx')
GSE26713_TALTLXL<-readxl::read_excel('datasets/GSE26713/Resultado_TALTLXL.xlsx')


GSE26713_TLX<-readxl::read_excel('datasets/GSE26713/Resultado_TLX.xlsx')
GSE26713_TLXBM<-readxl::read_excel('datasets/GSE26713/Resultado_TLXBM.xlsx')



GSE26713_TLXMYB<-readxl::read_excel('datasets/GSE26713/Resultado_TLXMYB.xlsx')

GSE26713_TLXTAL1<-readxl::read_excel('datasets/GSE26713/Resultado_TLXTAL1.xlsx')

GSE26713_TLXTAL2<-readxl::read_excel('datasets/GSE26713/Resultado_TLXTAL2.xlsx')

GSE26713_TLXTAL2LMO1<-readxl::read_excel('datasets/GSE26713/Resultado_TLXTAL2LMO1.xlsx')


```

```{r, include=TRUE, message=TRUE, echo=TRUE}
ls()
```

## Resultados sin comparar

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_BMHOXA$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_BMHOXA$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_BMHOXA$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_BMHOXA$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_BMHOXA$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_BMHOXA$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_BMHOXA$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_BMHOXA.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_BMLMO1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_BMLMO1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_BMLMO1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_BMLMO1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_BMLMO1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_BMLMO1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_BMLMO1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_BMLMO1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_BMLMO2$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_BMLMO2$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_BMLMO2$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_BMLMO2$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_BMLMO2$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_BMLMO2$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_BMLMO2$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_BMLMO2.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_BMMYB$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_BMMYB$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_BMMYB$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_BMMYB$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_BMMYB$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_BMMYB$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_BMMYB$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_BMMYB.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_BMTAL1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_BMTAL1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_BMTAL1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_BMTAL1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_BMTAL1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_BMTAL1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_BMTAL1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_BMTAL1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_BMTAL2$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_BMTAL2$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_BMTAL2$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_BMTAL2$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_BMTAL2$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_BMTAL2$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_BMTAL2$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_BMTAL2.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_BMTAL2LMO1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_BMTAL2LMO1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_BMTAL2LMO1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_BMTAL2LMO1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_BMTAL2LMO1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_BMTAL2LMO1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_BMTAL2LMO1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_BMTAL2LMO1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_BMTLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_BMTLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_BMTLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_BMTLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_BMTLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_BMTLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_BMTLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_BMTLX1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_BMTLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_BMTLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_BMTLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_BMTLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_BMTLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_BMTLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_BMTLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_BMTLX3.xlsx',
  col.names = TRUE)
```




```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_HOXALMO2$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_HOXALMO2$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_HOXALMO2$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_HOXALMO2$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_HOXALMO2$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_HOXALMO2$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_HOXALMO2$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_HOXALMO2.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_HOXATAL1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_HOXATAL1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_HOXATAL1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_HOXATAL1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_HOXATAL1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_HOXATAL1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_HOXATAL1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_HOXATAL1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_HOXATAL2LMO1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_HOXATAL2LMO1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_HOXATAL2LMO1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_HOXATAL2LMO1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_HOXATAL2LMO1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_HOXATAL2LMO1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_HOXATAL2LMO1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_HOXATAL2LMO1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_HOXATLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_HOXATLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_HOXATLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_HOXATLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_HOXATLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_HOXATLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_HOXATLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_HOXATLX1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_HOXATLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_HOXATLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_HOXATLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_HOXATLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_HOXATLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_HOXATLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_HOXATLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_HOXATLX3.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

# 16 por aqui
i<-grep("RNA", GSE26713_LMO1LMO2MYB$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMO1LMO2MYB$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMO1LMO2MYB$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMO1LMO2MYB$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMO1LMO2MYB$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMO1LMO2MYB$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMO1LMO2MYB$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMO1LMO2MYB.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_LMO2TAL2LMO1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMO2TAL2LMO1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMO2TAL2LMO1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMO2TAL2LMO1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMO2TAL2LMO1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMO2TAL2LMO1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMO2TAL2LMO1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMO2TAL2LMO1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_LMO2TLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMO2TLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMO2TLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMO2TLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMO2TLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMO2TLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMO2TLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMO2TLX1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_LMO2TLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMO2TLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMO2TLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMO2TLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMO2TLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMO2TLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMO2TLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMO2TLX3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_LMOBM$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMOBM$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMOBM$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMOBM$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMOBM$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMOBM$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMOBM$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMOBM.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_LMOTAL1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMOTAL1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMOTAL1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMOTAL1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMOTAL1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMOTAL1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMOTAL1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMOTAL1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_LMOTAL2$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMOTAL2$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMOTAL2$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMOTAL2$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMOTAL2$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMOTAL2$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMOTAL2$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMOTAL2.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_LMOTAL2LMO1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMOTAL2LMO1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMOTAL2LMO1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMOTAL2LMO1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMOTAL2LMO1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMOTAL2LMO1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMOTAL2LMO1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMOTAL2LMO1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_LMOTLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMOTLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMOTLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMOTLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMOTLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMOTLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMOTLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMOTLX1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_LMOTLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_LMOTLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_LMOTLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_LMOTLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_LMOTLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_LMOTLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_LMOTLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_LMOTLX3.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_MYBTAL2$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_MYBTAL2$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_MYBTAL2$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_MYBTAL2$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_MYBTAL2$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_MYBTAL2$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_MYBTAL2$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_MYBTAL2.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_MYBTLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_MYBTLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_MYBTLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_MYBTLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_MYBTLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_MYBTLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_MYBTLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_MYBTLX1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_MYBTLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_MYBTLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_MYBTLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_MYBTLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_MYBTLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_MYBTLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_MYBTLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_MYBTLX3.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TAL1TAL2LMO1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TAL1TAL2LMO1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TAL1TAL2LMO1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TAL1TAL2LMO1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TAL1TAL2LMO1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TAL1TAL2LMO1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TAL1TAL2LMO1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TAL1TAL2LMO1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TAL1TLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TAL1TLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TAL1TLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TAL1TLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TAL1TLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TAL1TLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TAL1TLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TAL1TLX1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TAL1TLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TAL1TLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TAL1TLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TAL1TLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TAL1TLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TAL1TLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TAL1TLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TAL1TLX3.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TAL2LMO1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TAL2LMO1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TAL2LMO1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TAL2LMO1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TAL2LMO1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TAL2LMO1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TAL2LMO1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TAL2LMO1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TALBM$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TALBM$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TALBM$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TALBM$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TALBM$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TALBM$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TALBM$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TALBM.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TALMYB$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TALMYB$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TALMYB$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TALMYB$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TALMYB$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TALMYB$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TALMYB$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TALMYB.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TALTAL2LMO1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TALTAL2LMO1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TALTAL2LMO1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TALTAL2LMO1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TALTAL2LMO1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TALTAL2LMO1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TALTAL2LMO1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TALTAL2LMO1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TALTLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TALTLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TALTLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TALTLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TALTLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TALTLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TALTLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TALTLX.xlsx',
  col.names = TRUE)
```





```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TALTLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TALTLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TALTLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TALTLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TALTLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TALTLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TALTLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TALTLX1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TALTLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TALTLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TALTLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TALTLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TALTLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TALTLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TALTLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TALTLX3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TALTLXL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TALTLXL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TALTLXL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TALTLXL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TALTLXL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TALTLXL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TALTLXL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TALTLXL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_GSE26713_TLX.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TLXBM$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TLXBM$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TLXBM$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TLXBM$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TLXBM$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TLXBM$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TLXBM$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TLXBM.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TLXMYB$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TLXMYB$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TLXMYB$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TLXMYB$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TLXMYB$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TLXMYB$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TLXMYB$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TLXMYB.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TLXTAL1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TLXTAL1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TLXTAL1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TLXTAL1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TLXTAL1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TLXTAL1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TLXTAL1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TLXTAL1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TLXTAL2$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TLXTAL2$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TLXTAL2$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TLXTAL2$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TLXTAL2$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TLXTAL2$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TLXTAL2$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TLXTAL2.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE26713_TLXTAL2LMO1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE26713_TLXTAL2LMO1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE26713_TLXTAL2LMO1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE26713_TLXTAL2LMO1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE26713_TLXTAL2LMO1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE26713_TLXTAL2LMO1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE26713_TLXTAL2LMO1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE26713/RNA_splicing_GSE26713_TLXTAL2LMO1.xlsx',
  col.names = TRUE)
```

# GSE28703

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
GSE28703<-readxl::read_excel('datasets/GSE28703/Resultado_GSE28703.xlsx')
```

## Resultados sin comparar

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE28703$Biological_Description, value = FALSE)
j<-grep("splicing", GSE28703$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE28703$Biological_Description, value = FALSE) 
l=grep("RBP", GSE28703$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE28703$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE28703$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE28703$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE28703/RNA_splicing_GSE28703.xlsx',
  col.names = TRUE)
```

# GSE32215

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
GSE32215<-readxl::read_excel('datasets/GSE32215/Resultado_GSE32215.xlsx')
```

## Resultados sin comparar

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE32215$Biological_Description, value = FALSE)
j<-grep("splicing", GSE32215$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE32215$Biological_Description, value = FALSE) 
l=grep("RBP", GSE32215$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE32215$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE32215$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE32215$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE32215/RNA_splicing_GSE32215.xlsx',
  col.names = TRUE)
```

# GSE33470

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
GSE33470_CD3<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3.xlsx')
GSE33470_CD34CD4<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD34CD4.xlsx')
GSE33470_CD34DPCD3<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD34DPCD3.xlsx')
GSE33470_CD34masmasCD4<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD34masmasCD4.xlsx')
GSE33470_CD34masmasDPCD3mas<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD34masmasDPCD3mas.xlsx')
GSE33470_CD34masmasDPCD3menos<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD34masmasDPCD3menos.xlsx')
GSE33470_CD34masmenosCD4<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD34masmenosCD4.xlsx')
GSE33470_CD34masmenosDPCD3mas<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD34masmenosDPCD3mas.xlsx')
GSE33470_CD34masmenosDPCD3menos<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD34masmenosDPCD3menos.xlsx')
GSE33470_CD3CD34<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3CD34.xlsx')
GSE33470_CD3CD4<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3CD4.xlsx')
GSE33470_CD3DPCD3<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3DPCD3.xlsx')
GSE33470_CD3masCD4CD34masmeno<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3masCD4CD34masmeno.xlsx')
GSE33470_CD3masCD4CD4<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3masCD4CD4.xlsx')
GSE33470_CD3masCD4DPCD3mas<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3masCD4DPCD3mas.xlsx')
GSE33470_CD3masCD4DPCD3menos<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3masCD4DPCD3menos.xlsx')
GSE33470_CD3masCD8CD34masmas<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3masCD8CD34masmas.xlsx')
GSE33470_CD3masCD8CD34masmenos<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3masCD8CD34masmenos.xlsx')
GSE33470_CD3masCD8CD4<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3masCD8CD4.xlsx')
GSE33470_CD3masCD8DPCD3mas<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3masCD8DPCD3mas.xlsx')
GSE33470_CD3masCD8DPCD3menos<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD3masCD8DPCD3menos.xlsx')
GSE33470_CD4CD34masmas<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD4CD34masmas.xlsx')
GSE33470_CD4DPCD3<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD4DPCD3.xlsx')
GSE33470_CD4DPCD3mas<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD4DPCD3mas.xlsx')
GSE33470_CD4DPCD3menos<-readxl::read_excel('datasets/GSE33470/Resultado_GSE33470_CD4DPCD3menos.xlsx')


```



```{r, include=TRUE, message=TRUE, echo=TRUE}
ls()
```

## Resultados sin comparar

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34CD4$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34CD4$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34CD4$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34CD4$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34CD4$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34CD4$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34CD4$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34CD4.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34DPCD3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34DPCD3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34DPCD3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34DPCD3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34DPCD3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34DPCD3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34DPCD3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34DPCD3.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmasCD4$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmasCD4$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmasCD4$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmasCD4$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmasCD4$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmasCD4$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmasCD4$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmasCD4.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmasDPCD3mas$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmasDPCD3mas$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmasDPCD3mas$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmasDPCD3mas$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmasDPCD3mas$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmasDPCD3mas$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmasDPCD3mas$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmasDPCD3mas.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmasDPCD3menos$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmasDPCD3menos$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmasDPCD3menos$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmasDPCD3menos$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmasDPCD3menos$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmasDPCD3menos$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmasDPCD3menos$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmasDPCD3menos.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmenosCD4$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmenosCD4$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmenosCD4$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmenosCD4$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmenosCD4$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmenosCD4$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmenosCD4$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmenosCD4.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmenosDPCD3mas$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmenosDPCD3mas$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmenosDPCD3mas$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmenosDPCD3mas$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmenosDPCD3mas$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmenosDPCD3mas$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmenosDPCD3mas$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmenosDPCD3mas.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmenosDPCD3menos$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmenosDPCD3menos$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmenosDPCD3menos$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmenosDPCD3menos$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmenosDPCD3menos$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmenosDPCD3menos$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmenosDPCD3menos$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmenosDPCD3menos.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3CD34$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3CD34$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3CD34$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3CD34$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3CD34$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3CD34$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3CD34$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3CD34.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3CD4$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3CD4$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3CD4$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3CD4$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3CD4$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3CD4$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3CD4$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3CD4.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3DPCD3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3DPCD3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3DPCD3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3DPCD3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3DPCD3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3DPCD3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3DPCD3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3DPCD3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmasCD4$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmasCD4$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmasCD4$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmasCD4$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmasCD4$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmasCD4$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmasCD4$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmasCD4.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmasDPCD3mas$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmasDPCD3mas$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmasDPCD3mas$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmasDPCD3mas$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmasDPCD3mas$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmasDPCD3mas$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmasDPCD3mas$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmasDPCD3mas.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmasDPCD3menos$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmasDPCD3menos$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmasDPCD3menos$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmasDPCD3menos$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmasDPCD3menos$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmasDPCD3menos$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmasDPCD3menos$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmasDPCD3menos.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmenosCD4$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmenosCD4$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmenosCD4$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmenosCD4$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmenosCD4$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmenosCD4$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmenosCD4$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmenosCD4.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmenosDPCD3mas$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmenosDPCD3mas$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmenosDPCD3mas$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmenosDPCD3mas$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmenosDPCD3mas$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmenosDPCD3mas$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmenosDPCD3mas$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmenosDPCD3mas.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD34masmenosDPCD3menos$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD34masmenosDPCD3menos$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD34masmenosDPCD3menos$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD34masmenosDPCD3menos$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD34masmenosDPCD3menos$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD34masmenosDPCD3menos$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD34masmenosDPCD3menos$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD34masmenosDPCD3menos.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3CD34$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3CD34$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3CD34$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3CD34$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3CD34$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3CD34$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3CD34$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3CD34.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3CD4$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3CD4$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3CD4$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3CD4$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3CD4$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3CD4$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3CD4$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3CD4.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3DPCD3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3DPCD3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3DPCD3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3DPCD3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3DPCD3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3DPCD3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3DPCD3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3DPCD3.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3masCD4CD34masmeno$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3masCD4CD34masmeno$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3masCD4CD34masmeno$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3masCD4CD34masmeno$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3masCD4CD34masmeno$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3masCD4CD34masmeno$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3masCD4CD34masmeno$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3masCD4CD34masmeno.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3masCD4CD4$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3masCD4CD4$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3masCD4CD4$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3masCD4CD4$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3masCD4CD4$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3masCD4CD4$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3masCD4CD4$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3masCD4CD4.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3masCD4DPCD3mas$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3masCD4DPCD3mas$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3masCD4DPCD3mas$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3masCD4DPCD3mas$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3masCD4DPCD3mas$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3masCD4DPCD3mas$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3masCD4DPCD3mas$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3masCD4DPCD3mas.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3masCD4DPCD3menos$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3masCD4DPCD3menos$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3masCD4DPCD3menos$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3masCD4DPCD3menos$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3masCD4DPCD3menos$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3masCD4DPCD3menos$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3masCD4DPCD3menos$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3masCD4DPCD3menos.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3masCD8CD34masmas$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3masCD8CD34masmas$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3masCD8CD34masmas$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3masCD8CD34masmas$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3masCD8CD34masmas$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3masCD8CD34masmas$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3masCD8CD34masmas$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3masCD8CD34masmas.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3masCD8CD34masmenos$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3masCD8CD34masmenos$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3masCD8CD34masmenos$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3masCD8CD34masmenos$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3masCD8CD34masmenos$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3masCD8CD34masmenos$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3masCD8CD34masmenos$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3masCD8CD34masmenos.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3masCD8CD4$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3masCD8CD4$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3masCD8CD4$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3masCD8CD4$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3masCD8CD4$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3masCD8CD4$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3masCD8CD4$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3masCD8CD4.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3masCD8DPCD3mas$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3masCD8DPCD3mas$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3masCD8DPCD3mas$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3masCD8DPCD3mas$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3masCD8DPCD3mas$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3masCD8DPCD3mas$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3masCD8DPCD3mas$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3masCD8DPCD3mas.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD3masCD8DPCD3menos$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD3masCD8DPCD3menos$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD3masCD8DPCD3menos$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD3masCD8DPCD3menos$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD3masCD8DPCD3menos$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD3masCD8DPCD3menos$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD3masCD8DPCD3menos$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD3masCD8DPCD3menos.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD4CD34masmas$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD4CD34masmas$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD4CD34masmas$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD4CD34masmas$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD4CD34masmas$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD4CD34masmas$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD4CD34masmas$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD4CD34masmas.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD4DPCD3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD4DPCD3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD4DPCD3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD4DPCD3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD4DPCD3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD4DPCD3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD4DPCD3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD4DPCD3.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD4DPCD3mas$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD4DPCD3mas$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD4DPCD3mas$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD4DPCD3mas$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD4DPCD3mas$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD4DPCD3mas$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD4DPCD3mas$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD4DPCD3mas.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE33470_CD4DPCD3menos$Biological_Description, value = FALSE)
j<-grep("splicing", GSE33470_CD4DPCD3menos$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE33470_CD4DPCD3menos$Biological_Description, value = FALSE) 
l=grep("RBP", GSE33470_CD4DPCD3menos$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE33470_CD4DPCD3menos$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE33470_CD4DPCD3menos$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE33470_CD4DPCD3menos$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE33470/RNA_splicing_GSE33470_CD4DPCD3menos.xlsx',
  col.names = TRUE)
```



# GSE62156

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
GSE62156_CALMAF10_CALMHOX<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMHOX.xlsx')
GSE62156_CALMAF10_CALMHOXAUnk<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMHOXAUnk.xlsx')
GSE62156_CALMAF10_CALMImmature<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMImmature.xlsx')
GSE62156_CALMAF10_CALMImmatureHOXA<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMImmatureHOXA.xlsx')
GSE62156_CALMAF10_CALMMLL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMMLL.xlsx')
GSE62156_CALMAF10_CALMTAL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMTAL.xlsx')
GSE62156_CALMAF10_CALMTLX1<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMTLX1.xlsx')
GSE62156_CALMAF10_CALMTLX3<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_CALMAF10_CALMTLX3.xlsx')
GSE62156_HOX_CALM<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOX_CALM.xlsx')
GSE62156_HOX_ImHOXA<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOX_ImHOXA.xlsx')
GSE62156_HOX_Immature<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOX_Immature.xlsx')
GSE62156_HOX_ImmHOXA<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOX_ImmHOXA.xlsx')
GSE62156_HOX_MLL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOX_MLL.xlsx')
GSE62156_HOX_TAL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOX_TAL.xlsx')
GSE62156_HOX_TLX<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOX_TLX.xlsx')
GSE62156_HOX_TLX1<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOX_TLX1.xlsx')
GSE62156_HOX_TLX3<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOX_TLX3.xlsx')
GSE62156_HOXAUnk_HOX<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXAUnk_HOX.xlsx')
GSE62156_HOXAUnk_Imm<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXAUnk_Imm.xlsx')
GSE62156_HOXAUnk_ImmHOXA<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXAUnk_ImmHOXA.xlsx')
GSE62156_HOXAUnk_MLL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXAUnk_MLL.xlsx')
GSE62156_HOXAUnk_TLX1<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXAUnk_TLX1.xlsx')
GSE62156_HOXAUnk_TLX3<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXAUnk_TLX3.xlsx')
GSE62156_HOXs_MLL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXs_MLL.xlsx')
GSE62156_HOXs_TAL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXs_TAL.xlsx')
GSE62156_HOXs_TLX1<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXs_TLX1.xlsx')
GSE62156_HOXs_TLX3<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_HOXs_TLX3.xlsx')
GSE62156_Imm_CALM<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Imm_CALM.xlsx')
GSE62156_Imm_HOX<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Imm_HOX.xlsx')
GSE62156_Imm_HOXAUnk<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Imm_HOXAUnk.xlsx')
GSE62156_Imm_MLL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Imm_MLL.xlsx')
GSE62156_Imm_TAL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Imm_TAL.xlsx')
GSE62156_Imm_TLX<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Imm_TLX.xlsx')
GSE62156_Imm_TLX1<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Imm_TLX1.xlsx')
GSE62156_Imm_TLX3<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Imm_TLX3.xlsx')
GSE62156_Immature_ImmHOXA<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Immature_ImmHOXA.xlsx')
GSE62156_Immature_MLL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Immature_MLL.xlsx')
GSE62156_Immature_TAL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Immature_TAL.xlsx')
GSE62156_Immature_TLX1<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Immature_TLX1.xlsx')
GSE62156_Immature_TLX3<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_Immature_TLX3.xlsx')
GSE62156_ImmHOXA_MLL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_ImmHOXA_MLL.xlsx')
GSE62156_ImmHOXA_TAL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_ImmHOXA_TAL.xlsx')
GSE62156_ImmHOXA_TLX1<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_ImmHOXA_TLX1.xlsx')
GSE62156_ImmHOXA_TLX3<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_ImmHOXA_TLX3.xlsx')
GSE62156_MLL_TAL<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_MLL_TAL.xlsx')
GSE62156_MLL_TLX1<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_MLL_TLX1.xlsx')
GSE62156_MLL_TLX3<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_MLL_TLX3.xlsx')
GSE62156_TLX<-readxl::read_excel('datasets/GSE62156/Resultado_GSE62156_TLX.xlsx')

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
ls()

```


## Resultados sin comparar

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_CALMAF10_CALMHOX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_CALMAF10_CALMHOX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_CALMAF10_CALMHOX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_CALMAF10_CALMHOX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_CALMAF10_CALMHOX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_CALMAF10_CALMHOX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_CALMAF10_CALMHOX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_CALMAF10_CALMHOX.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_CALMAF10_CALMHOXAUnk$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_CALMAF10_CALMHOXAUnk$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_CALMAF10_CALMHOXAUnk$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_CALMAF10_CALMHOXAUnk$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_CALMAF10_CALMHOXAUnk$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_CALMAF10_CALMHOXAUnk$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_CALMAF10_CALMHOXAUnk$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_CALMAF10_CALMHOXAUnk.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_CALMAF10_CALMImmature$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_CALMAF10_CALMImmature$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_CALMAF10_CALMImmature$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_CALMAF10_CALMImmature$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_CALMAF10_CALMImmature$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_CALMAF10_CALMImmature$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_CALMAF10_CALMImmature$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_CALMAF10_CALMImmature.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_CALMAF10_CALMImmatureHOXA$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_CALMAF10_CALMImmatureHOXA$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_CALMAF10_CALMImmatureHOXA$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_CALMAF10_CALMImmatureHOXA$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_CALMAF10_CALMImmatureHOXA$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_CALMAF10_CALMImmatureHOXA$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_CALMAF10_CALMImmatureHOXA$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_CALMAF10_CALMImmatureHOXA.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_CALMAF10_CALMMLL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_CALMAF10_CALMMLL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_CALMAF10_CALMMLL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_CALMAF10_CALMMLL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_CALMAF10_CALMMLL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_CALMAF10_CALMMLL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_CALMAF10_CALMMLL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_CALMAF10_CALMMLL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_CALMAF10_CALMTAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_CALMAF10_CALMTAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_CALMAF10_CALMTAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_CALMAF10_CALMTAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_CALMAF10_CALMTAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_CALMAF10_CALMTAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_CALMAF10_CALMTAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_CALMAF10_CALMTAL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_CALMAF10_CALMTLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_CALMAF10_CALMTLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_CALMAF10_CALMTLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_CALMAF10_CALMTLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_CALMAF10_CALMTLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_CALMAF10_CALMTLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_CALMAF10_CALMTLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_CALMAF10_CALMTLX1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_CALMAF10_CALMTLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_CALMAF10_CALMTLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_CALMAF10_CALMTLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_CALMAF10_CALMTLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_CALMAF10_CALMTLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_CALMAF10_CALMTLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_CALMAF10_CALMTLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_CALMAF10_CALMTLX3.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOX_CALM$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOX_CALM$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOX_CALM$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOX_CALM$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOX_CALM$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOX_CALM$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOX_CALM$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOX_CALM.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOX_ImHOXA$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOX_ImHOXA$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOX_ImHOXA$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOX_ImHOXA$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOX_ImHOXA$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOX_ImHOXA$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOX_ImHOXA$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOX_ImHOXA.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOX_Immature$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOX_Immature$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOX_Immature$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOX_Immature$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOX_Immature$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOX_Immature$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOX_Immature$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOX_Immature.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOX_ImmHOXA$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOX_ImmHOXA$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOX_ImmHOXA$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOX_ImmHOXA$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOX_ImmHOXA$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOX_ImmHOXA$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOX_ImmHOXA$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOX_ImmHOXA.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOX_MLL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOX_MLL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOX_MLL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOX_MLL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOX_MLL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOX_MLL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOX_MLL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOX_MLL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOX_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOX_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOX_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOX_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOX_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOX_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOX_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOX_TAL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOX_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOX_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOX_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOX_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOX_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOX_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOX_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOX_TLX.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOX_TLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOX_TLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOX_TLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOX_TLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOX_TLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOX_TLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOX_TLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOX_TLX1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOX_TLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOX_TLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOX_TLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOX_TLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOX_TLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOX_TLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOX_TLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOX_TLX3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXAUnk_HOX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXAUnk_HOX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXAUnk_HOX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXAUnk_HOX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXAUnk_HOX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXAUnk_HOX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXAUnk_HOX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXAUnk_HOX.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXAUnk_Imm$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXAUnk_Imm$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXAUnk_Imm$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXAUnk_Imm$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXAUnk_Imm$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXAUnk_Imm$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXAUnk_Imm$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXAUnk_Imm.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXAUnk_ImmHOXA$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXAUnk_ImmHOXA$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXAUnk_ImmHOXA$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXAUnk_ImmHOXA$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXAUnk_ImmHOXA$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXAUnk_ImmHOXA$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXAUnk_ImmHOXA$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXAUnk_ImmHOXA.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXAUnk_MLL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXAUnk_MLL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXAUnk_MLL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXAUnk_MLL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXAUnk_MLL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXAUnk_MLL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXAUnk_MLL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXAUnk_MLL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXAUnk_TLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXAUnk_TLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXAUnk_TLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXAUnk_TLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXAUnk_TLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXAUnk_TLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXAUnk_TLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXAUnk_TLX1.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXAUnk_TLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXAUnk_TLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXAUnk_TLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXAUnk_TLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXAUnk_TLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXAUnk_TLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXAUnk_TLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXAUnk_TLX3.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXs_MLL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXs_MLL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXs_MLL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXs_MLL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXs_MLL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXs_MLL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXs_MLL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXs_MLL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXs_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXs_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXs_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXs_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXs_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXs_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXs_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXs_TAL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXs_TLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXs_TLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXs_TLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXs_TLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXs_TLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXs_TLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXs_TLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXs_TLX1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_HOXs_TLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_HOXs_TLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_HOXs_TLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_HOXs_TLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_HOXs_TLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_HOXs_TLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_HOXs_TLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_HOXs_TLX3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Imm_CALM$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Imm_CALM$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Imm_CALM$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Imm_CALM$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Imm_CALM$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Imm_CALM$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Imm_CALM$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Imm_CALM.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Imm_HOX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Imm_HOX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Imm_HOX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Imm_HOX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Imm_HOX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Imm_HOX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Imm_HOX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Imm_HOX.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Imm_HOXAUnk$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Imm_HOXAUnk$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Imm_HOXAUnk$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Imm_HOXAUnk$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Imm_HOXAUnk$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Imm_HOXAUnk$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Imm_HOXAUnk$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Imm_HOXAUnk.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Imm_MLL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Imm_MLL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Imm_MLL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Imm_MLL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Imm_MLL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Imm_MLL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Imm_MLL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Imm_MLL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Imm_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Imm_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Imm_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Imm_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Imm_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Imm_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Imm_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Imm_TAL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Imm_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Imm_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Imm_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Imm_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Imm_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Imm_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Imm_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Imm_TLX.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Imm_TLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Imm_TLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Imm_TLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Imm_TLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Imm_TLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Imm_TLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Imm_TLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Imm_TLX1.xlsx',
  col.names = TRUE)
```



```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Imm_TLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Imm_TLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Imm_TLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Imm_TLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Imm_TLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Imm_TLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Imm_TLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Imm_TLX3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Immature_ImmHOXA$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Immature_ImmHOXA$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Immature_ImmHOXA$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Immature_ImmHOXA$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Immature_ImmHOXA$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Immature_ImmHOXA$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Immature_ImmHOXA$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Immature_ImmHOXA.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Immature_MLL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Immature_MLL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Immature_MLL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Immature_MLL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Immature_MLL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Immature_MLL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Immature_MLL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Immature_MLL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Immature_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Immature_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Immature_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Immature_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Immature_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Immature_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Immature_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Immature_TAL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Immature_TLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Immature_TLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Immature_TLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Immature_TLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Immature_TLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Immature_TLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Immature_TLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Immature_TLX1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_Immature_TLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_Immature_TLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_Immature_TLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_Immature_TLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_Immature_TLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_Immature_TLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_Immature_TLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_Immature_TLX3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_ImmHOXA_MLL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_ImmHOXA_MLL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_ImmHOXA_MLL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_ImmHOXA_MLL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_ImmHOXA_MLL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_ImmHOXA_MLL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_ImmHOXA_MLL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_ImmHOXA_MLL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_ImmHOXA_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_ImmHOXA_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_ImmHOXA_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_ImmHOXA_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_ImmHOXA_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_ImmHOXA_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_ImmHOXA_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_ImmHOXA_TAL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_ImmHOXA_TLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_ImmHOXA_TLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_ImmHOXA_TLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_ImmHOXA_TLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_ImmHOXA_TLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_ImmHOXA_TLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_ImmHOXA_TLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_ImmHOXA_TLX1.xlsx',
  col.names = TRUE)
```





```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_ImmHOXA_TLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_ImmHOXA_TLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_ImmHOXA_TLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_ImmHOXA_TLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_ImmHOXA_TLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_ImmHOXA_TLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_ImmHOXA_TLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_ImmHOXA_TLX3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_MLL_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_MLL_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_MLL_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_MLL_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_MLL_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_MLL_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_MLL_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_MLL_TAL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_MLL_TLX1$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_MLL_TLX1$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_MLL_TLX1$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_MLL_TLX1$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_MLL_TLX1$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_MLL_TLX1$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_MLL_TLX1$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_MLL_TLX1.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_MLL_TLX3$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_MLL_TLX3$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_MLL_TLX3$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_MLL_TLX3$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_MLL_TLX3$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_MLL_TLX3$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_MLL_TLX3$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_MLL_TLX3.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE62156_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE62156_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE62156_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE62156_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE62156_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE62156_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE62156_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE62156/GSE62156_TLX.xlsx',
  col.names = TRUE)
```

# GSE110633

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
GSE110633_HOXA_IMM<-readxl::read_excel('datasets/GSE110633/Resultado_GSE110633_HOXA_IMM.xlsx')
GSE110633_HOXA_TAL<-readxl::read_excel('datasets/GSE110633/Resultado_GSE110633_HOXA_TAL.xlsx')
GSE110633_HOXA_TLX<-readxl::read_excel('datasets/GSE110633/Resultado_GSE110633_HOXA_TLX.xlsx')
GSE110633_IMM_TAL<-readxl::read_excel('datasets/GSE110633/Resultado_GSE110633_IMM_TAL.xlsx')
GSE110633_IMM_TLX<-readxl::read_excel('datasets/GSE110633/Resultado_GSE110633_IMM_TLX.xlsx')
GSE110633_TAL_TLX<-readxl::read_excel('datasets/GSE110633/Resultado_GSE110633_TAL_TLX.xlsx')
```

## Resultados sin comparar

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110633_HOXA_IMM$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110633_HOXA_IMM$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110633_HOXA_IMM$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110633_HOXA_IMM$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110633_HOXA_IMM$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110633_HOXA_IMM$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110633_HOXA_IMM$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/RNA_splicing_GSE110633_HOXA_IMM.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110633_HOXA_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110633_HOXA_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110633_HOXA_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110633_HOXA_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110633_HOXA_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110633_HOXA_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110633_HOXA_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/RNA_splicing_GSE110633_HOXA_TAL.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110633_HOXA_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110633_HOXA_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110633_HOXA_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110633_HOXA_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110633_HOXA_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110633_HOXA_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110633_HOXA_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/RNA_splicing_GSE110633_HOXA_TLX.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110633_IMM_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110633_IMM_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110633_IMM_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110633_IMM_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110633_IMM_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110633_IMM_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110633_IMM_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/RNA_splicing_GSE110633_IMM_TAL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110633_IMM_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110633_IMM_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110633_IMM_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110633_IMM_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110633_IMM_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110633_IMM_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110633_IMM_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/RNA_splicing_GSE110633_IMM_TLX.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110633_TAL_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110633_TAL_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110633_TAL_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110633_TAL_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110633_TAL_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110633_TAL_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110633_TAL_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110633/RNA_splicing_GSE110633_TAL_TLX.xlsx',
  col.names = TRUE)
```


# GSE110636

## Carga de las bases

Cambiar las rutas para encontrar los archivos mencionados:

```{r, include=TRUE, message=TRUE, echo=TRUE}
library(readxl)
GSE110636_HOXA_IMM<-readxl::read_excel('datasets/GSE110636/Resultado_GSE110636_HOXA_IMM.xlsx')
GSE110636_HOXA_TAL<-readxl::read_excel('datasets/GSE110636/Resultado_GSE110636_HOXA_TAL.xlsx')
GSE110636_HOXA_TLX<-readxl::read_excel('datasets/GSE110636/Resultado_GSE110636_HOXA_TLX.xlsx')
GSE110636_IMM_TAL<-readxl::read_excel('datasets/GSE110636/Resultado_GSE110636_IMM_TAL.xlsx')
GSE110636_IMM_TLX<-readxl::read_excel('datasets/GSE110636/Resultado_GSE110636_IMM_TLX.xlsx')
GSE110636_TAL_TLX<-readxl::read_excel('datasets/GSE110636/Resultado_GSE110636_TAL_TLX.xlsx')
```

## Resultados sin comparar

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110636_HOXA_IMM$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110636_HOXA_IMM$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110636_HOXA_IMM$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110636_HOXA_IMM$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110636_HOXA_IMM$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110636_HOXA_IMM$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110636_HOXA_IMM$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/RNA_splicing_GSE110636_HOXA_IMM.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110636_HOXA_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110636_HOXA_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110636_HOXA_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110636_HOXA_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110636_HOXA_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110636_HOXA_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110636_HOXA_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/RNA_splicing_GSE110636_HOXA_TAL.xlsx',
  col.names = TRUE)
```

```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110636_HOXA_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110636_HOXA_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110636_HOXA_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110636_HOXA_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110636_HOXA_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110636_HOXA_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110636_HOXA_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/RNA_splicing_GSE110636_HOXA_TLX.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110636_IMM_TAL$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110636_IMM_TAL$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110636_IMM_TAL$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110636_IMM_TAL$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110636_IMM_TAL$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110636_IMM_TAL$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110636_IMM_TAL$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/RNA_splicing_GSE110636_IMM_TAL.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110636_IMM_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110636_IMM_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110636_IMM_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110636_IMM_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110636_IMM_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110636_IMM_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110636_IMM_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/RNA_splicing_GSE110636_IMM_TLX.xlsx',
  col.names = TRUE)
```


```{r, include=TRUE, message=TRUE, echo=TRUE}

i<-grep("RNA", GSE110636_TAL_TLX$Biological_Description, value = FALSE)
j<-grep("splicing", GSE110636_TAL_TLX$Biological_Description, value = FALSE) 
k<-grep("Splicing", GSE110636_TAL_TLX$Biological_Description, value = FALSE) 
l=grep("RBP", GSE110636_TAL_TLX$Biological_Description, value = FALSE) 

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
library(dplyr)

if (length(j)>0){
for (nvalj in 1:length(j)){
  if ((j %in% i)[nvalj]==FALSE){
    i=append(i,j[nvalj])
  }}}


if (length(k)>0){
for (nvalj in 1:length(k)){
  if ((k %in% i)[nvalj]==FALSE){
    i=append(i,k[nvalj])
  }}}


if (length(l)>0){
for (nvalj in 1:length(l)){
  if ((l %in% i)[nvalj]==FALSE){
    i=append(i,l[nvalj])
  }}}



i=i[order(i)]
```


```{r, include=TRUE, message=TRUE, echo=TRUE}
Pat<-c()
Biological_Des<-c()
Gen<-c()

for (long_i in 1:length(i)){
  Pat<-append(Pat, GSE110636_TAL_TLX$Path[i[long_i]])
  Biological_Des<-append(Biological_Des, GSE110636_TAL_TLX$Biological_Description[i[long_i]])
  Gen<-append(Gen, GSE110636_TAL_TLX$Gene[i[long_i]])
  }

```


```{r, include=TRUE, message=TRUE, echo=TRUE}
RNA_splicing<-data.frame('Path'=Pat, 'Biological_Description'=Biological_Des, 'Gene'=Gen)

```



```{r, include=TRUE, message=TRUE, echo=TRUE}
library("xlsx")
write.xlsx(RNA_splicing, file='C:/Users/Copito/OneDrive/Documentos/Master bioinf y bioest/TFM/datasets/GSE110636/RNA_splicing_GSE110636_TAL_TLX.xlsx',
  col.names = TRUE)
```


















